\chapter{Subcellular Localisation of Endogenous IFIT Proteins in the Context of RSV Inclusion Bodies} \label{ch:Subcellular Localisation of Endogenous IFIT Proteins in the Context of RSV Inclusion Bodies}
\section{Background and Aims} \label{sec:Background and Aims-Chapter3}
As elucidated in the preceding chapters, human \textit{IFIT} genes exhibit induction in response to both human and bovine RSV infections, suggesting their potential contribution to an anti-RSV role \textit{in vivo}. In contrast, the responsiveness of bovine \textit{IFITs} to human and bovine RSV infections appears limited, possibly influenced by baseline expression levels of these genes. Notably, Drori \textit{et al.} reported that ectopically expressed human IFIT1, IFIT2, and IFIT3 hindered the growth of hRSV, though the precise mechanism of action remains elusive \cite{Drori2020InfluenzaProteins}. IFIT proteins, known for their diverse antiviral functions, engage in various mechanisms, differing among the IFIT proteins. These mechanisms include direct interactions of IFIT1 and IFIT5 with 5$^{\prime}$-RNA modifications \cite{Abbas2013StructuralProteins, Diamond2014IFIT1:Translation}, IFIT2's pro-apoptotic action mediated through direct interactions with mitochondrial membranes \cite{Chen2017InhibitionApoptosis}, and IFIT3-mediated potentiation of RIG-I signalling via scaffold formation between MAVS and TBK1 on mitochondrial membranes \cite{Liu2011IFN-InducedTBK1}. Additionally, IFIT1, IFIT2, and IFIT3 form heterodimer and heterotrimer protein complexes, enhancing IFIT stability and potentially influencing their functions \cite{Mears2018BetterResponse}. Examining the subcellular localisation of IFIT proteins in infected cells may serve as a proxy for understanding their mechanism of RSV inhibition. We propose that the sites of IFIT/RSV interactions are the RSV inclusion bodies, which form early in the infection cycle (6 HPI). These membraneless viral organelles, created by liquid-liquid phase separation, contain essential components of the RSV viral polymerase complex and serve as functional sites for RSV RNA transcription and replication \cite{Rincheval2017FunctionalVirus, Weber1995NonstructuralSerum, Fricke2013P38Assembly, Jobe2021BovineResponses}. IBs house the RNA genome and polyadenylated RNA, with the latter spatially segregated into substructures called IBAGs (IB-associated granules), while the genome is distributed throughout the IB structure but excluded from IBAGs \cite{Rincheval2017FunctionalVirus}. RSV IBs increase in size and complexity as they mature, with only larger, more mature IBs containing IBAGs \cite{Rincheval2017FunctionalVirus, Jobe2021BovineResponses}. IBAGs further contain the M2/1 protein, along with the eukaryotic translation initiation factors eIF4G, eIF4E, eIF3A, eIF4H, eIF4A, eIF4A1, and eIF4B \cite{Rincheval2017FunctionalVirus, Jobe2023ViralCondensates}. Given the reported interactions of IFIT1 and IFIT2 with eIF3E and eIF3C, hindering 43S-mRNA complex formation \cite{Diamond2014IFIT1:Translation, Guo2000CharacterizationVirus}, further indicating that the mechanism of action of these proteins against RSV could be mediated via IB interaction. Moreover, the maturation process of RSV IBs, characterised by increased size and complexity, with larger, mature IBs containing IBAGs, further underscores the potential significance of IFIT/IB interactions \cite{Rincheval2017FunctionalVirus, Jobe2021BovineResponses}. Our objective was to delineate the specific functions of IFIT proteins contributing to the reduction in RSV fitness \cite{Drori2020InfluenzaProteins}. To achieve this, we initiated an assessment and comparison of the subcellular localisation of human and bovine IFIT proteins under various conditions, including mock infection, treatment with human/bovine IFN\(\alpha\), and human/bovine RSV infection. Subsequently, we undertook a thorough analysis of human and bovine IFIT proteins, focusing on their interactions with the corresponding RSV IBs across multiple cell lines. These observations were made using confocal microscopy, allowing for a comprehensive exploration of the dynamic interactions between IFIT proteins and RSV IBs.

\section{Results} \label{sec:Results-Chapter3}
\input{08. Chapter 3/Subchapters/01. Localisation intorduction.tex}
\input{08. Chapter 3/Subchapters/02. Infection.tex}

\section{Conclusions} \label{sec:Conclusions-Chapter3}
To gain insights into the mechanism underlying the impact of IFIT proteins on the reduction of RSV fitness, we conducted an examination of their subcellular localisation dynamics following RSV infection in both human and bovine cell lines. Notably, both human and bovine IFIT1 demonstrated cytoplasmic localisation with nuclear exclusion in both mock-infected and RSV-infected cells. This localisation pattern aligns with prior reports on murine Ifit1 and human IFIT1 in mock conditions documented in the literature \cite{Pichlmair2011IFIT1RNA, Terenzi2008Interferon-inducibleE1, Thul2017AProteome}. Additionally, we observed an increase in the concentration of both human and bovine IFIT1 within the non-infected cell population among RSV-infected cells. This suggests a potential prophylactic role for IFIT1, wherein elevated IFIT1 expression may serve to prevent infection in cells with increased IFIT1 levels. This observation complements the findings presented in Chapter \ref{ch:Assessment of Transcriptional Induction of Human IFITs in the Context of RSV} where we observed IFN-dependent activation of \textit{hIFITs} by RSV.

The subcellular localisation of the IFIT2 protein was assessed using two distinct polyclonal antibodies, denoted as IFIT2(A) and IFIT2(B). Notably, these antibodies exhibited differential staining patterns, particularly accentuated in bovine samples. In human mock-infected samples, the IFIT2(A) antibody revealed a vesicular staining pattern, excluding the nucleus. Conversely, in bovine mock-infected cells, it exhibited a diffused cytoplasmic pattern with nuclear exclusion and perinuclear IFIT2 condensates resembling the Golgi apparatus. Although there are no previous reports associating IFIT2 with the Golgi apparatus, the Human Protein Atlas did report vesicular cytoplasmic localisation for IFIT2 \cite{Thul2017AProteome}. IFIT2(B) consistently displayed a granular cytoplasmic stain with nuclear exclusion in both human and bovine mock-infected cells. Additionally, it detected IFIT2 concentration at the mitotic spindle during cell division, aligning with previous reports in murine cell lines \cite{Saha2006IdentificationProtein}. These observations suggest that, in terms of basal staining patterns, both antibodies recapitulate information available in the literature for IFIT2. Regarding changes observed between mock and RSV-infected samples, the IFIT2(B) antibody did not detect alterations in distribution in both human and bovine samples. Conversely, the IFIT2(A) antibody revealed a decreased cytoplasmic localisation of IFIT2 and instead observed a concentration of IFIT2 in the vicinity of RSV inclusion bodies. This translocation phenotype will be discussed in more detail below.

A differential staining pattern was observed between human and bovine IFIT3 in both mock and RSV-infected cells. Human IFIT3 exhibited a diffused, nuclear-excluded cytoplasmic distribution. However, during infection, IFIT3 was also localised in the nucleus of both infected and non-infected cells. In contrast, bovine IFIT3 displayed a vesicular cytoplasmic stain, accompanied by IFIT3 concentrations in the nucleoplasm in mock-treated cells. In RSV-infected cells, bovine IFIT3 was only detectable as inclusions inside the RSV inclusion bodies and was no longer present in the nuclei of the cells. Previous publications on IFIT3 localisation have reported exogenously expressed IFIT3 to have cytoplasmic localisation with nuclear exclusion in HEK293T cells \cite{Huang2008Interferon-inducedCells, Liu2011IFN-InducedTBK1}. The Human Protein Atlas also reported granular cytoplasmic localisation with partial nuclear staining in A431 and HeLa cells \cite{Thul2017AProteome}. Thus, our results regarding the basal IFIT3 localisations are consistent with previously reported findings, although the observed discrepancy between human and bovine samples is intriguing.

Differential staining patterns were observed between human and bovine IFIT5 in both mock and RSV-infected cells. Human IFIT5 exhibited diffuse cytoplasmic staining while being excluded from the nucleus in both conditions. In contrast, bovine IFIT5 showed diffuse cytoplasmic staining, localisation in cytoplasmic vesicles, nucleoplasm, and the formation of inclusions within the nucleoli. This staining pattern remained consistent between mock and RSV-infected samples, with the exception of diminished nucleoplasmic staining in infected cells. Previous studies reported IFIT5 at ruffled membranes, colocalising with actin-rich protrusions in WI-38 VA-13 and Huh7 cell lines \cite{Katibah2013TRNAIFIT5}, or being localised within structures resembling the Golgi apparatus in A431 and SK-MEL-30 cell lines \cite{Thul2017AProteome}. However, our findings contradict these results, suggesting that the localisation of IFIT5, along with other proteins, may be cell-type and species-dependent. Notably, in human samples infected with RSV, we observed non-infected cells to present higher levels of IFIT5, a phenotype not recreated in bovine samples. The same induction phenotype was observed in both human and bovine cells for IFIT1.

While we had hoped to observe subcellular localisations of IFIT proteins coinciding with their proposed functions, such as IFIT2 or IFIT3 concentration on mitochondria, the initial analysis did not provide evidence for these patterns. Intriguingly, despite minimal induction changes of bovine IFITs to RSV infection observed in Chapter \ref{ch:Assessment of Transcriptional Induction of Bovine IFITs in the Context of RSV}, we were able to detect these proteins within the cells. Some observations, particularly regarding interactions with RSV IBs (discussed in more detail below), were consistent between human and bovine cells. This suggests that we are indeed detecting bovine IFITs and not off-target proteins. Furthermore, it indicates that the low induction levels observed previously are attributed to the increased basal expression of \textit{IFITs} in bovine cells compared to human cells. Along with this, we have observed a discrepancy between the previously established fold changes to IFN$\alpha$ stimulation and RSV infection, and the observed staining intensities. Based on the results from Chapters \ref{ch:Assessment of Transcriptional Induction of Human IFITs in the Context of RSV} and \ref{ch:Assessment of Transcriptional Induction of Bovine IFITs in the Context of RSV}, we would expect human IFIT staining intensities increase between control and treated samples, while bovine IFIT staining remaining constant between the conditions. We however only observed the signal of bovine IFIT1 to sequentially increase between the mock, bIFN$\alpha$ treated, and bRSV infected samples; while we have seen human IFIT1 and IFIT5 to maintain the staining intensity between mock and hIFN$\alpha$ treated samples, but display increased staining in hRSV infected samples; and have observed human IFIT3 and bovine IFIT5 to display increase in signal between mock and IFN$\alpha$ treated samples, which is then maintained during infection. Notably, human and bovine IFIT5, consistently the least responsive in previous chapters, exhibited clear increases in signal. 

Efforts were made to quantify these differences in detected fluorescence using CellProfiler software \cite{McQuin2018CellProfilerBiology}. The aim was to segment acquired images for cellular objects, enabling the automated and unbiased determination of mean fluorescence per cellular object. However, challenges arose, particularly in the variability of the basal acquired signal intensity, which was most evident with the nuclear stain. Consequently, this analysis was omitted from this thesis. Despite this, the preliminary analysis did not reveal increases in intensity correlating with the observed differences in relative transcript abundance. Potential explanations include suboptimal practices during data acquisition using the confocal microscope or underlying biological factors. Literature suggests that, while protein abundance is generally expected to scale with mRNA abundance, various additional factors such as differential translation, protein degradation, contextual confounders, and pervasive protein-level buffering may lead to a lack of correlation between mRNA and protein quantities \cite{Liu2016OnAbundance, Buccitelli2020MRNAsControl}. These considerations could elucidate the disparities observed in this study.

Notably, we observed interactions of human IFIT1, IFIT2 (detected by IFIT2(A) antibody), and bovine IFIT3 with RSV inclusion bodies. These interactions varied from colocalisation with the IB edge (human IFIT1 and bovine IFIT2) to forming intra-IB inclusions (human IFIT2 and bovine IFIT3). As a result, we systematically assessed the interactions of all human and bovine IFITs with their respective RSV inclusion bodies. The summary of the frequencies of observed phenotypic interactions between human and bovine IFITs and their respective RSV inclusion bodies in A549, BEAS2B, and MDBK cell lines is presented in Figure \ref{fig:Summary of the Diverse Interactions of IFITs with RSV Inclusion Bodies}. Assuming that IFIT2(A) and IFIT2(B) antibodies detect two distinct, antigenically different IFIT2 populations, we can infer that, for all IFITs, irrespective of the species, the primary interaction with the inclusion bodies is exclusion. This predominant phenotype, suggesting that these structures are inaccessible to IFIT proteins, accounted for more than 50\% of observations and occurred in all inclusion bodies, regardless of their size and maturity. Additionally, we observed inclusion body colocalisation phenotypes for all IFITs, ranging from more subtle, such as diffusion and edge exclusion phenotypes, to more direct interactions, including intra-IB inclusions and colocalisation with the IB edges. We propose that these two pairs of interactions (i.e., diffusion followed by edge exclusion and inclusion followed by IB edge colocalisation) are linked processes that transition between each other as the inclusion bodies increase in size and mature.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{08. Chapter 3/Figs/heatmap_infection.pdf}
    \caption[Summary of the Diverse Interactions of IFITs with RSV Inclusion Bodies.]{\textbf{Summary of the Diverse Interactions of IFITs with RSV Inclusion Bodies.} Heatmap illustrating the frequency of observed interaction phenotypes for each IFIT across different cell lines. The phenotypes are ordered based on their implication of interaction with the IBs, with the least interactive on the top and the most interactive on the bottom. The frequencies are presented on a log10 scale to enhance the visualisation of less common phenotypes.}
    \label{fig:Summary of the Diverse Interactions of IFITs with RSV Inclusion Bodies}
\end{figure}

It is crucial to acknowledge that our data lacks super-resolution images, and we did not acquire z-stack measurements for all of the analysed images, which would enhance confidence in the accuracy of the collected data. There is a considerable possibility that some of the observations may be technical artefacts. Additionally, despite using confocal microscopy, the observed phenotype could be influenced by the section plane within certain observations. For instance, if the true interaction for a specific IFIT protein is colocalisation with the edge of the IB structure, and the viewing plane intersects the top of the IB, the phenotype might be incorrectly assigned as an inclusion phenotype. However, despite these considerations, the data obtained is notably consistent within a particular IFIT across different cell lines, particularly between the A549 and BEAS2B cell lines, instilling confidence in the validity of our results. Moreover, occasional observation of IB edge staining within the DAPI visualised images, though sporadic, was noted. As mentioned in Section \ref{sec:Confocal Microscopy}, we employed laser lines sequentially, a common practice recommended by published protocols and guides \cite{Jonkman2020Tutorial:Microscopy}. Given that the excitation spectra of DAPI (405 nm) and RSV N (488 nm) stains are not as separated as the IFIT fluorophore excitation spectra (568 nm), there remains a possibility of the 488 nm laser exciting DAPI-stained structures. However, we do not contend that the results indicate DAPI's ability to stain and detect RSV inclusion bodies independently.

Deciphering the implications of IFIT localisation with RSV IBs, their functionality, and their contribution to the previously reported decreased RSV fitness remains challenging. IFIT1 and IFIT2 have the potential to interact with eukaryotic translation initiation factors \cite{Diamond2014IFIT1:Translation, Guo2000CharacterizationVirus}, potentially inhibiting their assembly, although these IFIT proteins were not observed to concentrate within IBAGs, where such factors compartmentalise \cite{Jobe2023ViralCondensates}. During the diffusion and inclusion phenotypes, these IFIT proteins might have access to these structures, enabling them to exert their action. IFIT2, known to interact with double-stranded RNA \cite{Yang2012CrystalMechanisms}, raises the possibility that observed phenotypes, especially colocalisation to the IB edge, are mediated by IFIT2 interaction with nucleocapsid RNA imperfections. This claim, however, needs to be evaluated further. IFIT1 and IFIT5, reported to interact with Cap1 and 5$^{\prime}$PPP 5$^{\prime}$mRNA modifications \cite{Kumar2014InhibitionMRNAs}, could sense newly synthesised viral mRNA, hindering efficient replication. This process occurs within IB structures but not in IBAGs \cite{Rincheval2017FunctionalVirus}, suggesting IFIT1 and IFIT5 would have access to these. Regarding IFIT3, its function during RSV infection is less clear. One possibility, applicable to IFIT2 as well, is interaction with the MAVS protein. MAVS is reported to be sequestered inside IBs during RSV infection \cite{Lifland2012HumanMAVS}, and both IFIT2 and IFIT3 are known interactors of MAVS protein \cite{Chen2017InhibitionApoptosis, Diamond2013TheProteins, Liu2011IFN-InducedTBK1}, suggesting their localisation in these structures could result from passive shuttling with MAVS during its translocation into IBs. Alternatively, only one IFIT protein, likely IFIT2, could be the primary negative influencer of RSV fitness, while other proteins (e.g., IFIT1 and IFIT3) stabilise and improve their function. Considering that IFIT1, IFIT2, and IFIT3 can form heteromeric complexes \textit{in vitro} \cite{Mears2018BetterResponse}, this possibility remains open.

In conclusion, these initial findings are intriguing yet incomplete, requiring additional analysis to elucidate the nature of IFIT protein interactions with components of the RSV life cycle. The differential localisation results obtained by IFIT2(A) and IFIT2(B) antibodies warrant further investigation and validation to fully comprehend their implications. Two approaches with increasing complexity in model systems could be employed for a comprehensive understanding. Firstly, analysing endogenous IFIT localisation with respect to a simplified system of pseudo-inclusion bodies (pIB) could be undertaken. These pIBs, formed by the ectopic coexpression of RSV N and P proteins, retain functions observed in IBs \cite{Rincheval2017FunctionalVirus, Galloux2020MinimalVitro, Jobe2020RespiratorySignaling}, allowing the establishment and modelling of IFIT-IB interactions regardless of the IB maturation state. Finally, ectopic expression of tagged IFIT constructs could be utilised in both pIB induction and RSV infection. By specifically probing for the tag, concrete evidence of the true localisation and interaction of IFIT proteins, especially IFIT2, in the context of RSV infection could be obtained.

%Words in text: 8895
%Words in headers: 70
%Words outside text (captions, etc.): 2946
%total = 11,911