\chapter{Subcellular Localisation of Nascent IFIT Proteins in the Context of RSV Inclusion Bodies} \label{ch:Subcellular Localisation of Nascent IFIT Proteins in the Context of RSV Inclusion Bodies}
\section{Background and Aims} \label{sec:Background and Aims-Chapter3}
As elucidated in the preceding chapters, human \textit{IFIT} genes exhibit induction in response to both human and bovine RSV infections, suggesting their potential contribution to an anti-RSV role \textit{in vivo}. In contrast, the responsiveness of bovine \textit{IFITs} to human and bovine RSV infections appears limited, possibly influenced by baseline expression levels of these genes. Notably, Drori et al. reported that ectopically expressed human IFIT1, IFIT2, and IFIT3 hindered the growth of hRSV, though the precise mechanism of action remains elusive \cite{Drori2020InfluenzaProteins}. IFIT proteins, known for their diverse antiviral functions, engage in various mechanisms, differing among the IFIT proteins. These mechanisms include direct interactions of IFIT1 and IFIT5 with 5'-RNA modifications \cite{Abbas2013StructuralProteins, Diamond2014IFIT1:Translation}, IFIT2's pro-apoptotic action mediated through direct interactions with mitochondrial membranes \cite{Chen2017InhibitionApoptosis}, and IFIT3-mediated potentiation of RIG-I signaling via scaffold formation between MAVS and TBK1 on mitochondrial membranes \cite{Liu2011IFN-InducedTBK1}. Additionally, IFIT1, IFIT2, and IFIT3 form heterodimer and heterotrimer protein complexes, enhancing IFIT stability and potentially influencing their functions \cite{Mears2018BetterResponse}. Examining the subcellular localization of IFIT proteins in infected cells may serve as a proxy for understanding their mechanism of RSV inhibition. We propose that the sites of IFIT/RSV interactions are the RSV inclusion bodies, which form early in the infection cycle (6 HPI). These membraneless viral organelles, created by liquid-liquid phase separation, contain essential components of the RSV viral polymerase complex and serve as functional sites for RSV RNA transcription and replication \cite{Rincheval2017FunctionalVirus, Weber1995NonstructuralSerum, Fricke2013P38Assembly, Jobe2021BovineResponses}. IBs house the RNA genome and polyadenylated RNA, with the latter spatially segregated into substructures called IBAGs (IB-associated granules), while the genome is distributed throughout the IB structure but excluded from IBAGs \cite{Rincheval2017FunctionalVirus}. RSV IBs increase in size and complexity as they mature, with only larger, more mature IBs containing IBAGs \cite{Rincheval2017FunctionalVirus, Jobe2021BovineResponses}. IBAGs further contain the M2/1 protein, along with the eukaryotic translation initiation factors eIF4G, eIF4F, eIF3A, eIF4H, eIF4A, eIF4A1, and eIF4B \cite{Rincheval2017FunctionalVirus, Jobe2023ViralCondensates}. Given the reported interactions of IFIT1 and IFIT2 with eIF3E and eIF3C, hindering 43S-mRNA complex formation \cite{Diamond2014IFIT1:Translation, Guo2000CharacterizationVirus}, further indicating that the mechanism of action of these proteins against RSV could be mediated via IB interaction. Moreover, the maturation process of RSV IBs, characterized by increased size and complexity, with larger, mature IBs containing IBAGs, further underscores the potential significance of IFIT/IB interactions \cite{Rincheval2017FunctionalVirus, Jobe2021BovineResponses}. Our objective was to delineate the specific functions of IFIT proteins contributing to the reduction in RSV fitness. To achieve this, we initiated an assessment and comparison of the subcellular localization of human and bovine IFIT proteins under various conditions, including mock infection, treatment with human/bovine IFN\(\alpha\), and human/bovine RSV infection. Subsequently, we undertook a thorough analysis of human and bovine IFIT proteins, focusing on their interactions with the corresponding RSV IBs across multiple cell lines. These observations were made using confocal microscopy, allowing for a comprehensive exploration of the dynamic interactions between IFIT proteins and RSV IBs.

\section{Results} \label{sec:Results-Chapter3}
\input{08. Chapter 3/Subchapters/01. Localisation intorduction.tex}
\input{08. Chapter 3/Subchapters/02. Infection.tex}

\section{Conclusions} \label{sec:Conclusions-Chapter3}
To gain a proxy on the mechanism of action of IFIT proteins on the reduction of RSV fitness, we have assayed their subcellular localisation changes after RSV infection in human and bovine cell lines. Both human and bovine IFIT1 exhibited cytoplasmic localization with nuclear exclusion in both mock and RSV infected cells. This is in line to what was reported about murine Ifit1 and human IFIT1 mock localisation in the literature \cite{Pichlmair2011IFIT1RNA, Terenzi2008Interferon-inducibleE1, Thul2017AProteome}. Furthermore, the concentration of both human and bovine IFIT1 was increased in the non-infected cell within the population of RSV infected cells, suggesting the role of IFIT1 might be prophylactic, preventing cells with the increased IFIT1 expression from being infected. This complements the data observed in Chapter \ref{ch:Assessment of Transcriptional Induction of Human IFITs in the Context of RSV}. 

IFIT2 protein localisation was assayed using two disticnt polyclonal antibodies that we refer to as IFIT2A and IFIT2B. These two antibodies showed a differential staining pattern, which was more pronounced in the bovine samples. In the mock-infected human samples IFIT2A antibody showed vesicular staining pattern, which was excluded from the nucleus, while in bovine mock-infected cells it showed diffuesd cytoplasmic, nuclearly excluded pattern with perinuclear IFIT2 condensates, which resemble the Golgi aparatus. To date, there are no publications reporting IFIT2 being associated with the Golgi apparatuses, however the human protein atlas reported IFIT2 to dysplay vesicular cytoplasmic, nuclear excluded localisation \cite{Thul2017AProteome}. On the other hand, IFIT2B consistently showed both human and bovine mock-cells to present grannular cytoplasmic stain with nuclear exclusion. This was also associated with detecting IFIT2 concentrating with the mitotic spindle in dividing cells. This observation is concisted to what was prevoisly reported in murine cell lines \cite{Saha2006IdentificationProtein}. This suggest that with regards to the basal staining pattern of IFIT2 proteins, both antibodies replicate something that is reported in the literature for IFIT2. With regards to the changes observed between mock and RSV-infected samples IFIT2B antibody did not detect a change of distribution in both human and bovine samples. IFIT2A antibody on the other hand uncovered decreased IFIT2 cytoplasmic localisation and insted observed IFIT2 to concentrate to to the viscinity of RSV IBs, which was consistent in both human and bovine samples. The latter translocation phenotype will be discussed more in detail below.

We observed a differential staining patter between human and bovine IFIT3 in both mock and RSV-infected cells. We observed human IFIT3 to have diffused, nuclear exluded cytoplamic distribution, however during infection we observed IFIT3 to also localise in the nucleus of both infected and non infected cells. On the other hand, the bovine IFIT3 was detected to have vesicular cytoplasmic stain, accomplanied by IFIT3 concentrartions in nucleoplasm in mock-treated cells. In RSV-infected cells bovine  IFIT3 was only detectable as inclusions inside the RSV IBs and was no longer present in the nucli of the cells. Previously published data on IFIT3 localisation reports exogenously expressed IFIT3 to be have cytoplasmic localisation with nuclear exclusion in HEK293T cells \cite{Huang2008Interferon-inducedCells, Liu2011IFN-InducedTBK1}, with human protein atlas reporting granular cytoplasmic localisation, partial nuclear staining in A431 and HeLa cells \cite{Thul2017AProteome}. This means that our results regarding the basal IFIT3 localisations are consistent with previously reported results, although it is intreeging to see the discrepancy between human and bovine samples.

We have also observed a differential staining patterns between human and bovine IFIT5, both with regards to the staining during mock and RSV-infected cells. Human IFIT5 was diffused through the cytoplams while being excluded from the nucleus in both conditions. Bovine IFIT5 likewise showed diffused cytoplasmic staining, but additionally it was localised in cytoplasmic vesicles, nucleoplasm, and formed inclusions within the nucleoli. This staining pattern was consistent between mock and RSV infected samples, with the exeption of diminished nucleoplasmic staining in infected cells. IFIT5 was observed at the ruffled membranes, colocalising with actin-rich protrusions in WI-38 VA-13 and Huh7 cell lines \cite{Katibah2013TRNAIFIT5}, or being localised within structures resembling the Golgi apparatuses in A431 and SK-MEL-30 cell lines \cite{Thul2017AProteome}, however our findings contradict these results. It is possible that the localisaiton of IFIT5, along with oiter proteins is cell type and species dependant. In human samples infected with RSV we observed non-infected cells to present with higher levels of IFIT5, a phenotype which was not recreated in bovine samples. The same induction phenotype was observed in both human and bovine cells for IFIT1.

Although we were hoping to see subcellular localisation of the IFIT proteins that would coinside with their proposed function i.e. observing IFIT2 or IFIT3 being concentrated on mitochondria, which are the proposed sites of their action, however the initial analysis did not provide these. Intreegingly, although in Chapter \ref{ch:Assessment of Transcriptional Induction of Bovine IFITs in the Context of RSV} we observed minimal induction changes of bovine \textit{IFITs} to RSV infection we were able to observe these proteins within the cells, with some of these observations, especially with regards to the interactions with the RSV IBs (which will be discussed in more detail below) being consistent between human and bovine cells. This indicate that we are indeed detecting bovine IFITs and not of target proteins. This also suggest that the low induction levels observed previously are due to the increased basal \textit{IFIT} expression in bovine cell compared to human cells. We have although observed human IFIT1, IFIT2 detected by IFIT2A antibody, and bovine IFIT3 interacting with RSV inclusion bodies. These interactions ranged from colocalsation with the IB edge (human IFIT1 and bovine IFIT2), to forming intra-IB inclusion (human IFIT2 and bovine IFIT3). Due to this, we set to systematically assess the interactions of all human and bovine IFITs with their respective RSV inclusion bodies.

The summary of the frequencies of the observed phenotipic interactions between human and bovine IFITs and the respective RSV inclusion bodies in A549, BEAS2B, and MDBK cell lines is shown in Figure \ref{fig:Summary of the Diverse Interactions of IFITs with RSV Inclusion Bodies}. If we assume that IFIT2A and IFIT2B antibodies are detecting two disctnt, antigenically disticnt IFIT2 populations, we can conclude that for all IFITs, regardless of the species, the main interaction with the IBs is exclusion. This phenotype, which suggest that these structures are inaccessible to the IFIT proteins often accounted for more than 50\% of the pbservation and occured in all IBs, indicriminate of their size and thus maturity. We have although also observed IB colocaliastion phenotypes for all of the IFITs, with these colocalisation phenotypes ranging from more minute such as the diffusion and edge exclusion phenotypes, to more direct, which included intra-IB inclusions and colocalisaiton with the IB edges. We believe that these two pairs of interactions (i.e. diffusion followed by edge exclusion, and inclusion followed by IB edge colocalisaiton) are linked processes which transition betwenn each other as the inclusion bodies increase in size and mature. 

It is to be noted that our data is not based on super-resolution images, nor we obtained z-stack measurements for each of the images that were analysed, which would allow us to have higher confidence in the faithfulness of the data collected. There is a high posibility that a proportion of the observations are technical artefacts. It is also possible that within some of the obsevations the section plane could affect the observed phenotype e.g. if we assume that the true interaction for a particular IFIT protein is colocalisation with the edge of the IB structure and our viewing plane intersects the top of the IB, we can incorectly assign the phenotype to be an inclusion phenotype. Regardless, the data we obtained is fairly consistent within a specific IFIT between the different cell lines, especially between the A549 and BEAS2B cell lines, which give us confidence in the validity of data obtained. We have also often observed IB edge stain within the DAPI visualised images, something that we have not discussed previously. This sporadisticaly appear in our analyses in Chapter \ref{ch:Subcellular Localisation of Nascent IFIT Proteins in the Context of RSV Inclusion Bodies} and Chapter \ref{ch:Investigating the Nature of IFIT and IB Interactions}. As mentioned in Section \ref{sec:Confocal Microscopy} we have operated the laser lines in a sequential order, a common practice which recomended by published protocols and guides \cite{Jonkman2020Tutorial:Microscopy}. As the DAPI (405 nm) and RSV N (488 nm) stains excitation spectra is not as separated as the IFIT flurophore excitation spectra (568 nm) there is still a possibility of 488 nm laser exciting DAPI stained structures. We do not believe that the results indicate DAPI being able to stain and detect RSV inclusion bodies bu itself.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{08. Chapter 3/Figs/heatmap_infection.pdf}
    \caption[Summary of the Diverse Interactions of IFITs with RSV Inclusion Bodies.]{\textbf{Summary of the Diverse Interactions of IFITs with RSV Inclusion Bodies.} Heatmap illustrating the frequency of observed interaction phenotypes for each IFIT across different cell lines. The frequencies are presented on a log10 scale to enhance visualization of less common phenotypes.}
    \label{fig:Summary of the Diverse Interactions of IFITs with RSV Inclusion Bodies}
\end{figure}

asdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdf

summary
HYPOTHETISE ABOUT INTERACTION BETWEEN IFITS AND IB maturity

ABOUT POSSIBLE MODE OF ACTION

ABOUT POSSIBLE INTRACTION OF THE DIFFERENT IFITS


next steps
SAY ABOUT THE NEXT STEPS WITH pIB AS A SIMPLIFIED SYSTEM THAT COULD ANDWER THE DIVERSITY QUESTION


ALL OF THIS IN ONE PARAGRAPH

