\subsection{IFIT Subcellular Localisation during Interferon Induction and RSV Infection} \label{subsec:IFIT Subcellular Localisation During Interferon Induction and RSV Infection}
A549, MDBK, and later BEAS2B cell lines were seeded on a 13 mm diameter glass coverslip in a 24-well plate at an initial concentration of 100,000 cells per well, as described in Section \ref{sec:Cell Culture}. 24 hours post-seeding, these cells were either mock-infected, treated with either human or bovine IFN$\upalpha$ at concentrations of 1,000 international units per mL for the former or 5 ng/mL for the latter (these concentrations are equivalent to each other), or infected with either human or bovine wild-type RSV at an MOI of 1. After 24 hours, the samples were fixed with paraformaldehyde and prepared as described in Section \ref{sec:Confocal Microscopy} for subsequent confocal microscopy analysis. The samples were stained with DAPI, allowing nuclear detection (consistently shown in yellow), antibodies against the RSV N protein (consistently shown in cyan), and the IFIT proteins (consistently shown in magenta). We initially assessed a panel of anti-IFIT antibodies, generously provided by the Viral Gene Expression Group at The Pirbright Institute. This panel included two antibodies for each of the human IFIT proteins, routinely employed for Western blotting by the group. In our confocal microscopy experiments, only one antibody per IFIT1, IFIT3, and IFIT5 yielded satisfactory results, while both antibodies against IFIT2 demonstrated efficacy. Consequently, we proceeded with single specific antibodies for IFIT1, IFIT3, and IFIT5, and utilised both IFIT2 antibodies, which we will refer to as IFIT2(A) and IFIT2(B) for simplicity.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{08. Chapter 3/Figs/01. Localisation introduction/07. a549 merges.pdf}
    \caption[Alterations in the Subcellular Localisation of Human IFITs in A549 Cells Exposed to hIFN$\upalpha$ or hRSV.]{\textbf{Alterations in the Subcellular Localisation of Human IFITs in A549 Cells Exposed to hIFN$\upalpha$ or hRSV.} A549 cells underwent mock treatment, were treated with 1000 IU/mL of hIFN$\upalpha$ for 24 hours, or were infected with hRSV MOI 1 for 24 hours. Subsequently, cells were fixed and stained, using DAPI for nuclei detection (depicted in yellow), anti-RSV N antibody (shown in cyan), or antibodies targeting IFIT proteins (illustrated in magenta). Notably, two distinct antibodies against IFIT2 were utilised, designated as IFIT2(A) and IFIT2(B). Insets featuring magnified selections were generated from infected images, facilitating a clearer presentation of the underlying subcellular localisations.}
    \label{fig:Alterations in the Subcellular Localisation of Human IFITs in A549 Cells Exposed to hIFNa or hRSV}
\end{figure}

Figure \ref{fig:Alterations in the Subcellular Localisation of Human IFITs in A549 Cells Exposed to hIFNa or hRSV} illustrates the subcellular localisation of human IFIT proteins in mock-, hIFN$\upalpha$-, or human RSV-treated samples observed in the A549 cell line. Human IFIT1 is primarily cytoplasmic and excluded from the nucleus in a basal state. Remarkably, hIFN$\upalpha$ stimulation does not alter this localisation pattern, and there is no discernible impact on the abundance of IFIT1, as evidenced by signal intensity. Throughout infection, IFIT1 maintains its cytoplasmic localisation, remaining excluded from the nucleus, potentially colocalising with N on the outer surface of the IB structure. The staining intensity exhibits an increase, particularly in uninfected cells, providing support for the hypothesis proposed in Chapter \ref{ch:Assessment of Transcriptional Induction of Human IFITs in the Context of RSV}â€”specifically, that RSV infection prophylactically induces IFIT expression in non-infected cells in an IFN-dependent manner. The IFIT2(A) antibody reveals hIFIT2 to be located in a cytoplasmic, diffused, but nuclearly excluded vesicular pattern in both basal and IFN$\upalpha$-induced states. The intensity between these two conditions appears equal. During hRSV infection, the overall intensity seems to decrease, and the localisation phenotype changes into a phenotype with fewer vesicles and inclusions inside the RSV IB structures. Conversely, the IFIT2(B) antibody detects IFIT2 to be granular and cytoplasmic, also depicting it as excluded from the nucleus. Similar to the IFIT2(A) antibody, the intensity and localisation phenotype between mock and IFN$\upalpha$ treated cells are equal. During hRSV infection, this antibody detects IFIT2 to be excluded from both the nucleus and the inclusion body, while retaining a vesicular cytoplasmic stain. IFIT3 appears to be cytoplasmic with nuclear exclusion under basal conditions. After IFN$\upalpha$ treatment, an increase in staining intensity and signs of IFIT3 nuclear translocation can be observed. During hRSV infection, IFIT3 seems to be evenly diffused throughout the whole cell, including the RSV inclusion body and the nucleus. The intensity of staining during infection appears similar between IFN$\upalpha$ treated and hRSV-infected cells. IFIT5 is also primarily cytoplasmically located under basal conditions while being excluded from the nucleus. During IFN$\upalpha$ treatment, the phenotype and intensity of the staining remain the same. During hRSV infection, an increase in IFIT5 levels is observed in uninfected cells. Regarding the staining pattern, it remains cytoplasmic and excluded from both the nucleus and RSV IB.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{08. Chapter 3/Figs/01. Localisation introduction/09. mdbk-merges-test.pdf}
    \caption[Modulations in the Subcellular Localisation of Bovine IFITs in MDBK Cells Exposed to bIFN$\upalpha$ or bRSV.]{\textbf{Modulations in the Subcellular Localisation of Bovine IFITs in MDBK Cells Exposed to bIFN$\upalpha$ or bRSV.} MDBK cells underwent either mock treatment, exposure to 5 ng/mL of bIFN$\upalpha$ for 24 hours, or infection with bRSV MOI 1 for 24 hours. Subsequently, cells were fixed and stained using DAPI for nuclei detection (depicted in yellow), anti-RSV N antibody (shown in cyan), or antibodies targeting IFIT proteins (illustrated in magenta). Notably, two distinct antibodies against IFIT2 were employed, referred to as IFIT2(A) and IFIT2(B). Insets featuring magnified selections were derived from infected images, facilitating a clearer presentation of the underlying subcellular localisations.}
    \label{fig:Modulations in the Subcellular Localisation of Bovine IFITs in MDBK Cells Exposed to bIFNa or bRSV}
\end{figure}

Figure \ref{fig:Modulations in the Subcellular Localisation of Bovine IFITs in MDBK Cells Exposed to bIFNa or bRSV} presents the subcellular localisation of bovine IFIT proteins in MDBK cell line under mock, bIFN$\upalpha$, or bovine RSV treatment conditions. Under basal conditions, bIFIT1 is primarily located in the cytoplasm with nuclear exclusion, accompanied by vesicles dispersed throughout cells, reminiscent of cytoplasmic bodies as classified by the Human Protein Atlas \cite{Thul2017AProteome}. Upon bIFN$\upalpha$ stimulation, the subcellular localisation remains unchanged, while the overall staining intensity increases. In samples infected with bRSV, cytoplasmic staining is observed with nuclear and IB exclusion, without apparent vesicles. Surrounding non-infected cells display an increased IFIT1 signal, while infected cells exhibit staining intensity lower than what was observed with interferon-stimulated cells. The IFIT2(A) antibody detects bIFIT2 predominantly in the cytoplasm, concentrated near the nucleus in a staining pattern resembling the Golgi apparatus, as classified by the Human Protein Atlas \cite{Thul2017AProteome} (Figure \ref{fig:Additional Confirmation of IFIT Localisation with Organelles}). This signal is also excluded from the nucleus. In interferon-treated cells, the overall pattern appears to be the same, with the exception of decreased staining intensity and decreased size of nuclearly proximal condensations. During bRSV infection, results are identical to what was observed in the A549 cell line, with the cytoplasmic signal greatly reduced. Instead, IFIT2 either colocalises with RSV N at the boundary of the IB or concentrates inside the inclusion bodies. IFIT2(B) shows bIFIT2 to be cytoplasmic with vesicles present throughout the cytoplasm and nucleus under basal conditions. It also appears to concentrate within the mitotic spindle, as classified by the Human Protein Atlas \cite{Thul2017AProteome} (Figure \ref{fig:Additional Confirmation of IFIT Localisation with Organelles}). Unfortunately, cells treated with bIFN$\upalpha$ and stained with IFIT2(B) antibody are lacking. During bRSV infection, this antibody shows bIFIT2 to have the same localisation and intensity as observed in mock cells. It also demonstrates IFIT2 to be excluded from the IB structure. bIFIT3 appears to be localised in vesicles with cytoplasmic and nuclear localisation (Figure \ref{fig:Additional Confirmation of IFIT Localisation with Organelles}) in both mock and bIFN$\upalpha$ treated cells. The intensity signal appears slightly stronger in interferon-treated samples, although the difference does not seem substantial. During bRSV infection, IFIT3 subcellular localisation changes drastically. It is barely detectable in the cytoplasm and nucleus, except for strong intra-IB inclusions. There also appear to be vesicles inside the inclusion that resemble the IBAGs. bIFIT5 shows cytoplasmic localisation with vesicles diffused through the cytoplasm, weak nuclear staining, and inclusions within nucleoli, as classified by the Human Protein Atlas \cite{Thul2017AProteome} (Figure \ref{fig:Additional Confirmation of IFIT Localisation with Organelles}). In samples treated with bIFN$\upalpha$, although the pattern of subcellular localisation remains the same, the intensity increases, especially in the nucleus. Finally, in cells infected with bRSV, nuclear exclusion occurs with inclusion within the nucleoli, and cytoplasmic staining with no clear colocalisation or exclusion with RSV IBs. The observed cytoplasmic vesicles are only evident in non-infected cells. The intensity of the staining resembles that of mock-treated cells.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{08. Chapter 3/Figs/01. Localisation introduction/11. organelles.pdf}
    \caption[Additional Confirmation of Bovine IFIT Localisation with Organelles.]{\textbf{Additional Confirmation of Bovine IFIT Localisation with Organelles.} Zoomed in images from the mock panels of Figure \ref{fig:Modulations in the Subcellular Localisation of Bovine IFITs in MDBK Cells Exposed to bIFNa or bRSV} which depicts the staining of IFIT2(A), IFIT2(B), IFIT3, and IFIT5 antibodies. Nuclear staining is shown in yellow (DAPI panel), while the IFIT staining is shown in magenta. The last column contains images taken from Thul \textit{et al.}, 2017 \cite{Thul2017AProteome}, which depicts stainings of known markers of Golgi apparatus, mitotic spindle, nucleoplasm, and nucleoli staining. These merge images show nuclei in blue, the actin cytoskeleton in red, and the organelle markers in green.}
    \label{fig:Additional Confirmation of IFIT Localisation with Organelles}
\end{figure}

We can observe a wide variety of subcellular localisations and interactions with the RSV inclusion bodies when these results are considered collectively. Notably, a consistent observation across cell lines and IFITs is the infrequent occurrence of changes in localisations due to IFN$\upalpha$ stimulation. Instead, it predominantly results in an increased concentration or no discernible difference compared to mock-treated samples. A noteworthy exception is the increased nuclear localisation observed in human IFIT3 and the altered pattern in bovine IFIT2, as detected by the IFIT2(A) antibody, showcasing a decreased overall signal and a modified Golgi apparatus-like staining pattern. Human and bovine IFIT1 exhibit cytoplasmic localisation with nuclear exclusion. During infection, both seem to be more expressed in uninfected cells. Bovine IFIT1, in particular, displays the presence of cytoplasmic vesicles and differential interaction with IBs, being excluded, while human IFIT1 appears to colocalise with the IB boundary. The IFIT2 antibodies provide distinct outcomes concerning the detected interaction of IFIT2 with IBs, while maintaining consistent results across human and bovine cell lines. IFIT2(A) antibody demonstrates the interaction of human and bovine IFIT2 with RSV IBs, with human IFIT2 forming intra-IB inclusions and bovine IFIT2 colocalising with the IB boundary. In contrast, IFIT2(B) consistently detects the exclusion of IFIT2 from IBs in both cell lines. Discrepancies are observed in basal localisation, both between antibodies and cell lines. In human cells, IFIT2(A) is vesicular, while IFIT2(B) exhibits a more granular cytoplasmic stain. In bovine cells, IFIT2(A) shows cytoplasmic staining with IFIT2 concentrations proximal to the nucleus, and IFIT2(B) displays a vesicular stain without these structures. Further investigation is necessary to elucidate the differential staining patterns observed with the two polyclonal antibodies. We hypothesise that these antibodies may be targeting two distinct IFIT2 epitopes. Possibilities include detection of its monomeric or dimeric state, identification of IFIT2 in complex with its interaction partners, or recognition of an unfamiliar antigenic state that requires further elucidation. Differential IFIT3 signals are noted between human and bovine cells, both in basal localisation and interaction with RSV IBs. Human IFIT3 is cytoplasmic and nuclearly excluded under basal conditions, with nuclear translocation after stimulation with either IFN$\upalpha$ or hRSV. In contrast, bovine IFIT3 is vesicular with nuclear staining under basal conditions, but nuclear staining disappears in bRSV-infected cells. Regarding RSV IB interaction, human IFIT3 is diffused evenly throughout the cytoplasm, nucleus, and IB structures, while bovine IFIT3 forms intra-IB inclusions with signs of IBAGs. Small differences are also observed between human and bovine IFIT5 staining. Both are basally cytoplasmic, with increased intensity after IFN$\upalpha$ stimulation. Bovine IFIT5 is additionally observed in cytoplasmic vesicles, nuclei, and intra-nucleolar inclusions. During infection, both human and bovine IFIT5 exhibit cytoplasmic and nuclear exclusion, with bovine IFIT5 retaining the intra-nucleolar inclusion. Regarding the interaction with RSV IBs, human IFIT5 was evidently excluded, whereas we faced challenges in definitively determining the bovine IFIT5 interaction phenotype with bovine RSV IBs.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{08. Chapter 3/Figs/01. Localisation introduction/01. IB-zooms.pdf}
    \caption[Inclusion Bodies Within RSV-Infected Cells: Zoom Sequence.]{\textbf{Inclusion Bodies Within RSV-Infected Cells: Zoom Sequence.} A representative image of RSV-infected cells detected using confocal microscopy. Cellular nuclei were stained with DAPI and are depicted in yellow; RSV inclusion bodies, stained for RSV P, are represented in cyan; and the cytoplasm, stained for IFIT5, is illustrated in magenta. The figure showcases a zoom sequence starting from a population of cells, progressing to a single syncytial view, and ultimately focusing on an individual inclusion body.}
    \label{fig:Inclusion Bodies Within RSV Infected Cells: Zoom Sequence}
\end{figure}

In light of the potentially interesting findings concerning the colocalisation of IFITs with RSV IBs, we expanded our investigation with hRSV infections in the BEAS2B cell line. A comprehensive analysis was conducted, systematically examining 1727 IB sizes and their underlying interaction phenotypes with various IFIT proteins. The phenotypic categorisation was based solely on IFIT staining in regions where the presence of IBs was confirmed by viral nucleoprotein staining. Figure \ref{fig:Inclusion Bodies Within RSV Infected Cells: Zoom Sequence} illustrates a representative region of interest, showcasing its location within the cell and the broader cellular population. Based on our observations, we classified the phenotypes as follows: \textbf{Diffusion} phenotype was assigned when the IFIT signal spread equally across the area of interest; \textbf{Exclusion} phenotype was designated by an obvious, partial, or complete decrease in signal within the IB boundary; \textbf{Edge Exclusion} phenotype was assigned when the IFIT signal spread uniformly across the area of interest, except for exclusion from the IB boundary; \textbf{Inclusion} phenotype was attributed to a clear increase in the IFIT signal within the IB boundary; and \textbf{Colocalisation} phenotype was assigned when the IFIT signal spread uniformly across the area of interest, except for an increased signal in the IB boundary. Additionally, these main phenotypes were sometimes complemented by either their interaction or the presence of spots within the IB boundary. Notably, a common occurrence was colocalisation accompanied by exclusion, indicating a marked increase on the IB boundary compared to the surrounding signal, alongside a noticeable decrease within the region of interest. For the former, these spots are hypothesised to represent the inclusion body-associated granules (IBAGs). Representative examples of these phenotypes can be seen in Figure \ref{fig:Examples of Interaction Phenotypes}. The frequency of occurrence of these phenotypes was calculated, and an arbitrary cutoff of 5\% was established to distinguish phenotypes that, due to their low frequency, were considered not relevant in the context of RSV infection.

\begin{figure}
    \centering
    \includegraphics[width=0.85\linewidth]{08. Chapter 3/Figs/01. Localisation introduction/10. examples of phenotypes.pdf}
    \caption[Examples of Interaction Phenotypes.]{\textbf{Examples of Interaction Phenotypes.} Interaction phenotypes are shown in decending order of increasing implied interactivity. Cellular nuclei were stained with DAPI and are depicted in yellow; RSV inclusion bodies, stained for RSV N, are represented in cyan; and the IFIT staining, which is illustrated in magenta originates from figures used later in this thesis. Arrows indicate the boundaries of RSV inclusion bodies to ease the phenotype determination.}
    \label{fig:Examples of Interaction Phenotypes}
\end{figure}

We systematically observed and annotated a total of 1727 inclusion bodies across different cell lines, categorising their IFIT interaction phenotypes and measuring their radii (Âµm) and areas (\(\mbox{Âµm}^2\)) based on the observed 2D projections. Specifically, we analysed 1008 hRSV IBs in the A549 cell line, 99 hRSV IBs in the BEAS2B cell line, and 620 bRSV IBs in the MDBK cell line. Figure \ref{fig:Size Characterization of Inclusion Bodies Observed Across Different Cell Lines} illustrates the relationship between IB area and radius, both as an aggregate of all observed IBs and IBs detected per cell line. The data, in both the aggregate and individual cell line view, broadly follows a logarithmic curve, indicating the predominant circular shape of the majority of IBs. However, a deviation from this pattern is noticeable in each cell line, particularly in larger IBs, where the area increases disproportionately with the radius, suggesting a more elongated ellipsoid shape. Most IBs, irrespective of their origin, conform to areas below 10 \(\mbox{Âµm}^2\). Regarding the most common radii, both BEAS2B and MDBK IBs tend to be below 1 Âµm, while A549 IBs exhibit a range of the most common radii sizes between 0.5 and 1.5 Âµm. A more detailed view of the distribution of measured areas per cell line can be observed in Figure \ref{fig:The Distributions of IB Areas Observed Per Cell Line}. Here, all three cell lines encompass IBs ranging from sub 0.5 \(\mbox{Âµm}^2\) to supra 30 \(\mbox{Âµm}^2\), with median sizes of 5 \(\mbox{Âµm}^2\), 3 \(\mbox{Âµm}^2\), and 2 \(\mbox{Âµm}^2\) for A549, BEAS2B, and MDBK, respectively. MDBK has the highest number of sub 1 \(\mbox{Âµm}^2\) IBs, while A549 exhibits the most supra 10 \(\mbox{Âµm}^2\) IBs.

\begin{figure}
    \begin{subfigure}{0.495\textwidth}
        \caption{}
        \includegraphics[width=\textwidth]{08. Chapter 3/Figs/01. Localisation introduction/02. heatmap_all.pdf} 
    \end{subfigure}
    \hfill
    \begin{subfigure}{0.495\textwidth}
        \caption{}
        \includegraphics[width=\textwidth]{08. Chapter 3/Figs/01. Localisation introduction/03. heatmap_a549.pdf}
    \end{subfigure}

    \medskip
    \begin{subfigure}{0.495\textwidth}
        \caption{}
        \includegraphics[width=\textwidth]{08. Chapter 3/Figs/01. Localisation introduction/04. heatmap_beas2b.pdf} 
    \end{subfigure}
    \hfill
    \begin{subfigure}{0.495\textwidth}
        \caption{}
        \includegraphics[width=\textwidth]{08. Chapter 3/Figs/01. Localisation introduction/05. heatmap_mdbk.pdf} 
    \end{subfigure}
    \caption[Size Characterisation of Inclusion Bodies Observed Across Different Cell Lines.]{\textbf{Size Characterisation of Inclusion Bodies Observed Across Different Cell Lines.} This figure elucidates the relationship between the measured area (\(\mbox{Âµm}^2\)) and radius (Âµm) of individual RSV inclusion bodies observed in this study. The figure includes distinct population distributions alongside the plots: (a) an aggregate of 1727 IB observations across all cell lines, (b) 1008 observations from the A549 cell line, (c) 99 observations from the BEAS2B cell line, and (d) 620 observations from the MDBK cell line. Contour plots are incorporated to highlight the density of individual IBs within the plots.}
    \label{fig:Size Characterization of Inclusion Bodies Observed Across Different Cell Lines}
\end{figure}

From the published literature, it is established that an increase in IB area correlates with its maturation state and internal complexity. Notably, in the HEp-2 cell line infected with hRSV, it has been reported that IBs containing IBAGs are significantly larger than those without (6.4 \(\mbox{Âµm}^2\) versus 2.3 \(\mbox{Âµm}^2\)) \cite{Rincheval2017FunctionalVirus}. Additionally, in the MDBK cell line, during the early infection timepoint (6 hours post-infection), only small IBs are present, with a mean area value of 1 \(\mbox{Âµm}^2\), and these do not contain IBAGs. At 16 hours post-infection, IBs without IBAGs retain the mean area value, while larger IBs with IBAGs (mean area of 10 \(\mbox{Âµm}^2\)) are also observed \cite{Jobe2021BovineResponses}. Importantly, at 24 HPI, a critical timepoint for the analysis in this thesis, small IBs without IBAGs maintain a median size of 1 \(\mbox{Âµm}^2\), while larger IBs containing IBAGs cluster into two sizes, with mean areas of 2.5 \(\mbox{Âµm}^2\) and 17 \(\mbox{Âµm}^2\). Based on this information, we can assess the observed IFIT-IB interaction phenotypes per mature or immature IBs.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{08. Chapter 3/Figs/01. Localisation introduction/06. box-infection.pdf}
    \caption[The Distributions of IB Areas Observed per Cell Line.]{\textbf{The Distributions of IB Areas Observed per Cell Line.} The distribution of RSV inclusion body areas (\(\mbox{Âµm}^2\)), detected in this study are shown. A total of 1008 observations were made in the A549 cell line, 99 observations in the BEAS2B cell line, and 620 observations in the MDBK cell line.}
    \label{fig:The Distributions of IB Areas Observed Per Cell Line}
\end{figure}