\chapter{Assessment of Transcriptional Induction of Bovine IFITs in the Context of RSV} \label{ch:Assessment of Transcriptional Induction of Bovine IFITs in the Context of RSV}
\section{Background and Aims} \label{sec:Background and Aims Chapter2}
From published literature, it is evident that induced human IFIT expression negatively impacts RSV replication. The results obtained in Chapter \ref{ch:Assessment of Transcriptional Induction of Human IFITs in the Context of RSV} confirm differential expression of human IFITs in response to both hRSV and bRSV infections. However, the investigation into bovine IFIT induction and its potential influence on bovine RSV has not been undertaken to date.

Our hypothesis posited that bovine IFITs would be induced by both human and bovine RSV, similar to the observation made for human IFITs in the previous chapter. Our objective was to systematically test this hypothesis, employing methodologies akin to those used in Chapter \ref{ch:Assessment of Transcriptional Induction of Human IFITs in the Context of RSV}. Initially, our efforts were directed towards preparing essential technological materials and information, such as qPCR primers for bIFIT detection. Presently, no commercially available primers exist for this purpose. Additionally, we aimed to assess bRSV growth in various model bovine cell lines employed in our study. Subsequently, we sought to confirm the capability of our model cell lines to induce IFITs following the treatment with known activators of the innate immune system, such as interferon alpha and LPS. Following this, our strategy involved evaluating IFIT induction during both human and bovine RSV infections, utilizing a range of viral concentrations and various end assay time points.

\section{Results} \label{sec:Results Chapter2}
\subsection{Technology Establishment} \label{subsec:Technology Establishment}
\subsubsection{Primer Design and Validation for Bovine \textit{IFIT} Quantification} \label{Primer Design and Validation for Bovine IFIT Quantification}
Due to the initial lack of commercially available primers for the detection of bovine \textit{IFIT} transcripts at the outset of the project, we devised a panel comprising three primer sets (PS) for each bovine \textit{IFIT} gene. Detailed information about this process is outlined in Section \ref{subsec:Primer Design and Assay Setup}. In a nutshell, we inputted the coding sequences into the PrimerQuest software (Integrated DNA Technologies) to identify the most suitable oligonucleotides. To evaluate the amplification efficiencies of each primer set, we employed \textit{IFIT} DNA clones from a bovine ISG library as standards (accessible through a collaboration with CVR Glasgow). The outcomes are depicted in Figure \ref{fig:Validation of custom-made bIFIT qPCR primers}. The graph demonstrates that primer sets 1 exhibited the most favourable amplification efficiencies. All primer sets, except for \textit{bIFIT3}, yielded nearly impeccable amplification efficiencies of around 100\%. Consequently, they were chosen for subsequent experiments. While \textit{bIFIT3} primer sets demonstrated similar outcomes in terms of standard curve slopes and amplification efficiencies, PS1 consistently outperformed the others in repeated testing rounds (data not presented). As a result, it was singled out for further experimentation.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/01. Technologies/02. primer validation.pdf}
    \caption[Validation of Custom-Made \textit{bIFIT} qPCR Primers.]{\textbf{Validation of Custom-Made \textit{bIFIT} qPCR Primers.} The custom-designed primers were evaluated by creating a serial dilution of bovine \textit{IFIT}-containing plasmids, provided by the CVR Glasgow. The resulting standard curves are shown here. Primer set (PS) 1 (a), 2 (b), and 3 (c) are depicted for bovine \textit{IFIT1} (1.), \textit{IFIT2} (2.), \textit{IFIT3} (3.), and \textit{IFIT5} (4.), along with their calculated amplification efficiencies.}
    \label{fig:Validation of custom-made bIFIT qPCR primers}
\end{figure}

The PSs behaviour was monitored throughout the project, as fresh standard curves were created per experiment. Figure \ref{fig:The Performance of Custom Made Primer Sets Over Time} shows that the average data is consistent with what was observed in initial testing (Figure \ref{fig:Validation of custom-made bIFIT qPCR primers}), however, there were per experiment deviations in slope angles for each of the selected primer pairs. The underlying amplification efficiencies stayed consistent, as is highlighted by the averaged efficiencies displayed. The initial \textit{bIFIT3} PSs differential amplification slopes compared to the other \textit{bIFIT} PSs, as well as the variable nature of PSs performance throughout the project prohibits the usage of \(\Delta\)\(\Delta\)Ct methodologies for transcript quantification as the increase in cycle threshold would not be proportional to the decrease of transcript abundance between the \textit{bIFITs}, and thus a different methodology had to be adopted. This is described in detail in Section \ref{subsec:Data Processing}. In short, the copy numbers were deducted from standard curves and factorised by the relative abundance of bovine \textit{GAPDH}. This ensured the slope-independent establishment of relative expression values, mirroring and complementing data from \(\Delta\)\(\Delta\)Ct methodologies.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/01. Technologies/03. standard curves behaviour.pdf}
    \caption[The Performance of Custom-Made Primer-Sets Over Time.]{\textbf{The Performance of Custom-Made Primer-Sets Over Time.} During the experiments with custom-made \textit{bIFIT} qPCR primers, standard curves had to be always constructed. Here, the underlying average amplification efficiencies and standard curves, along with the individual data, from all the experiments and coloured by the experiment are displayed.}
    \label{fig:The Performance of Custom Made Primer Sets Over Time}
\end{figure}

\subsubsection{Growth Curves of Bovine RSV in Bovine Cell Lines} \label{Growth Curves of Bovine RSV in Bovine Cell Lines}
Although my colleagues in the Viral Glycoproteins Group from the Pirbright Institute routinely perform hRSV infections of the MDBK cell line, the data in the BT cell line is currently lacking. Therefore we set up bRSV growth curves in both cell lines side by side to investigate which time points would be relevant for further infection experiments. Crude-extracted bRSV, isolated as described in Section \ref{subsec:Virus Propagation and Production} and quantified as described in Section \ref{subsec:Virus Quantification by TCID50 Assay} was used for the establishment of growth curves as described in Section \ref{subsec:Viral Growth Curves}. In brief, the MDBK and the BT cell line were seeded in 96-well plates and infected with WT bRSV at MOI 0.1. The supernatant and cellular fractions were collected at time intervals of 24, 48, 72, 96, 120, 144, and 168 hours post-infection, snap frozen in dry ice ethanol mixture, and later tittered as described in Section \ref{subsec:Virus Quantification by TCID50 Assay}. As can be observed in Figure \ref{fig:bRSV growth curves in MDBK and BT cell lines}, there is a differential infection profile between the cell lines. bRSV growth in the MDBK cell line was initially exponential, starting at \(10^{2.5}\) viral titre solely detected in the cytosolic fraction at 24 HPI, it increased to \(10^{4}\) and \(10^{3}\) for the cellular and supernatant fractions respectively at 48 hours. At 72 hours, the cellular viral titre reached \(10^{4.5}\), while the supernatant one further increased to \(10^{3.5}\). Afterwards, the cellular titre remained stable until 120 hours, after which it declined by more than an order of magnitude to \(10^{3}\) at 144 hours and then further increased again to \(10^{3.5}\) at 168 hours. The supernatant titre increased to \(10^{4.5}\) and \(10^{5.5}\) at 120 and 144 hours respectively, and afterwards it decreased to \(10^{4.5}\) again at 168 hours. The peak combined titre occurred at 144 hours but it was robust from 48 hours onwards. bRSV growth in BT cell line displayed virtually no titre from supernatant fraction at 24 and 48 hours. Regardless, the 24-hour time point was the peak value of the combined, as well as cellular titre with cellular titres of \(10^{4}\) at 24 hours and \(10^{3}\) at 48 hours. Cellular titres further declined to \(10^{2}\) at 72 hours, and since then they remained stable other than a small increase to \(10^{2.5}\) at 120 HPI. On the other hand, supernatant titre peaked at 72 hours at \(10^{4}\), slightly decreased to \(10^{3.5}\) at 120 hours and afterwards stayed stable until the end of the experiment. From this experiment, we can conclude that the viral replication kinetics differ substantially between the two cell lines and the data suggests that we can be certain of viral replication at 48 HPI in MDBK cell line and 24 HPI in the BT cell line.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/01. Technologies/01. growth_curves.pdf}
    \caption[bRSV Growth Curves in MDBK and BT Cell Lines.]{\textbf{bRSV Growth Curves in MDBK and BT Cell Lines.} MDBK and BT cell lines were infected with 0.1 MOI wild-type bRSV and the cellular (top panel) and supernatant (bottom panel) fractions were extracted at time intervals of 24, 48, 72, 96, 120, 144, and 168 hours post-infection. These were subsequently quantified by TCID50 methodology. The total viral titre is shown in the middle panel.}
    \label{fig:bRSV growth curves in MDBK and BT cell lines}
\end{figure}

\input{07. Chapter 2/Subchapters/01. bovine induction}

\section{Conclusions} \label{sec:Conclusions Chapter2}
Recap bovine

\cite{Johnston2019ExperimentalResponse.} bovine respiratory syncytial virus in dairy calves: bronchial lymph node - all ifits upregulated
\cite{Johnston2021MessengerCalves} Messenger RNA biomarkers of Bovine Respiratory Syncytial Virus infection in the whole blood of dairy calves - ifit2 and ifit5


\nomenclature[z-PS]{PS}{Primer Set}
\nomenclature[z-cDNA]{cDNA}{Complementary DNA}
\nomenclature[z-MDBK]{MDBK}{Madin-Darby Bovine Kidney}
\nomenclature[z-ISG]{ISG}{Interferon-Stimulated Gene}
\nomenclature[z-TLR4]{TLR4}{Toll-like Receptor 4}
\nomenclature[z-BVDV]{BVDV}{Bovine Viral Diarrhea Virus}
\nomenclature[z-HPI]{HPI}{Hours Post-Infection}
\nomenclature[z-hRSV]{hRSV}{Human RSV}
\nomenclature[z-bRSV]{bRSV}{Bovine RSV}

% Words in text: 3574
% Words in headers: 55