\subsection{Transcriptional Changes of Human and Bovine \textit{IFITs}} \label{Transcriptional Changes of Human and Bovine \textit{IFITs}}
To elucidate the effect of stimulating the cells with either the activators of the innate immune response and human RSV or bovine RSV on human and bovine \textit{IFIT} expression respectively, qPCR analysis was performed as described in Section \ref{Quantitative Real Time/Reverse Transcription PCR}. In short, cells were seeded in 12 well plates and subsequently treated with the stimulants. At the endpoint of the experiments, the RNA was extracted, followed by cDNA synthesis and the transcript quantification by qPCR. All transcripts were normalised to human or bovine \textit{GAPDH} levels using either the \(\Delta\)\(\Delta\)Ct method or using the copy number evaluation first (bovine \textit{IFITs}). All values were subsequently standardised to mock levels. This allowed for the aggregation of data points and comparison of induction values between experiments. Statistical analysis was performed as described in Section \ref{Statistical Analysis}. As the underlying statistical test is dependent on the normality of distribution and equality of variance of the data, this information will be highlighted in the text below.




\subsubsection{Bovine \textit{IFIT} qPCR Primer Validation} \label{Bovine IFIT qPCR Primer Validation}
Due to the absence of commercial primers for detecting bovine \textit{IFIT} transcripts at the beginning of the project, I developed a panel of 3 primer sets (PS) per bovine \textit{IFIT} gene. This is described in detail in Section \ref{Primer Design and Assay Setup}. In brief, coding sequences were fed into the PrimerQuest software (Integrated DNA Technologies) and the most suitable oligonucleotides were selected. I tested the amplification efficiencies of each PS, using \textit{IFIT} DNA clones from a bovine ISG library as standards (available to us through a collaboration with CVR Glasgow). The result can be observed in Figure \ref{Validation of custom-made bIFIT qPCR primers}. We can see that PSs 1 provide the best amplification efficiencies and all PSs, with the exception of \textit{bIFIT3}, yield almost perfect amplification efficiencies of circa 100\%, therefore they were selected for subsequent experiments. \textit{bIFIT3} PSs yield very similar results in terms of the slopes of standard curves and their amplification efficiencies, however, PS1 was the most consistent in the repeated rounds of testing (data not shown) and thus selected for further experiments. 

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/01. Technologies/02. primer validation.pdf}
    \caption[Validation of Custom-Made \textit{bIFIT} qPCR Primers.]{\textbf{Validation of Custom-Made \textit{bIFIT} qPCR Primers.} The custom-designed primers were evaluated by creating a serial dilution of bovine \textit{IFIT}-containing plasmids, provided by the CVR Glasgow. The resulting standard curves are shown here. Primer set (PS) 1 (a), 2 (b), and 3 (c) are depicted for bovine \textit{IFIT1} (1.), \textit{IFIT2} (2.), \textit{IFIT3} (3.), and \textit{IFIT5} (4.), along with their calculated amplification efficiencies.}
    \label{Validation of custom-made bIFIT qPCR primers}
\end{figure}

The PSs behaviour was monitored throughout the project, as fresh standard curves were created per experiment. Figure \ref{The Performance of Custom-Made Primer-Sets Over Time} shows that the average data is consistent with what was observed in initial testing (Figure \ref{Validation of custom-made bIFIT qPCR primers}), however, there were per experiment deviations in slope angles for each of the selected primer pairs. The underlying amplification efficiencies stayed consistent, as is highlighted by the averaged efficiencies displayed. The initial \textit{bIFIT3} PSs differential amplification slopes compared to the other \textit{bIFIT} PSs, as well as the variable nature of PSs performance throughout the project prohibits the usage of \(\Delta\)\(\Delta\)Ct methodologies for transcript quantification as the increase in cycle threshold would not be proportional to the decrease of transcript abundance between the \textit{bIFITs}, and thus a different methodology had to be adopted. This is described in detail in Section \ref{Data Processing}. In short, the copy numbers were deducted from standard curves and factorised by the relative abundance of bovine \textit{GAPDH}. This ensured the slope-independent establishment of relative expression values, mirroring and complementing data from \(\Delta\)\(\Delta\)Ct methodologies.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/01. Technologies/03. standard curves behaviour.pdf}
    \caption[The Performance of Custom-Made Primer-Sets Over Time.]{\textbf{The Performance of Custom-Made Primer-Sets Over Time.} During the experiments with custom-made \textit{bIFIT} qPCR primers, standard curves had to be always constructed. Here, the underlying average amplification efficiencies and standard curves, along with the individual data, from all the experiments and coloured by the experiment are displayed.}
    \label{The Performance of Custom-Made Primer-Sets Over Time}
\end{figure}





















\subsubsection{Responses of to Known Activators of Innate Immune Response} \label{Responses to Known Activators of Innate Immune Response}
In order to establish the expression competency of \textit{IFIT} in the A549 cell line along with elucidating how different innate immune pathways contribute to the overall expression profile, I treated A549 cells with human interferon (IFN) alpha, hIFN gamma, lipopolysaccharide, and transfected poly I:C. 


What inducers were used, why, and what concentrations \newline
Reference papers for concentrations: \newline
Ifn Alpha (\cite{Terenzi2006DistinctISG56}; \cite{Santhakumar2018ChickenViruses})\newline
Ifn gamma ()\newline
Lps (\cite{Mears2019Ifit1Cells}; \cite{Zhang2019GrouperResponse})\newline
Transfected polyIC (\cite{Mears2019Ifit1Cells}; \cite{Palchetti2015TransfectedCells}) \newline

As described in Section \ref{Routes of IFIT Expression Activation}, and depicted in Figure \ref{Pathways Inducing ISG mRNA Production.},  interferon-stimulated genes (ISGs) 


\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{06. Chapter 1/Figs/01. Induction/01. a549_treat_ifna.pdf}
    \caption[qPCR Analysis of A549 \textit{hIFIT} Response to hIFN\(\alpha\).]{\textbf{qPCR Analysis of A549 \textit{hIFIT} to hIFN\(\alpha\).} The relative abundance of (a) \textit{hIFIT1}, (b) \textit{hIFIT2}, (c) \textit{hIFIT3}, and (d) \textit{hIFIT5} genes, extracted from the A549 cell line, with response to human interferon alpha (IFN\(\alpha\)) at a concentration of 1000 IU per mL for a treatment duration of 6 or 24 hours. The shown values are relative to standardised mock values. The red circles signify median values. The black dotted line indicates mock expression, while the red dotted lines indicate biologically significant levels of induction. Numeric values signify the p-values compared to mock.}
    \label{A549 Response to hIFNa}
\end{figure}

In line with bovine IFIT responses investigation, human IFIT responses to human (h) RSV were assessed. The virus was prepared as described in section 7.1. Briefly, infected cells were sonicated, cell debris was separated by centrifugation and virus-containing supernatant was gathered and titred. Human epithelial type 2 (HEp-2) cells were infected with the hRSV-containing supernatant at multiplicities of infection (MOI) of 0.1 and 2. These cells were chosen for the initial preliminary experiments as they are known to grow the virus well and their tissue of origin (laryngeal carcinoma) is relevant to the native site of RSV replication. It is to be noted that this cell line is believed to be contaminated by HeLa cell line and in later experiments should be replaced by more physiological models of human lung epithelial tissue. As a positive control, 1000 U of hIFNα was used. Samples were collected 24 hours post-infection. Cellular RNA was extracted and converted to complementary DNA, as described in section 7.3. Human IFIT transcripts were quantified relative to mock-infected cells. GAPDH-normalised qPCR data can be seen in Figure 8. We observed no significant IFIT induction at the MOI of 0.1, however, infection with hRSV at an MOI of 2 provided significant induction for all targets tested. All human IFITs were induced by approximately 5-fold. With regards to interferon responses, all four targets were transcriptionally upregulated following IFNα treatment. The induction of hIFIT2 was equal to the one induced by viral infection at an MOI of 2, while the rest of hIFITs were induced more greatly. Compared to data described in section 8.3 here the variation was minimal and thus the data is more reliable. We can conclude that both interferon-alpha treatment, as well as RSV infection, transcriptionally upregulate human IFITs in HEp-2 cells.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{06. Chapter 1/Figs/01. Induction/02. a549_treat_ifng.pdf}
    \caption[qPCR Analysis of A549 \textit{hIFIT} Response to hIFN\(\gamma\).]{\textbf{qPCR Analysis of A549 \textit{hIFIT} Response to hIFN\(\gamma\).} The relative abundance of (a) \textit{hIFIT1}, (b) \textit{hIFIT2}, (c) \textit{hIFIT3}, and (d) \textit{hIFIT5} genes, extracted from the A549 cell line, with response to human interferon-gamma (IFN\(\gamma\)) at concentrations of 500, 1000, and 2000 IU per mL for a treatment duration of 6 hours. The shown values are relative to standardised mock values. The red circles signify median values. The black dotted line indicates mock expression, while the red dotted lines indicate biologically significant levels of induction. Numeric values signify the p-values compared to mock.}
    \label{A549 Response to hIFNg}
\end{figure}

some text some text some text


\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{06. Chapter 1/Figs/01. Induction/03. a549_treat_lps.pdf}
    \caption[qPCR Analysis of A549 \textit{hIFIT} Response to LPS.]{\textbf{qPCR Analysis of A549 \textit{hIFIT} Response to LPS.} The relative abundance of (a) \textit{hIFIT1}, (b) \textit{hIFIT2}, (c) \textit{hIFIT3}, and (d) \textit{hIFIT5} genes, extracted from the A549 cell line, with response to lipopolysaccharide (LPS) at concentrations of 5 and 5000 ng/mL for a treatment duration of 6 hours. The shown values are relative to standardised mock values. The red circles signify median values. The black dotted line indicates mock expression, while the red dotted lines indicate biologically significant levels of induction. Numeric values signify the p-values compared to mock.}
    \label{A549 Response to LPS}
\end{figure}

some text some text some text


\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{06. Chapter 1/Figs/01. Induction/04. a549_treat_polyic.pdf}
    \caption[qPCR Analysis of A549 \textit{hIFIT} Response to Transfected poly I:C.]{\textbf{qPCR Analysis of A549 \textit{hIFIT} Response to Transfected poly I:C.} The relative abundance of (a) \textit{hIFIT1}, (b) \textit{hIFIT2}, (c) \textit{hIFIT3}, and (d) \textit{hIFIT5} genes, extracted from the A549 cell line. The cells were transfected with 2 \(\mu\)g of poly I:C for 24 hours. The shown values are relative to standardised mock values. The red circles signify median values. The black dotted line indicates mock expression, while the red dotted lines indicate biologically significant levels of induction. Numeric values signify the p-values compared to mock.}
    \label{A549 Response to poly I:C}
\end{figure}

Describe data: \newline
Asdasdasd \newline
All human IFITs get induced to biologically significant levels    by IFN alpha and gamma (regardless of time and concentration), transfected polyIC and high concentration of LPS. This shows that the cells are responsive to interferon and other stimulants and have the ability to induce IFITs.


\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{06. Chapter 1/Figs/01. Induction/09. beas2b_ifna.pdf}
    \caption[BEAS-2B responses to hIFNa.]{\textbf{BEAS-2B responses to hIFNa.} Text text tet text text.}
    \label{BEAS-2B responses to hIFNa.}
\end{figure}

Old text:
The Madin-Darby bovine kidney (MDBK) cell line, derived from bovine renal epithelium in 1958 (Madin and Darby, 1958), is an established model system used in bovine virology studies. In order to assess the bIFIT induction potential in these cells, we have treated the cells with ranging concentrations of known activators of the innate immune system. These were bacterial lipopolysaccharide (LPS), polyinosinic:polycytidylic acid (poly I:C), bovine interferon-alpha (bIFNα), and bovine interferon-gamma (bIFNγ). After surveying the literature, we chose concentration ranges of 10, 5, 2.5, 1, and 0.5 µg/mL for LPS and poly I:C, and 7.5, 5, 2.5, 1 and 0.5 ng/mL for both bovine interferons (Palchetti et al., 2015; Zhang et al., 2015; Santhakumar et al., 2018; Yang et al., 2018; Lin, Kuo and Huang, 2019; Mears et al., 2019; X. Zhang et al., 2019; Y. Zhang et al., 2019). Unfortunately, due to the COVID-19 outbreak, we were only able to assess bIFIT responses to LPS and the lowest concentration of bIFNα. This data will also allow us to establish the proper positive control for bIFIT induction in these cells. MDBK cells were plated and each plate was treated with different reagent for 6 hours. Mock-treated cells on each plate allowed for assessment of cross-plate basal bIFIT detection differences. Cells were lysed and their RNA was extracted as described in section 7.3. 0.5 µg of RNA was used for final qPCR analysis.
As shown in Figure 6, the basal levels of the different bovine IFITs range quite widely. bIFIT1 was detected around the assay detection limit, with bIFIT2 and bIFIT3 having similar values to each other. We detected c. 5000 copies of bIFIT5 per 0.5 µg of RNA under basal conditions. bIFIT1 and bIFIT2 seem not to be induced by LPS, regardless of the concentration, whereas bIFIT3 and especially bIFIT5 responded in a concentration-dependant manner. For all the bIFITs, 0.5 ng/mL of bIFNα was able to cause induction. This was most significant for bIFIT1 which was induced 100-fold. This data confirms the notion that IFITs expression is minimal in unstimulated cells, however, interferon treatment causes their induction even at low concentration. This suggests that bIFNα could be a suitable candidate for positive control of bIFIT induction, although the rest of its concentration range should be assessed as well.

Describe data: \newline
asdasdas

Bovine Mx1 is included in all experiments with bovine cells as it is an ISG that is induced by range of infections and the activators of innate immune response. It is also a control for bIFN alpha (if it works properly and is not degraded).

Neither bMx1 nor bIFITs are induced by LPS in the concentration range tested. bMx1 was induced by all bIFN alpha at different concentrations and timepoints, but at 5 ng/mL for 24h. This means that either that treatment failed or 24h post adding the bIFN treatment is too late to capture the ISG induction. All targets are induced by bIFN alpha treatment at 5 ng/mL for 3 hours. bIFIT1 also responds to low concentration (0.5 ng/mL) for 6h treatment, but not the other IFITs. No other concentration/time combination induced bIFITs. 

This all suggests that MDBK are responsive to bIFN alpha and capable of bIFIT induction, but the responses are week, especially compared to human cells.

I have a hypothesis that in bovine cells the IFITs are basally expressed to higher levels  (that would explain why we can detect them by IF in mock and infected cell although the qPCR data suggest there is no induction).

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/02. Induction/01. mdbk_treat_bifna.pdf}
    \caption[MDBK responses to bIFNa.]{\textbf{MDBK responses to bIFNa.} interferon!!!!!!!!!!}
    \label{MDBK responses to bIFNa}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/02. Induction/02. mdbk_treat_lps.pdf}
    \caption[MDBK responses to LPS.]{\textbf{MDBK responses to LPS.} lpss!!!!!!!!!!}
    \label{MDBK responses to LPS}
\end{figure}













\subsubsection{Species-Specific Responses of \textit{IFITs} to RSV} \label{Species-Specific Responses of IFITs to RSV}

\myparagraph{Growth curves of bovine RSV in bovine cell lines} \label{Growth curves of bovine RSV in bovine cell lines}
Describe data: \newline
asdasdasd \newline
This was done as a complement to existing data done by other people in Dalan’s group on human cell lines with hRSV growth curves. The novelty is especially growth curves in BT cells. I do not know if I totally knew what I was doing at the time, so I do not know how much to trust this data.  

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/01. Technologies/01. growth_curves.pdf}
    \caption[bRSV growth curves in MDBK and BT cell lines.]{\textbf{bRSV growth curves in MDBK and BT cell lines.} sdfgsdfg sdfg sdfg sdfg sdfg sdfg }
    \label{bRSV growth curves in MDBK and BT cell lines}
\end{figure}


Old text:
In line with bovine IFIT responses investigation, human IFIT responses to human (h) RSV were assessed. The virus was prepared as described in section 7.1. Briefly, infected cells were sonicated, cell debris was separated by centrifugation and virus-containing supernatant was gathered and titred. Human epithelial type 2 (HEp-2) cells were infected with the hRSV-containing supernatant at multiplicities of infection (MOI) of 0.1 and 2. These cells were chosen for the initial preliminary experiments as they are known to grow the virus well and their tissue of origin (laryngeal carcinoma) is relevant to the native site of RSV replication. It is to be noted that this cell line is believed to be contaminated by HeLa cell line and in later experiments should be replaced by more physiological models of human lung epithelial tissue. As a positive control, 1000 U of hIFNα was used. Samples were collected 24 hours post-infection. Cellular RNA was extracted and converted to complementary DNA, as described in section 7.3. Human IFIT transcripts were quantified relative to mock-infected cells. GAPDH-normalised qPCR data can be seen in Figure 8. We observed no significant IFIT induction at the MOI of 0.1, however, infection with hRSV at an MOI of 2 provided significant induction for all targets tested. All human IFITs were induced by approximately 5-fold. With regards to interferon responses, all four targets were transcriptionally upregulated following IFNα treatment. The induction of hIFIT2 was equal to the one induced by viral infection at an MOI of 2, while the rest of hIFITs were induced more greatly. Compared to data described in section 8.3 here the variation was minimal and thus the data is more reliable. We can conclude that both interferon-alpha treatment, as well as RSV infection, transcriptionally upregulate human IFITs in HEp-2 cells.

How were viruses harvested and cells infected \newline
Justify moi and timepoints \newline
Justify uv inactivated virus and how it was done \newline

Describe data: \newline
asdasdasd \newline
UV inactivated RSV does not induce IFIT expression, meaning IFITs do not get activated by TLR4 detection of viral particles on the cell surface. Low MOI infection (0.1 MOI) does not induce IFITs. Infection at MOI 1 and 2 induce all IFITs regardless of the length of infection (24 and 48 HPI). The purification methodology for virus extraction (ultra purification on sucrose cushion vs just clearing the cells + supernatant by centrifugation) does not influence IFIT induction.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{06. Chapter 1/Figs/01. Induction/05. a549_hrsv_timepoints.pdf}
    \caption[Responses of A549 to hRSV.]{\textbf{Responses of A549 to hRSV.} Some filler text afterwards.}
    \label{Responses of A549 to hRSV}
\end{figure}

some text between these two suckers that will say about how the human ifit induction is dependent on replication and interferon.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{06. Chapter 1/Figs/01. Induction/06. a549_hrsv_uv_roxo.pdf}
    \caption[Responses of A549 to modified hRSV.]{\textbf{Responses of A549 to modified hRSV.} Some filler text afterwards.}
    \label{Responses of A549 to modified hRSV.}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{06. Chapter 1/Figs/01. Induction/10. beas2b_hrsv.pdf}
    \caption[BEAS-2B responses to hRSV.]{BEAS-2B responses to hRSV.}
    \label{BEAS-2B responses to hRSV.}
\end{figure}



Describe data: \newline
asdasdasd \newline
Validation in more physiologically relevant cell line. UV inactivated hRSV does not cause induction. Ultracentrifugation purified hRSV causes induction in all IFITs. All IFITs but IFIT5 are induced by IFN alpha, but only after incubation for 24 hours (3h incubation does not cause induction). This shows that the cells are interferon competent. IFIT5 does not get induced by any bRSV infection tried (probably because IFIT5 has either basally higher levels and thus the same end mRNA concentration equates to lower induction or because it is really not induced by bRSV (although the IFITs should have the same promoters)). Low MOI infection (0.001 and 0.01) of dNS1 and dNS2 bRSV induces IFITs 1,2 and 3, while dSH MOI1 and dNS1/2 MOI 0.001 does not. 

The trends seen with A549 are kind of recapitulated.


Old text:
Alongside confirming the induction potential of bIFITs in MDBK cells, we assessed the effect of bovine respiratory syncytial virus (bRSV) on IFIT induction. As bRSV is a known inducer of the interferon response we expect bovine IFIT genes to be upregulated following infection. MDBK cells were infected with purified bRSV at MOI of 1 for 24 and 48 hours. As a positive control human IFNα was used. This was due to the unavailability of a bovine counterpart at that point, however, Gresser and his colleagues (1974), reported that hIFNα does indeed affect bovine cells as well. Cellular RNA was extracted and converted to complementary DNA, as described in section 7.3. Bovine IFIT transcripts were quantified relative to mock-infected cells. qPCR results (Figure 7A) were normalised to bovine GAPDH levels. We observed large variation in the transcriptional response, even in mock-infected cells. bIFITs 2, 3 and 5 mRNA levels did not change in response at any time point post bRSV infection. bIFIT1 was induced at 48 hours post-infection, but the variation is too high to make firm conclusions at this point. However, consistent with Gresser et al’s previous findings, bIFITs were responsive to 1000 units of hIFNα; all genes were highly induced (Figure 7B), although the fold expression increases varied greatly. bIFIT1 increased 10000-fold, followed by bIFIT2 and bIFIT3 with both having c. 700-fold increase. bIFIT5 mRNA expression was induced by c. 25-fold after treatment with hIFNα. This experiment was performed once and will be repeated. As a control for infection, quantification of viral RNA using qPCR will provide us with a clearer picture about the observed changes.

Describe data: \newline
asdasdas

UV inactivated bRSV causes no change for bMx1 and bIFIT2 and seems to downregulate bIFIT1,3,5. Low, mid, and high MOI (0.1, 1, 2) and two different time points (24 and 48 HPI) do not seem to influence the levels of any genes tested other than for bMx1 for wt ultracentrifugation purified bRSV 24 HPI MOI 1. I have one experiment where bRSV MOI 1 24 HPI downregulates all genes tested but it might be a technical error. 
Very low MOI (0.001) wt bRSV along with MOI 1 dSH and very low MOI dNS1, dSN2 and dNS1/2 bRSV do not up or downregulate any genes tested in a biologically significant way.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/02. Induction/03. mdbk_brsv_timepoints.pdf}
    \caption[MDBK responses to bRSV.]{\textbf{MDBK responses to bRSV.} timepoints infection }
    \label{MDBK responses to bRSV}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/02. Induction/04. mdbk_brsv_uv_roxo.pdf}
    \caption[MDBK responses to modified bRSV.]{\textbf{MDBK responses to modified bRSV.} asdf as asdf asdf asdf as asdf asdf assd }
    \label{MDBK responses to modified bRS}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/02. Induction/05. mdbk_brsv_moi1_dsh.pdf}
    \caption[MDBK responses to dSH.]{\textbf{MDBK responses to dSH.} timepoints infection }
    \label{MDBK responses to dSH}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{07. Chapter 2/Figs/02. Induction/06. mdbk_brsv_low_moi.pdf}
    \caption[MDBK responses to low MOI mutant bRSV.]{\textbf{MDBK responses to low MOI mutant bRSV.} asdf as asdf asdf asdf as asdf asdf assd }
    \label{MDBK responses to low MOI mutant bRSV}
\end{figure}



\subsubsection{Validation in More Physiological Cell Lines} \label{Validation in More Physiological Cell Lines}
Why beas2b \newline
How it was cultured + all treatment the same as in a549











