\chapter{Introduction} \label{ch:Introduction}
\section{Interferon-Induced Proteins with Tetratricopeptide Repeats} \label{sec:Interferon-Induced Proteins with Tetratricopeptide Repeats}
The interferon-induced proteins with tetratricopeptide repeats (\textit{IFIT}) gene family consist of interferon-stimulated genes (ISGs) that are activated early in the antiviral response. In unstimulated cells, with the exception of some myeloid cell subsets, they are barely detectable; however, upon viral infection, their translated products become some of the most abundant proteins in the cell \cite{Diamond2013TheProteins}. \textit{IFITs} are conserved in higher animals yet absent in the genomes of plants, insects, and yeast. With regards to their genomic composition, they are composed of two exons, with the first encoding the ATG, two additional nucleotides, and a 5$^{\prime}$ untranslated region (UTR), with the second exon containing the rest of the coding sequence and 3$^{\prime}$ UTR \cite{deVeer1998IFI60/ISG60/IFIT4Genes}. Most IFIT family gene members contain multiple copies of specific sequences in their promoter regions called interferon-stimulated response elements (IRSE), which are targets for downstream effectors of the interferon signalling cascade, enabling the fast activation of IFIT gene transcription \cite{Lou2009IFR-9/STAT2STAT1}. Both human and bovine IFIT gene loci confront to the same genomic organisation, in terms of \textit{IFIT} gene orientation and order, and contain either two or three ISRE regions preceding each of the IFIT genes \cite{Liu2013Lineage-SpecificFamily}.

Most mammalian IFIT genes are categorised into four subgroups named IFIT1, IFIT2, IFIT3, and IFIT5, all with clear orthologous relationships \cite{Sarkar2004NovelGenes}. Primates, along with some other mammalian species, have a duplication of IFIT1 called IFIT1B, which lacks the ISRE in its gene promoter regions, and thus effectively only acts as a pseudogene. Several rodents, including mice and rats, have lost the IFIT1 and IFIT5 genes and have duplications of IFIT1B and IFIT3 \cite{Daugherty2016Evolution-guidedMammals.}, resulting in IFIT1B (typically referred to as IFIT1), IFIT1B-like gene 1, IFIT1B-like 2 gene, IFIT3 and IFIT3B genes. In contrast, avian species have lost most IFIT genes, with only one left resembling human IFIT5 \cite{Liu2013Lineage-SpecificFamily}. With regard to the information relevant to this project, human IFIT proteins phylogenetically cluster with other old and new-world primates, while bovine IFIT proteins cluster with porcine IFITs \cite{Zhou2013InterferonDefense.}. These variations in IFIT gene numbers are most probably the result of differing evolutionary pressures posed by different viruses affecting their respective host species, although it is evident that maintaining IFIT genes in the genome must be beneficial. Altogether, understanding the intricacies of IFIT induction and their interactions during infections, such as with respiratory syncytial virus (RSV), not only expands our knowledge of host-virus dynamics but also unveils potential avenues for antiviral therapeutic development. The impact of unravelling IFIT mechanisms extends to devising strategies for enhancing viral sensitivity to innate immune responses, holding promise for novel antiviral interventions.

\subsection{Routes of \textit{IFIT} Expression Activation} \label{subsec:Routes of IFIT Expression Activation}
\subsubsection{Interferon Signalling} \label{Interferon Signalling}
\textit{IFIT} induction can be achieved by activating several arms of the innate immune system (Figure \ref{fig:Pathways Inducing ISG mRNA Production.}). The strongest inducers are type I interferons, such as Interferon alpha and beta (IFN\(\alpha\)/\(\beta\)). Their signalling cascade is mediated via activation of the IFN\(\alpha\)/\(\beta\) receptor and subsequent downstream Janus kinase (JAK), and Signal transducer and activator of transcription (STAT) signal transduction. As a result, interferon-stimulated gene factor 3 (ISGF3), consisting of phosphorylated STAT1 and STAT2 proteins, bound to interferon regulatory factor (IRF) 9, translocates to the nucleus, binds to the ISRE in the IFIT promoters and induces their transcription \cite{Der1998IdentificationArrays, Mesev2019DecodingInfection, Schoggins2011Interferon-stimulatedFunctions}.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/01. IFIT transcription activation figure.png}
    \caption[Pathways Inducing ISG mRNA Production.]{\textbf{Pathways Inducing ISG mRNA Production.} Three routes of ISGs transcription induction are depicted. Pathway A shows the virus releasing its genome upon its entry and subsequent foreign nucleic acid detection by cytosolic sensors RIG-I and MDA5. These detect single-stranded and double-stranded RNA respectively. Both activate MAVS, which in turn activates TBK1. STING protein facilitates this activation by enhancing MAVS and TBK1 interaction. Pathway B shows PAMP detection by TLR receptors. TLR4 recognises LPS in the extracellular space while TLR3, TLR7, and TLR9 detect foreign nucleic acids in endosomes. TBK1, as well as TLR activation, leads to the translocation of activated IRF3, IRF7, and NF\(\kappa\)B into the nucleus where they promote ISGs transcription activation. Pathway C depicts the type 1 interferon signalling pathway. IFN\(\alpha\)/\(\beta\) receptor activation leads to STAT1 and STAT2 activation. Further addition of IRF9 forms ISGF3 complex, which translocates into the nucleus onto ISRE and promotes ISGs transcription activation. ISG, interferon-stimulated genes; RIG-I, retinoic acid-inducible gene-I; MDA5, melanoma differentiation-associated gene 5; MAVS, mitochondrial antiviral signalling protein; TBK1, TANK-binding kinase 1; STING, stimulator of interferon genes; PAMP, pathogen-associated molecular pattern; TLR, toll-like receptor; IRF, interferon regulatory factor; NF\(\kappa\)B, nuclear factor kappa B; IFN, interferon; STAT, signal transducer and activator of transcription; ssRNA, single-stranded RNA; dsRNA, double-stranded RNA; ISGF, interferon-stimulated gene factor; ISRE, interferon-stimulated response elements. The figure was adapted from Diamond and Farzan, (2013) \cite{Diamond2013TheProteins} and Natalya Odoardi's BioRender template. Created with BioRender.com.}
    \label{fig:Pathways Inducing ISG mRNA Production.}
\end{figure}

\subsubsection{Pattern Recognition Receptors} \label{Pattern Recognition Receptors}
Another arm of \textit{IFIT} activation is mediated through several pattern recognition receptors (PRRs), which can recognise various pathogen-associated molecular patterns (PAMPs). \textit{IFITs} have been reported to be induced by bacterial PAMPs such as lipopolysaccharide (LPS) from \textit{Neisseria meningitidis} via activation of Toll-like receptor (TLR) 4 \cite{Zhou2013InterferonDefense.}. Interestingly, TLR4 has also been observed to be activated by the glycoprotein of respiratory syncytial virus \cite{Funchal2015RespiratoryNeutrophils}. Other TLRs such as TLR3, TLR7, and TLR9 are capable of sensing PAMPs in the form of foreign nucleic acids in the endosomes. TLRs then interact with their adaptor proteins to in turn activate IRF3, IRF7, and nuclear factor kappa B (NF\(\kappa\)B), all of which have the capability to induce \textit{IFIT} family genes \cite{Diamond2013TheProteins}. These pathways are often prevalent in lymphocytes, monocytes, and mast cells; however, cytosolic nucleic acid sensors are functional in a broader subset of cells \cite{Ablasser2011WhereFit}.

\subsubsection{Cytosolic RNA Sensors} \label{Cytosolic Nuclec Acid Sensors}
Cytosolic RNA sensors are proteins within the cell that detect the presence of RNA molecules in the cytoplasm. These sensors play a crucial role in the innate immune response by recognising various forms of foreign or aberrant RNA, often indicating viral infections. When activated, these sensors trigger signalling pathways that lead to the induction of antiviral defences. These include melanoma differentiation-associated gene 5 (MDA5) and retinoic acid-inducible gene I (RIG-I) \cite{Vladimer2014IFITs:Proteins}. Both signal through mitochondrial antiviral signalling protein (MAVS), which in turn activates IRF3, IRF7, and NF\(\kappa\)B as their downstream effectors \cite{Ashley2019Interferon-IndependentCytomegalovirus}. In order to prevent activation by cellular RNA molecules, a precise RNA-recognition mechanism has to be conducted by the sensors. While MDA5 senses long double-stranded RNA (dsRNA) \cite{Brisse2019ComparativeMDA5}, RIG-I is able to sense differences in the 5$^{\prime}$ RNA modifications \cite{Schlee2016DiscriminatingSensing}. During mRNA maturation in higher eukaryotes, 7-methyl guanosine (m7G) is connected by a 5$^{\prime}$-5$^{\prime}$ triphosphate bridge, which is referred to as capping \cite{Devarkar2016StructuralRIG-I, Ramanathan2016MRNAApplications}. Several viruses carry uncapped, 5$^{\prime}$-triphosphorylated (5$^{\prime}$-PPP) or incompletely capped RNA (cap 0), whereas host animals possess cap-1 and cap-2 mRNA moieties \cite{Choi2018ACaps}, which are all depicted in Figure \ref{fig:Overview of 5'RNA Modifications.}.

\begin{figure}
    \centering
    \includegraphics[width=0.75\linewidth]{04. Introduction//Figs/02. 5-RNA Modifications.pdf}
    \caption[Overview of 5$^{\prime}$RNA Modifications.]{\textbf{Overview of 5$^{\prime}$RNA Modifications.} Mature mRNA is displayed. Higher eukaryotes modify their mRNA by the initial addition of guanosine (green) via 5$^{\prime}$-5$^{\prime}$ triphosphate bridge to the 5$^{\prime}$ end. Subsequently, the guanosine is methylated into 7-methylguanosine (magenta) and this modification is referred to as Cap 0 structure. Furthermore, the first 2 bases of RNA can be methylated (blue) and make either Cap 1 or Cap 2 structural modifications. The figure was adapted from Picard-Jean, 2013 \cite{Picard-Jean2013RNAGenomes}.}
    \label{fig:Overview of 5'RNA Modifications.}
\end{figure}

\subsection{Structural Features of IFIT Proteins} \label{subsec:Structural Features of IFIT Proteins}
IFITs are composed of multiple copies of tetratricopeptide repeats (TPR). These are motifs comprised of 3-16 tandem repeats of 34 amino acids, which adopt helix-turn-helix conformations \cite{DAndrea2003TPRHelix}. TPRs are conserved in all species from bacteria through plants to higher animals and are commonly found in scaffolding proteins \cite{Vladimer2014IFITs:Proteins}. IFIT TPRs are comprised of degenerate sequences meaning the conservation of the motif is limited. This allows different IFIT proteins to have a broader profile of protein interactors while still maintaining the overall conformation \cite{Fensterl2015Interferon-InducedPathogenesis}. IFIT proteins are often subjected to post-translational modifications such as ubiquitination or ISGylation (addition of small IFN-induced ubiquitin-like proteins), which may alter IFIT stability and function \cite{Thery2021ProteomicsImmunity, Zhao2005HumanPathways, Radoshevich2015ISG15Infection, Zhu2021ProteomicInterferon}.

\subsubsection{IFITs and Their Interaction with RNA} \label{IFITs and Their Interaction with RNA}
IFIT1 and IFIT5 both form a positively charged channel with an affinity for 5$^{\prime}$ ends of single-stranded RNA (ssRNA) molecules in a sequence non-specific manner. IFIT5 can accommodate 5$^{\prime}$PPP RNA and effectively acts as a sensor for these molecules \cite{Abbas2013StructuralProteins, Pichlmair2011IFIT1RNA}. On the other hand, IFIT1 can accommodate m7G, but certain residues inside its channel prevent efficient binding of cap1 and cap2 moieties \cite{Diamond2014IFIT1:Translation, Mears2018BetterResponse}. Through these interactions, they can effectively discriminate between self and non-self RNA moieties. IFIT2 is also capable of RNA binding, albeit independent of the 5$^{\prime}$-capping state. The C terminals of an IFIT2 homo-dimer create a super-helical structure with a positively-charged nucleotide-binding channel on its inner surface, which was observed to bind to AU-rich RNA molecules \cite{Yang2012CrystalMechanisms, Vladimer2014IFITs:Proteins}. Mechanistic studies identified IFIT2 binding to mRNAs as important to prevent ribosomal pausing and thus increasing the translational efficiency of IFIT2-bound mRNAs, a process which has been observed to be hijacked by influenza A virus \cite{Tran2020InfluenzaMRNAs}.

\subsubsection{Formation of IFIT Protein Complexes} \label{Formation of IFIT Protein Complexes}
As shown in Figure \ref{fig:IFIT Structures and Multimer Formation.}, all IFIT proteins, with the exception of IFIT5, can form homo- and hetero- dimeric and trimeric complexes. IFIT1 homodimerises via its C-terminal domain \cite{Abbas2013StructuralProteins}. The same domain has been shown to interact with the C termini of both IFIT2 and IFIT3, although the IFIT1:IFIT3 complex is more thermodynamically stable \cite{Fleith2018IFIT3RNA, Stawowczyk2011TheApoptosis}. Compared to IFIT1, IFIT5 has its dimerisation motif shielded by its C terminal TPR and thus stays monomeric in solution \cite{Kumar2014InhibitionMRNAs}. In contrast, IFIT2 and IFIT3 are rarely seen as monomeric in solution and rather stay as their respective homodimers or more stable IFIT2:3 heterodimers. This is predicted to be done by swapping the third TRP domain in their N-terminal domains (Figure \ref{fig:IFIT Structures and Multimer Formation.}), which keeps them in a more thermodynamically stable configuration \cite{Yang2012CrystalMechanisms}. IFIT3 interacts with IFIT1 via its C-terminal domain, and this interaction increases the half-life of IFIT1 and its specificity for cap0 RNA. Thus, IFIT3 acts as an enhancer of IFIT1 action \cite{Fleith2018IFIT3RNA, Johnson2018HumanStability}. Recently, it has been shown that IFIT1, IFIT2, and IFIT3 form a heterotrimer, although the precise function of this complex has yet to be elucidated \cite{Fleith2018IFIT3RNA}. 

In summary, TPR motifs allow IFITs to have a multitude of possible interaction partners, including themselves. The formation of IFIT homo- and hetero-oligomers influences their function and half-life, allowing for variable possible outcomes to occur following IFIT protein production based on the level of each of the proteins. Investigating the therapeutic implications of modulating the oligomeric states of IFIT proteins unveils intriguing possibilities for antiviral strategies. Understanding the conditions that favour the formation or disruption of specific oligomeric forms could provide crucial insights into developing targeted interventions against viral infections, where the modulation of IFIT protein interactions could be strategically employed to bolster the host's innate immune response.

\begin{figure}
    \centering
    \includegraphics[width=0.75\linewidth]{04. Introduction//Figs/05. IFIT-complexes.png}
    \caption[IFIT Structures and Multimer Formation.]{\textbf{IFIT Structures and Multimer Formation.} IFIT tertiary and quaternary protein conformations are illustrated. IFIT1 is shown in yellow, IFIT2 in green, IFIT3 in blue, and IFIT5 is highlighted in purple. IFIT1 is able to homodimerise or heterooligomerise with IFIT2 and IFIT3 via its C terminal domain. IFIT5 has the corresponding interaction domain occluded by a helix and thus does not interact with the rest of the IFIT proteins. IFIT2 and IFIT3 are capable of forming homodimers or heterodimers by 'swapping' TPRs in their N terminal regions. By combining the interactions IFIT1:2:3 heterotrimer is formed. The figure was adapted from Mears, 2018 \cite{Mears2018BetterResponse}.}
    \label{fig:IFIT Structures and Multimer Formation.}
\end{figure}

\subsection{Localisation and Functions of IFIT Proteins} \label{subsec:Localisation and Functions of IFIT Proteins}
\subsubsection{Localisation of IFIT Proteins} \label{Localisation of IFIT Proteins}
To date, IFIT protein localisation has been well-studied in human and mouse models. Most publications indicate IFITs to be predominantly cytoplasmic. In more detail, endogenous murine Ifit1 and human IFIT1 have been observed to localise diffusely in the cytoplasm while being excluded from the nucleus in NIH3T3 and P2.1 cell lines, respectively \cite{Pichlmair2011IFIT1RNA, Terenzi2008Interferon-inducibleE1}. The Human Protein Atlas, utilising two distinct anti-IFIT1 antibodies, indicates IFIT1 to have granular cytoplasmic and nuclear localisation in A431 and U2OS cells; and cytoplasmic, nuclear-excluded localisation in HeLa cells \cite{Thul2017AProteome}. Reported localisation of IFIT2 in the literature yields inconsistent results. While endogenous murine Ifit2 was observed to be cytoplasmic while also associated with the mitotic spindle in all phases of the cell cycle in NIH3T3 and B16F10 cell lines \cite{Saha2006IdentificationProtein}, exogenously expressed human IFIT2 was reported to be cytosolic and thus excluded from mitochondria in HeLa cell line \cite{Stawowczyk2011TheApoptosis}. The latter is in conflict with what the Human Protein Atlas reports on an endogenous IFIT2 localisation in the HeLa and U2OS cell lines, wherein both report cytoplasmic vesicular localisation distribution \cite{Thul2017AProteome}. IFIT3 has been indicated to localise cytoplasmically, with the evidence arising from exogenous overexpression experiments in THP-1 and human embryonic kidney (HEK) 293T cells, with the latter further confirming IFIT3 colocalisation with mitochondria \cite{Huang2008Interferon-inducedCells, Liu2011IFN-InducedTBK1}. The Human Protein Atlas confirms these findings by reporting endogenous IFIT3 to show granular cytoplasmic localisation with partial nuclear staining in A431 and HeLa cell lines \cite{Thul2017AProteome}. Lastly, there is a discrepancy with regard to IFIT5 subcellular localisation. While exogenously expressed IFIT5 has been observed to localise at the ruffled membrane at the cell surface and to colocalise with actin-rich protrusions from the apical cell surface in both fibroblast-derived WI-38 VA-13 and hepatocyte-derived Huh7 cell lines \cite{Katibah2013TRNAIFIT5}. However, the Human Protein Atlas reports endogenous IFIT5 displays granular cytoplasmic localisation with partial nuclear localisation and concentrations resembling the Golgi apparatus in A431 and SK-MEL-30 cell lines \cite{Thul2017AProteome}. 

Exploring the functional significance of IFIT protein localisation promises insights into its antiviral potential across diverse cellular environments. This becomes particularly intriguing when considering the prospect of targeting IFIT proteins for more precise interventions against viral infections. Further research is essential to investigate the interactions between IFIT proteins and other cellular components within their designated subcellular compartments. Exploring whether IFIT proteins engage with specific organelles or cellular structures and understanding the functional implications of these interactions is crucial for a comprehensive grasp of their roles. Additionally, a comparative analysis of IFIT protein localisation across different species would facilitate discerning potential species-specific differences. Investigating whether such differences correlate with species-specific antiviral mechanisms is a key avenue for advancing our understanding of IFIT proteins and uncovering potential variations in their antiviral activities across diverse biological contexts.

\subsubsection{Inhibition of Translation and Viral Replication} \label{Inhibition of Translation and Viral Replication}
IFITs restrict viral replication by several mechanisms. IFIT1 and IFIT5 can physically prevent non-self RNA from interacting with eukaryotic initiation factor (eIF) 4F \cite{Kumar2014InhibitionMRNAs}. IFIT1 and IFIT2 block the binding of eIF3 to the eIF2-GTP-Met-tRNA ternary complex by interacting with the eIF3E subunit, whereas human IFIT2, along with mouse IFIT1 and IFIT2, can block the formation of the 43S-mRNA complex by binding to the eIF3C subunit \cite{Diamond2014IFIT1:Translation, Guo2000CharacterizationVirus}. This is a cap-independent mechanism for viral translation inhibition, however, extensive IFIT expression may negatively influence the whole cellular translation processes via this mechanism and hinder normal inflammatory responses. Overexpression of IFIT1 in HEK293T cells inhibited the activation of IRF3 and NF\(\kappa\)B, as well as the transcription of IFN\(\beta\) in response to polyinosinic-polycytidylic acid (poly I:C) \cite{Li2009ISG56Response}. The previously mentioned affinity of IFIT2 for AU-rich RNA can also regulate cellular translation as a lot of transcripts for cytokines or apoptotic factors are rich in adenine and uracil \cite{Palanisamy2012ControlMicroRNAs}. 

\subsubsection{Modulation of Innate Immune Response Signalling} \label{Modulation of Innate Immune Response Signalling}
IFIT proteins also have the potential to influence innate immune responses through direct interaction with signalling cascade proteins. IFIT3 can potentiate RIG-I signalling by acting as a scaffold between MAVS and TANK-binding kinase 1 (TBK1), \cite{Liu2011IFN-InducedTBK1}, whereas IFIT5 enhances this pathway upstream by recruiting RIG-I to MAVS \cite{Zhang2013IFIT5Pathways}. RIG-I is a key sensor in the innate immune system that recognises viral RNA during infections. Upon binding to viral RNA, RIG-I activates downstream signalling pathways, leading to the production of interferons and other antiviral molecules, orchestrating an effective immune response against the invading viruses \cite{Thoresen2021TheSignaling}. IFIT5 has also been reported to help facilitate NF\(\kappa\)B activation via scaffolding its activation kinases \cite{Zhang2013IFIT5Pathways}. On the other hand, studies show that IFIT1 can both negatively and positively influence RIG-I signalling upstream of MAVS. IFIT1 interacts with the stimulator of interferon genes (STING), an enhancer of MAVS and TBK1 interaction and it was proposed that these conflicting actions are caused by its multiple binding sites on STING \cite{Li2009ISG56Response, Reynaud2015IFIT1Interferon}. STING plays a crucial role in the innate immune response to viral infections by sensing cytosolic DNA and initiating downstream signalling pathways. Upon activation, STING triggers the production of type I interferons and pro-inflammatory cytokines, contributing to the antiviral defence mechanisms of the host \cite{Barber2015STING:Cancer}.

\subsubsection{Effects on Cell Cycle Progression and Apoptosis} \label{Effects on Cell Cycle Progression and Apoptosis}
IFIT2 and IFIT3 also have roles in cellular homeostasis. Overexpression of IFIT2 induces the mitochondrial-dependent apoptotic pathway. IFIT2 has been shown to localise on mitochondrial membranes and to directly interact with pro-apoptotic factors which lead to apoptosome formation and subsequent caspase-9 activation \cite{Chen2017InhibitionApoptosis, Diamond2013TheProteins}. Expression of IFIT3 has been observed to alleviate these effects, most probably by the formation of IFIT2:3 dimers \cite{Mears2018BetterResponse, Stawowczyk2011TheApoptosis}. IFIT3 has also been observed to have negative effects on proliferation via indirect degradation of cyclin-dependent kinase inhibitor p27. As a result, cell cycle progression halts at the G1/S transition checkpoint \cite{Xiao2006RIG-GProteins}. An overview of the IFIT mechanisms of action is depicted in Figure \ref{fig:Overview of IFIT Mechanisms of Action}. In brief, IFIT proteins play crucial roles in diverse cellular processes, impacting key pathways with far-reaching consequences. In apoptosome formation, IFIT2:2 involvement carries significant implications for cell survival. The IFIT2:3 complex acts as a regulator, attenuating the pro-apoptotic activity of IFIT2:2. This modulation of apoptotic pathways suggests that IFIT proteins not only contribute to antiviral responses but also exert influence over fundamental cellular decisions related to life and death. Furthermore, IFIT3:3's role in cell cycle arrest adds another layer of complexity, potentially influencing cell proliferation and maintaining cellular homeostasis.

Therefore, IFITs not only act as complementary RNA sensors to RIG-I but also play crucial roles as mediators of a plethora of other processes. These range from the regulation of transcription, to the activation and inhibition of innate immune signalling, as well as the regulation of the cell life cycle. It is again possible that preference for these actions is influenced by the levels of particular IFIT proteins and their interplay. Taking into consideration the above-described modes of IFIT expression activation, we can speculate that different stimuli such as viruses or signalling molecules at different concentrations will result in quite diverging actions of IFIT proteins on the phenotype of the cell.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/06. IFIT Mechanism of Action.png}
    \caption[Overview of IFIT Mechanisms of Action.]{\textbf{Overview of IFIT Mechanisms of Action.} IFITs have been described to have multiple actions in cells. Pathway A shows their involvement in innate immune signalling modulation. IFIT5 potentiate RIG-I activation of MAVS by scaffolding the two proteins together. IFIT3:3 further scaffolds MAVS to its effector, TBK1, which induces IRF3, IRF7 and NF\(\kappa\)B nuclear translocation when activated. STING protein also potentiates MAVS and TBK1 interaction. IFIT1 has been observed to inhibit STING interaction with MAVS while potentiating its interaction with TBK1. IFIT5 can also indirectly activate NF\(\kappa\)B. Pathway B shows IFIT2:2 involvement with apoptosome formation. IFIT3 titration promotes IFIT2:3 complex formation, which in turn reduces the pro-apoptotic activity of IFIT2:2. IFIT3:3 is involved in cell cycle arrest, as depicted in pathway C. Pathway D shows IFIT inhibition of viral replication and cellular translation. IFIT1 and IFIT5 can directly bind to cap0 and 5’PPP mRNA respectively and prevent its translation. IFIT1 and IFIT2 can both prevent 43S complex formation, while IFIT2 can also inhibit 48S complex formation. IFIT, Interferon-Induced Protein with Tetratricopeptide Repeats; RIG-I, retinoic acid-inducible gene I; MAVS, mitochondrial antiviral signalling protein; TBK1, TANK-binding kinase 1; STING, stimulator of interferon genes; IRF, interferon regulatory factor; NF\(\kappa\)B, nuclear factor kappa B; eIF, eukaryotic elongation factor; ssRNA, single-stranded RNA; tRNA, transfer RNA. The figure was adapted from Mears, 2018 \cite{Mears2018BetterResponse} and Diamond, 2013 \cite{Diamond2013TheProteins}. Created using BioRender.com.}
    \label{fig:Overview of IFIT Mechanisms of Action}
\end{figure}

\subsection{IFIT Responses to Viral Infections} \label{subsec:IFIT Responses to Viral Infections}
Human IFIT effects on single-stranded RNA viruses have been studied extensively within the past decade. Rabbani \textit{et al.} reported that IFIT1 and IFIT3 restrict human parainfluenza virus 3 (PIV3, negative-sense ssRNA virus), while ectopic expression of IFIT2 and IFIT5 did not affect the viral fitness \cite{Rabbani2016Identification3}. \textit{In vitro} \textit{IFIT1} knockdown experiments using short hairpin RNA showed marked enhancement of protein expression of human PIV2, PIV5, and mumps virus, all negative-sense ssRNA viruses \cite{Andrejeva2013ISG56/IFIT1Synthesis, Young2016HumanFamily}. Hantaviruses, a family of negative-sense ssRNA viruses such as Prospect Hill virus (PHV), and Tula virus (TULV), strongly induce IFIT3 expression during the course of their infection \cite{Matthys2011TheInduction}. Most importantly within the non-segmented negative-sense ssRNA viruses, IFIT1, IFIT2, and IFIT3 have been reported to be antiviral against respiratory syncytial virus (RSV). Drori and colleagues either overexpressed or silenced these three IFITs and assessed RSV fitness by quantifying the viral mRNA produced. They reported decreased viral fitness following overexpression, while silencing of IFITs increased the viral mRNA production. Another negative-sense ssRNA virus, which bears a segmented genome, influenza A virus, has been shown to upregulate IFIT1, IFIT2, and IFIT3 in primary macrophages during infection \cite{Lietzen2011QuantitativeMacrophages}. Recent studies also show the ability of IFIT1 and IFIT2 to negatively influence the viral fitness of influenza, as the knock-down of these IFITs resulted in increased viral RNA and protein production, while the opposite was true if IFIT1 and IFIT2 were overexpressed \cite{Zhu2023TheSynthesis}. Human IFIT1 has been observed to be differentially expressed during hepatitis C virus (HCV) infection. HCV was described to suppress IFIT1 upregulation and subsequent experiments showed an inverse relationship between artificial levels of IFIT1 and HCV's ability to infect host cells \cite{Raychoudhuri2011ISG56Replication, Ishida2019HepaticInfection}. Currently, there is very limited information about the effect of bovine IFIT expression on the viral fitness of bovine viruses.


\section{Respiratory Syncytial Virus} \label{sec:Respiratory Syncytial Virus}
Human and bovine Respiratory Syncytial Viruses (hRSV and bRSV) are enveloped, negative-sense, single-stranded RNA viruses belonging to the Orthopneumovirus genus \cite{Afonso2016Taxonomy2016}. These viruses are recognised for inducing acute respiratory disease, which is confined to their respective hosts. Despite this host specificity, shared genetic and antigenic characteristics are observed among these closely related viruses \cite{Buchholz2000ChimericVaccine}. hRSV primarily infects humans, while bRSV infects cattle. They are known to be the leading causes of lower respiratory tract illness in their respective hosts \cite{Nair2013GlobalAnalysis, Sacco2014RespiratoryCattle}. Although these viruses can infect individuals of all ages, severe respiratory illness, including bronchiolitis and pneumonia, is more common in infants, the elderly, and immunocompromised individuals \cite{Falsey2005RespiratoryAdults, Coultas2019RespiratoryAge}. On a global scale, hRSV affects nearly all children by the age of five, resulting in approximately 35 million cases and 3.5 million hospital admissions annually in the United States of America \cite{Shi2017GlobalStudy}. Regrettably, around 60,000 hospitalised children under the age of five succumb to the infection, with a notably higher mortality rate among infants under six months, especially those at high risk due to factors such as premature birth and pre-existing chronic lung and heart conditions \cite{Shi2017GlobalStudy, Jha2016RespiratoryVirus, Coultas2019RespiratoryAge}. Bovine RSV infection also poses a significant threat to the global cattle farming industry, resulting in substantial economic losses \cite{Brodersen2010BovineVirus, Valarcher2007BovineInfection}.

Despite extensive research and development efforts spanning several decades, the availability of approved vaccines against hRSV remains limited. Historical challenges include the failure of the formalin-inactivated RSV vaccine, which led to enhanced natural infection in some vaccinated children \cite{Fulginiti1969RespiratoryVaccine., Kim1969RespiratoryVaccine.}. Additional hurdles involve the heterogeneity of RSV subtypes and the incomplete immune responses generated against the virus. However, recent advances in the field have introduced a range of vaccine strategies, including live-attenuated wild-type virus, vector-based, viral protein sub-unit, mRNA-based, and DNA-based candidates. Presently, 24 vaccines are in various stages of clinical development, including two licensed vaccines: Arexvy (GSK) and Abrysvo (Pfizer), both of which are F protein-based. These vaccines are used as immunising agents to prevent lower respiratory tract disease in older adults, with Abrysvo additionally indicated for passive immunisation of infants through maternal administration during pregnancy \cite{Topalidou2023RespiratoryVaccines}. In contrast, there are several vaccines available for bRSV, with the first one dating back to the 1970s. However, these vaccines offer only mild protection. Therefore, it is crucial to intensify efforts in the development of more effective vaccines for both bRSV \cite{Ellis2017HowCattle} and hRSV, particularly those designed and approved for use in human infants.

\subsection{Genomic and Virion Composition} \label{subsec:Genomic and Virion Composition}
The size of the RNA genome of human and bovine RSV is reported to be 15,223 nucleotides for hRSV A2 subtype and 15,140 nucleotides for the bRSV A51908 subtype, as reported by GenBank (GenBank identifiers KT992094 and NC038272 respectively). The RSV genome contains ten genes in the order 3$^{\prime}$-\textit{NS1}-\textit{NS2}-\textit{N}-\textit{P}-\textit{M}-\textit{SH}-\textit{G}-\textit{F}-\textit{M2}-\textit{L}-5$^{\prime}$ (Figure \ref{fig:The Composition of RSV Virion and Genome}, panel b), which are transcribed into 10 monocistronic, Cap 1-capped, methylated, and polyadenylated mRNAs \cite{Collins2013RespiratoryDisease, Bohmwald2016HumanPathology}. These encode for 11 proteins as \textit{M2} gene is composed of two overlapping open reading frames (ORFs), which are accessed by coupled translation, where the ribosomes translating the first ORF (\textit{M2-1}) move a short distance upstream after termination and reinitiate translation from a second (\textit{M2-2}) overlapping ORF \cite{Gould2007CouplediPneumovirinae/i}.

RSV virions were observed to be pleomorphic, confronting to spherical (circa 100 nm radius), asymmetric, or filamentous (circa 5 \(\mu\)m in length) structures, as observed by electron microscopy, with a conserved structural organisation \cite{Kiss2014StructuralComplex}. Virions produced in cell culture are predominantly observed to be filamentous in several cell types leading to a hypothesis that spherical and asymmetric particles are the breakdown and aggregation by-products of filamentous particles \cite{Ke2018TheTomography, Conley2022HelicalVirus}. Within an \textit{in vitro} context, a substantial portion (95\%) of the progeny virus remains attached to the cell surface, appearing as particles that have seemingly undergone incomplete budding. Thus the standard procedure for cultivating virus stocks involves subjecting infected cells to processes such as sonication, freeze-thawing, or vortexing, aimed at detaching the virus. It is important to note, however, that these methods lead to a reduction in infectivity and an increase in the risk of cellular contamination \cite{Collins2013RespiratoryDisease}.

A representative filamentous RSV virion, based on the current literature, is depicted in Figure \ref{fig:The Composition of RSV Virion and Genome}, panel a. It is composed of the structural proteins, which include F, G, SH, M, M2-1, N, P and L proteins. M2-1 (transcription processivity factor) and matrix (M) proteins form subsequent lattice structures which support the viral membrane and dictate the filamentous shape of the virion \cite{Conley2022HelicalVirus}. The attachment (G) and fusion (F) glycoproteins are embedded in the viral membrane and helically ordered in pairs, as dictated by the M lattice. The small hydrophobic (SH) protein is also embedded in the membrane \cite{Ke2018TheTomography, Conley2022HelicalVirus}. Inside the virion resides nucleocapsid which is composed of RSV genomic RNA-bound within the groves nucleoprotein (N) multimeric protein complexes. These have been observed to be found in two forms, either as a polymer of helical nucleocapsid, which spans several RSV genomes in its structure (Figure \ref{fig:Nucleocapsid Polymer and Its Interaction with Viral RNA}) \cite{Tawar2009CrystalVirus, Conley2022HelicalVirus} or double-decameric N-RNA rings \cite{Gonnin2023StructuralNucleocapsids, Gonnin2022ImportanceVirus, Conley2022HelicalVirus}. These further interact with the large polymerase subunit (L) and the phosphoprotein polymerase cofactor (P) to form the ribonucleoprotein complex (RNP) \cite{Gonnin2023StructuralNucleocapsids}. The P stabilises the RNP structure with the virion by interacting with M2-1 protein \cite{Mason2003InteractionActivity}.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/07. RSV-composition.png}
    \caption[The Composition of RSV Virion and Genome.]{\textbf{The Composition of RSV Virion and Genome.} a) The most biologically correct diagram of the filamentous RSV virion is depicted. M2-1 (transcription processivity factor) and matrix (M) proteins form subsequent lattice structures which support the viral membrane and dictate the filamentous shape of the virion. The attachment (G) and fusion (F) glycoproteins are embedded in the viral membrane and helically ordered in pairs, as dictated by the M lattice. The small hydrophobic (SH) protein is also embedded in the membrane. Inside the virion, a helical nucleocapsid is present, composed of viral RNA genome-bound nucleoprotein (N) polymer, with its precise protein structure shown in grey. The large polymerase subunit (L) and the phosphoprotein polymerase cofactor (P) are also associated with N. b) The human RSV genome from the A2 strain is shown to scale. it contains 10 genes encoding 11 proteins, with the M2 gene encoding the M2-1 and M2-2 proteins. ssRNA; single-stranded RNA. Figure taken from Battles, 2019 \cite{Battles2019RespiratoryIt} and the structural biology basis for this figure is from Conley, 2022 \cite{Conley2022HelicalVirus}.}
    \label{fig:The Composition of RSV Virion and Genome}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/08. N-structure.jpeg}
    \caption[Nucleocapsid Polymer and Its Interaction with Viral RNA.]{\textbf{Nucleocapsid Polymer and Its Interaction with Viral RNA.} Panel A shows a negative-stained electron micrograph of recombinant RSV nucleocapsid-like helices. The scale bar indicates 50 nm. Panel B presents the 26 \r{A}-resolution 3D reconstruction of the observed helices formed by the nucleocapsid polymer. A single helix comprises 9.8 N subunits per turn. Panel C provides a more detailed view of the structure, with the encapsulated RNA shown in cyan and one N protein labelled based on its domains, with the N-terminal domain coloured in yellow and the C-terminal domain coloured in red. Figure adapted from Tawar, 2009 \cite{Tawar2009CrystalVirus}.}
    \label{fig:Nucleocapsid Polymer and Its Interaction with Viral RNA}
\end{figure}

\subsection{The Function of RSV Proteins} \label{subsec:The Function of RSV Proteins}
F and G are the main antigenic proteins of RSV \cite{Collins2011ProgressYears, Battles2019RespiratoryIt}. The F protein is involved in viral entry and assembly. It is generated as an inactive F0 precursor in the cytoplasm, after which it is activated by furin-like proteases in the Golgi apparatus \cite{Collins1984NucleotideVirus.}. The active form of the F protein in the viral particles is a trimer in a prefusion conformation \cite{Ternette2007ImmunogenicityVirus}. The fusion protein has been reported to interact with several host surface receptors, such as toll-like receptor 4 \cite{Marr2012RoleReplication}, intercellular adhesion molecule-1 (ICAM-1) \cite{Behera2001BlockingInfection}, or nucleolin, with the latter believed to be the main RSV receptor and determinant of viral tropism \cite{Tayyari2011IdentificationVirus}. The G protein is not vital for successful infection and is present in two forms, either as a membrane-bound form or a secreted isoform. Membrane-bound G enhances F protein binding to cellular surface receptors by facilitating membrane attachment via the interaction with heparin and annexin 2 \cite{Collins2013RespiratoryDisease, Krusat1997Heparin-dependentCells, Malhotra2003IsolationCells}, while the secreted form is responsible for immune system evasion by antibody binding saturation \cite{Bukreyev2008TheLeukocytes}. 

The transmembrane SH protein, also situated at the virion surface, forms pentameric pore-like structures that exhibit cation-selective channel-like activity \cite{Carter2010DirectPermeability, Gan2012TheChannels}. While the significance of this activity for RSV remains unclear, the SH protein is identified as a viroporin, belonging to a category of small viral proteins capable of modifying membrane permeability and influencing processes such as budding and apoptosis. Reports suggest that SH has a modest effect in reducing apoptosis via the interaction with B-cell receptor-associated protein 31 (BAP31) \cite{Fuentes2007FunctionProtein}. Moreover, SH appears to inhibit signalling from the tumour necrosis factor-alpha (TNF-\(\alpha\)), an antiviral cytokine \cite{Fuentes2007FunctionProtein}. Recombinant RSV lacking SH demonstrates a somewhat more efficient \textit{in vitro} replication compared to the wild-type virus, possibly attributed to its smaller genome size and reduced number of genes. Additionally, this SH-deficient variant shows a slight attenuation in both mice and chimpanzees \cite{Whitehead1999RecombinantChimpanzees}. It has been observed that \(\Delta\)SH RSV virus infection results in enhanced TNF signalling and increased production of IL-1\(\beta\), which could explain the observations mentioned above \cite{Pollock2017ModulationProtein}. 

In this context, the primary function of the non-structural proteins NS1 and NS2 appears to be the modulation of innate immune responses, achieved through various documented mechanisms. They play a crucial role in disrupting host innate immunity by forming a "Nonstructural degradosome complex", acting as a proteasome-like entity that degrades numerous proteins involved in the innate immune system \cite{Boyoglu-Barnum2019BiologyDevelopment.}. Studies involving single- and double-gene-deleted NS mutant RSV infections (\(\Delta\)NS1, \(\Delta\)NS2, or \(\Delta\)NS1/2) have demonstrated that both proteins function individually and collaboratively to achieve a comprehensive inhibitory effect on type I and III interferons, with NS1 exhibiting a more individual role \cite{Sedeyn2019RespiratoryResponses, Spann2004SuppressionMacrophages}. Cytosolic NS1 exhibits a propensity to translocate to the host nucleus and interact with the gene regulatory domains of immune response genes, thereby influencing gene transcription and modulating the host response against RSV infection \cite{Pei2021Nuclear-localizedTranscription}. NS1 localised to mitochondria inhibits type-I IFN signalling by binding with MAVS, as the MAVS-RIG-1 complex is essential for type-I IFN activation \cite{Boyapalle2012RespiratoryInfection}. NS1 also stimulates \textit{miR-29a} expression, affecting mRNA coding for interferon alpha/beta receptor 1 (IFNAR1) \cite{Sedeyn2019RespiratoryResponses, Zhang2016RespiratoryReceptor}. The NS1 protein interferes with the activation of the IFN gene promoter by inhibiting the phosphorylation of IRF3 \cite{Spann2005EffectsCytokines}. Similarly, NS2 can interfere with the activation of IRF3 through its interaction with RIG-I, inhibiting the activation of IFN response genes \cite{Wright2006TheHumans}. The NS1 protein is capable of interrupting the signalling of JAK/STAT pathways activated by IFN receptor pathways, particularly through the degradation of STAT2 \cite{Wright2006TheHumans, Sedeyn2019RespiratoryResponses}. Both NS1 and NS2 proteins can promote phosphoinositide 3-kinase (PI3K) pathways, enhancing the survival of infected cells and increasing viral yield \cite{Wu2012TheBiology.}. Additionally, NS2 plays a significant role in modulating cell morphology, leading to the shedding of infected cells and the increased dissemination of RSV virions \cite{Sedeyn2019RespiratoryResponses, Liesman2014RSV-encodedObstruction}. Recombinant RSV lacking the NS1 and/or NS2 genes displays heightened sensitivity to interferon, increased apoptosis, and reduced replication efficiency in both cultured cells and experimental animals, with the impact of deleting NS1 being more pronounced \cite{Whitehead1999RecombinantChimpanzees, Teng2000RecombinantChimpanzees}.

The M protein assumes a crucial role in the morphogenesis of virions through its interactions with the cell membrane, viral envelope, and viral nucleocapsid \cite{Li2008AssociationProtein, Marty2003AssociationCells}. Although not imperative for the initiation of viral filament formation, the absence of M results in stunted and immature filaments, highlighting its significant contribution \cite{Mitra2012TheFilaments}. During the early stages of infection, M is detected in the nucleus and may modestly inhibit host transcription during RSV infection. Subsequently, M is found associated with cytoplasmic viral inclusion bodies (IBs), presumed sites of viral RNA synthesis, and the plasma membrane, which serves as the location for virion formation \cite{Ghildyal2006CentralInfection}. M seems to play a role in suppressing viral RNA synthesis by nucleocapsids, likely in preparation for their packaging into virions, \cite{Ghildyal2006CentralInfection, NarayanTalukdar2022RespiratoryVirus}. Additionally, M is deemed necessary for the transport of nucleocapsids from viral inclusion bodies to the plasma membrane \cite{Mitra2012TheFilaments, Collins2013RespiratoryDisease}.

M2-2 functions as an accessory protein and, although not indispensable for virus growth, is expressed at a low level in infected cells \cite{Cheng2005OverexpressionReplication, Melero2006MolecularVirus}. However, its precise status as a virion component remains unknown \cite{Collins2013RespiratoryDisease}. Deletion of M2-2 from recombinant RSV results in a virus displaying delayed and reduced RNA replication along with increased "runaway" transcription. This is in contrast to wild-type RSV, where transcription appears to be downregulated later in infection in favour of RNA replication. These findings suggest that M2-2 plays a crucial role in regulating RNA synthesis. Specifically, as the level of M2-2 increases during the course of infection, it is associated with a reduction in transcription and a concurrent promotion of RNA synthesis \cite{Teng1998IdentificationParticles}.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/09. N_p_l_m21-interaction-overview.pdf}
    \caption[A Model of RNP-Mediated RSV RNA Transcription and Replication.]{\textbf{A Model of RNP-Mediated RSV RNA Transcription and Replication.} The L protein, composed of the connector domain (CD), the methyltransferase domain (MT), the C-terminal domain (CTD), the Cap domain, and the RdRp domain, undergoes conformational rearrangement after interacting with tetrameric P protein in the pre-initiation step. During the initiation stage, the nucleocapsid, composed of N polymer bound to the RSV RNA template, is recruited by interaction with P protein. This causes a partial liberation of the RNA template and allows L to access it. If the RNA transcription follows, the Cap and the MT domains rearrange to catalyze the cap addition and cap methylation. Afterwards, the elongation stage of the transcription continues until the transcript polyadenylation step, which is mediated by M2-1 recruitment. Upon the supply of the N° (RNA-free N), the RNA replication continues. Figure adapted from Cao, 2021 \cite{Cao2021StructuralComplexes}.}
    \label{fig:A Model of RNP-Mediated RSV RNA Transcription and Replication}
\end{figure}

The M2-1 protein, an integral component of the RNP complex, acts as an anti-termination and elongation factor, facilitating the transcription of all hRSV genes, thereby assisting the L protein in transcribing viral genes \cite{Collins1996TranscriptionVirus., Noton2015InitiationReplication}. Predominantly localised within cytoplasmic viral IBs formed during infection \cite{Rincheval2017FunctionalVirus}, M2-1 exists in phosphorylated and nonphosphorylated forms. Both forms interact with the P protein, with phosphorylation dictating binding partners, localisation (cytoplasmic or intra-IB), and the subsequent function \cite{Richard2018RSVTranscription}. Phosphorylated M2-1 in the cytoplasm interacts with P bound to phosphatase PP1, facilitating dephosphorylation and recruitment into IBs, where they regulate viral mRNA transcription \cite{Richard2018RSVTranscription}. Subsequently, M2-1 binds and concentrates with newly synthesised mRNA, displacing P \cite{Blondot2012StructureProtein.}. Phosphorylation then releases M2-1 from mRNA designated for translation, allowing free M2-1 to potentially initiate the cycle anew by interacting with P \cite{Richard2018RSVTranscription}. As mentioned in Section \ref{subsec:Genomic and Virion Composition}, the N protein associates with RSV genomic RNA to form the nucleocapsid, exhibiting either as a long helical polymer or a decameric ring structure \cite{Gonnin2023StructuralNucleocapsids}. This configuration protects the genome from cellular RNA sensors and RNAses \cite{Tawar2009CrystalVirus}, compartmentalising and concentrating it within RSV inclusion bodies where N localises \cite{Rincheval2017FunctionalVirus}. These structures house other RNP components (M2-1, P, and L), facilitating efficient RSV RNA translation and replication. The RSV L protein, a single polypeptide of 2165 residues, serves as the catalytic core of the RSV polymerase, bearing distinct enzymatic domains (RdRp, Cap, and MT) and structural domains (CD and CTD) \cite{Gilman2019StructureComplex, Cao2020Cryo-EMPolymerase}. Its principal role involves the replication and transcription of the viral genome, regulated and supported by the RNP complex. The L protein directly transcribes the genome into mRNA for the expression of each hRSV gene \cite{Cowton2006UnravellingSynthesis, Noton2015InitiationReplication}. During replication, the L protein converts the entire virus genome from negative-sense RNA to positive-sense RNA (antigenome), which serves as a template for generating new negative-sense RNA encapsulated in virions \cite{Cowton2006UnravellingSynthesis, Fearns2000FunctionalVirus}. A characteristic feature of the L protein is its ability, during transcription, to create a gradient of gene expression from 3$^{\prime}$ to 5$^{\prime}$, resulting in higher production of mRNAs for genes located 3$^{\prime}$ compared to those at 5$^{\prime}$ in the genome \cite{Hardy1998TheTranscription, Kuo1996TheMinigenome}. The phosphoprotein (P) acts as a cofactor for the RNP complex, serving as a multimodular adaptor for RNA synthesis through interactions with N-RNA, L, and M2-1 \cite{Blondot2012StructureProtein., Cardone2021APhosphoprotein}. It facilitates N protein interaction, allowing access to L \cite{Sourimant2015FinePhosphoprotein} and acts as a chaperone for newly synthesised, RNA-free N monomer (N0) protein. The N0-P complex prevents N0 association with host RNA, delivering them to nascent RSV genomes or antigenomes \cite{Castagne2004BiochemicalDomain, Galloux2015IdentificationNucleoprotein}. A proposed model of sequential RSV RNA synthesis suggests that P coordinates L activity, revealing a previously unappreciated role for L domain arrangements. Upon binding, a tetrameric P locks CD, MT, and CTD domains of L into a closed conformation (pre-initiation). This complex recruits N:RNA to form an initiation complex, initiating \textit{de novo} RNA synthesis. The domains adopt an open conformation upon promoter recognition, remaining open during elongation, resulting in increased mobility of CD, MT, and CTD domains of L. Depending on N°, RNA synthesis proceeds to transcription (capping and methylation, continued elongation, and polyadenylation) or replication \cite{Cao2020Cryo-EMPolymerase} (Figure \ref{fig:A Model of RNP-Mediated RSV RNA Transcription and Replication}).

\subsection{RSV Inclusion Bodies} \label{subsec:RSV Inclusion Bodies}
A characteristic feature of RSV infections is the presence of cytoplasmic inclusion bodies (IBs), which are structurally and functionally similar to inclusions formed during infections of other negative-sense RNA viruses \cite{Fricke2013P38Assembly, Rincheval2017FunctionalVirus, Li2022PhaseInfections}. This is exemplified by Negri bodies in rabies virus infection \cite{Nikolic2017NegriOrganelles, Nevers2020NegriCompartments}. Similar structures have been identified in the context of measles virus \cite{Zhou2019MeaslesOrganelles}, human metapneumovirus \cite{Cifuentes-Munoz2017HumanTranscription}, Ebola virus \cite{Hoenen2012InclusionReplication}, and Nipah virus \cite{Ringel2019NipahMembrane}. Remarkably, these IBs likely constitute an integral aspect of the cellular cycle for numerous negative-sense RNA viruses. These viral structures share characteristics (i.e., principles of formation, composition, fluidity, and electron density) with cellular biomolecular condensates such as the mediator condensates, stress granules, Cajal bodies, nuclear speckles, and P-bodies \cite{Dumelie2023BiomolecularMicroenvironments, Garabedian2022ProteinSequences, Darling2023KnownCondensates, Li2022PhaseInfections}. All are believed to be formed by the process of liquid-liquid phase separation (LLPS), resulting in membrane-less electron-dense structures that are held together by a highly concentrated scaffold of RNA-protein and protein-protein interactions mediated via intrinsically disordered regions of proteins \cite{Mohanty2022PrinciplesProteins, Hu2022SequestrationPathological}. Recently, these structures were identified to harbour increased concentrations of cellular phospholipids, most likely generated by passive transport generated by electrostatic forces inside these concentrates \cite{Dumelie2023BiomolecularMicroenvironments}. These, along with cholesterol-esters and raft lipids were reported to be contained within RSV IBs two decades ago \cite{Brown2005EvidenceInfection}. The minimal components required for the formation of IB-like structures (called pseudo IBs (pIBs)) are the N and P proteins. Their ectopic expression in cells or the presence of purified N and P \textit{in vitro} have been reported to spontaneously form these structures, with the latter confirming that the RNA acts as a catalyst for pIB formation \cite{Rincheval2017FunctionalVirus, Galloux2020MinimalVitro}. RSV IBs appear at 6 hours, coinciding with early viral protein production, and increase in size and internal complexity as the infection progresses \cite{Risso-Ballester2023SpatialBodies, Rincheval2017FunctionalVirus}. These structures have been observed to localise proximal to the lipid rafts located on the Golgi apparatuses \cite{McDonald2004EvidenceAnalysis}. Inclusion body-associated granules (IBAGs) are subcompartments of RSV IBs, which have distinct protein and RNA composition, biophysical properties and functions \cite{Rincheval2017FunctionalVirus, Jobe2020RespiratorySignaling}. These appear in larger, more mature IBs (2 dimensional mean IB area of 6.4 \(\mu \mbox{m}^2\), as detected in HeLa cell line) \cite{Rincheval2017FunctionalVirus}, which are present as early as 10 hours post-infection \cite{Jobe2021BovineResponses}.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction/Figs/11. RSV IBs.png}
    \caption[Viral RNA Discrete Localisation in hRSV Infected Cells.]{\textbf{Viral RNA Discrete Localisation in hRSV Infected Cells.} hRSV-infected HeLa cells were fixed 24 hours post-infection (HPI) and stained for the N protein (inclusion body identification) and specific RNA species using fluorescence \textit{in situ} hybridisation (FISH). The latter were NS1 mRNA, N mRNA, polyadenylated RNA (PolyA), and viral genomic RNA which all localised within the inclusion body structures. Circles in the zoom panel represent IB boundaries. The scale bar in the top panel indicates 5 \(\mu\)m, while the one present in the zoom panel is 2 \(\mu\)m. Figure taken from Rincheval, 2017 \cite{Rincheval2017FunctionalVirus}.}
    \label{fig:Viral RNA Discrete Localisation in hRSV Infected Cells}
\end{figure}

With regard to the function and composition of RSV IBs and IBAGs, the main seems to be the functional compartmentalisation of viral proteins and replication intermediates, which increase the RSV RNA translation and replication efficiency as well as shielding it from the cellular PAMP sensors \cite{McDonald2004EvidenceAnalysis, Rincheval2017FunctionalVirus, Jobe2020RespiratorySignaling}. RSV proteins M and NS2, along with all of the components of viral polymerase complex (N, P, L, M2-1) have been observed to concentrate inside the IB structures, with M2-1 further sublocalising within the IBAGs \cite{Weber1995NonstructuralSerum, Fricke2013P38Assembly, Rincheval2017FunctionalVirus, Jobe2021BovineResponses}. Rincheval \textit{et. al} observed both the RSV genome and mRNA to localise within the IB structures, with mRNA concentrating within IBAGs and the genomic RNA located in the proximity of the inner IB boundary \cite{Rincheval2017FunctionalVirus} (Figure \ref{fig:Viral RNA Discrete Localisation in hRSV Infected Cells}). Furthermore, several cellular proteins have been observed to be re-localised to the IB structures. These include proteins such as globular actin, which localised to the inner edge of the IBs \cite{Brown2005EvidenceInfection} and the heat-shock protein 70, p38 mitogen-activated protein kinase (MAPK), and O-linked N-acetylglucosamine transferase (OGT), which all concentrate throughout the IB structures \cite{Brown2005EvidenceInfection, Fricke2013P38Assembly}; components of the innate immune signalling cascades such as MAVS and MDA5 \cite{Lifland2012HumanMAVS}, which are sequestered into IBs, and NF-\(\kappa\)B subunit p65, which concentrates within the IBAGs during RSV infection \cite{Jobe2020RespiratorySignaling}; and the poly(A)-binding protein (PABP) and the translation initiation factors eIF4G, eIF4E, eIF3A, eIF4H, eIF4A, eIF4A1, and eIF4B, which were all observed to concentrate within the IBAGs \cite{Rincheval2017FunctionalVirus, Jobe2023ViralCondensates}. This current evidence implies that IBs are functional viral organelles with dual roles of viral RNA replication and transcription and functional inactivation of components of the innate immune response.

\subsection{Host Restriction Factors of RSV} \label{subsec:Host Restriction Factors of RSV}
To date, a limited number of host restriction factors have been identified as pivotal components inhibiting RSV infection in humans. In addition to the actions of IFIT1, IFIT2, and IFIT3 proteins, as discussed in Section \ref{subsec:IFIT Responses to Viral Infections} \cite{Drori2020InfluenzaProteins}, a cadre of other proteins has emerged as key players in curtailing RSV propagation. Notable among these factors is guanylate binding protein 5 (GBP5), a member of the IFN-inducible guanosine triphosphatase family, implicated in diverse cellular processes, including inflammasome activation, signal transduction, translation, and exocytosis \cite{Feng2017InducibleInfection}. GBP5, in the context of RSV infection, has been identified as a negative regulator capable of directly interacting with the SH protein, mediating its excessive secretion from infected cells \cite{Li2020GBP5Virus}. Another crucial player, L13a, a ribosomal protein released from the 60S ribosomal subunit, exerts its inhibitory effect by binding a specific RNA hairpin in the 3$^{\prime}$ untranslated regions of proinflammatory mRNAs, thereby inducing translational silencing \cite{Sampath2004NoncanonicalSynthetase, Vyas2009Genome-WideMonocytes}. In the context of hRSV infection, L13a associates with the RSV M gene mRNA 3$^{\prime}$UTR, suppressing M protein translation and impeding viral replication \cite{Mazumder2014ExtraribosomalDefense}. The interferon-induced transmembrane (IFITM) protein family, comprising small transmembrane interferon-induced proteins \cite{Diamond2013TheProteins}, has demonstrated the capacity to impede RSV infection by disrupting viral entry and replication, although the precise mechanisms remain incompletely characterised \cite{Smith2019Interferon-InducedMembrane}. The apolipoprotein B mRNA-editing enzyme-catalytic polypeptide (APOBEC) family, encompassing 11 enzymes crucial in humoral immunity, features RNA and ssDNA editing functions \cite{Chelico2009StochasticAPOBEC3G}. APOBEC-3G, within the context of RSV, interacts with viral RNA, impairing transcription, inhibiting replication, and elevating the virus's mutation rate \cite{Fehrholz2012TheViruses}. Lastly, interferon-induced protein 44 (IFI44) and interferon-induced protein 44-like protein (IFI44L), both IFN-stimulated genes, were identified through bioinformatic screening as upregulated following hRSV infection \cite{McDonald2016ADisease, Li2021IdentificationVirus}. Subsequent investigations revealed their restrictive role, with overexpression diminishing RSV fitness \cite{Busse2020Interferon-InducedVirus}. Clearly, a diverse array of host proteins acts at various stages of the RSV life cycle. A comprehensive understanding of these restriction factors, especially from a mechanistic standpoint, holds the promise of advancing our comprehension of RSV-host interactions, elucidating its pathology, and paving the way for potential therapeutic avenues.

\section{Aims of the Study} \label{sec:Aims}
It has been reported that exogenously expressed human IFIT1, IFIT2, and IFIT3 restrict hRSV fitness, while their absence increases its replication efficiency \cite{Drori2020InfluenzaProteins}. It is however not known if this restriction is relevant \textit{in vivo} i.e., if under physiological conditions these genes are expressed in RSV infected cells, or if all. Neither is it known how the IFIT proteins enact their anti-RSV functions. We first aimed to elucidate if hRSV infection induces human \textit{IFITs} expression in several cultured cell lines. This would be done by first establishing the \textit{IFITs} induction competency and the integrity of the innate immune signalling pathways within the cell lines used by utilising a range of known activators of these pathways (as depicted in Figure \ref{fig:Pathways Inducing ISG mRNA Production.}). Next, we aimed to dissect what components of hRSV infection, if any, are responsible for the \textit{IFIT} induction by assessing the use of differentially purified hRSV, replication-incompetent hRSV, and pharmacological inhibition of interferon signalling during hRSV infection. Lastly, we aimed to assess if the host restriction of RSV observed \textit{in vivo} \cite{Nair2013GlobalAnalysis, Sacco2014RespiratoryCattle}, especially bovine RSV, could be mediated via enhanced interferon-stimulated genes induction by assessing the \textit{hIFIT} responses to bRSV infection. Alongside these, we mirrored this analysis using bovine cell lines and bovine RSV, as to date the information about bovine \textit{IFIT} induction by bRSV and their potential interaction is currently non-existent. This would also allow us to construct a comprehensive comparative study of human and bovine \textit{IFITs} and their interaction with RSV.

Further, the outstanding question of the mechanism of action of IFIT's negative influence on the RSV life cycle remains to be elucidated. Each IFIT protein has been reported to exhibit its antiviral functions in a diverse way. While IFIT1 acts mainly as a non-self RNA sensor via the recognition of Cap 1 5$^{\prime}$-RNA structures \cite{Mears2018BetterResponse}; IFIT2 has been mainly reported to promote apoptosis, which could be in turn detrimental to the viral replication and progeny production \cite{Chen2017InhibitionApoptosis}; and lastly IFIT3 has been reported to halt the cell cycle progression at the G1/S transition checkpoint \cite{Xiao2006RIG-GProteins}. IFIT1 and IFIT2 have also been reported to inhibit the formation of eukaryotic 48S preinitiation complex \cite{Diamond2014IFIT1:Translation, Guo2000CharacterizationVirus}. Trying to elucidate which of these mechanisms could be important for RSV restriction is complex, however by probing the subcellular localisation of these proteins during infection we can aim to gain insights that would guide us towards the understanding of their mechanism of action. One potential target for IFIT proteins could be the RSV inclusion bodies, as these structures are the sites of viral RNA transcription and replication, as well as the place where the components of the eukaryotic 48S preinitiation complex were described to be concentrated during infection \cite{Rincheval2017FunctionalVirus, Jobe2020RespiratorySignaling, Jobe2023ViralCondensates}. Along with our aim of assessing the subcellular localisation of endogenous human and bovine IFIT proteins during human and bovine RSV infections, we aimed to comprehensively analyse the interaction phenotypes of endogenous and exogenously expressed IFIT proteins with the IB structures. Likewise, we aimed to assess the interactions of endogenous IFIT proteins with ectopically induced RSV pseudo-inclusion bodies.

%Words in text: 6396
%Words in headers: 93
%Words outside text (captions, etc.): 1144
%total = 7633

%acronyms
% chap 1
\nomenclature[z-IFIT]{IFIT}{Interferon-Induced Proteins with Tetratricopeptide Repeats}
\nomenclature[z-ISGs]{ISGs}{Interferon-Stimulated Genes}
\nomenclature[z-UTR]{UTR}{Untranslated Region}
\nomenclature[z-IRSE]{IRSE}{Interferon-Stimulated Response Elements}
\nomenclature[z-JAK]{JAK}{Janus Kinase}
\nomenclature[z-STAT]{STAT}{Signal Transducer and Activator of Transcription}
\nomenclature[z-ISGF3]{ISGF3}{Interferon-Stimulated Gene Factor 3}
\nomenclature[z-IRF]{IRF}{Interferon Regulatory Factor}
\nomenclature[z-PRR]{PRR}{Pattern Recognition Receptor}
\nomenclature[z-PAMP]{PAMP}{Pathogen-Associated Molecular Pattern}
\nomenclature[z-LPS]{LPS}{Lipopolysaccharide}
\nomenclature[z-TLR]{TLR}{Toll-Like Receptor}
\nomenclature[z-RSV]{RSV}{Respiratory Syncytial Virus}
\nomenclature[z-NF\(\kappa\)B]{NF\(\kappa\)B}{Nuclear Factor kappa B}
\nomenclature[z-MDA5]{MDA5}{Melanoma Differentiation-Associated Gene 5}
\nomenclature[z-RIG-I]{RIG-I}{Retinoic Acid-Inducible Gene I}
\nomenclature[z-MAVS]{MAVS}{Mitochondrial Antiviral Signalling Protein}
\nomenclature[z-dsRNA]{dsRNA}{Double-Stranded RNA}
\nomenclature[z-m7G]{m7G}{7-Methyl Guanosine}
\nomenclature[z-5$^{\prime}$-PPP]{5$^{\prime}$-PPP}{5$^{\prime}$-Triphosphoate}
\nomenclature[z-TPR]{TPR}{Tetratricopeptide Repeat}
\nomenclature[z-TBK1]{TBK1}{TANK-Binding Kinase 1}
\nomenclature[z-STING]{STING}{Stimulator of Interferon Genes}
\nomenclature[z-ssRNA]{ssRNA}{Single-Stranded RNA}
\nomenclature[z-eIF]{eIF}{Eukaryotic Initiation Factor}
\nomenclature[z-HEK]{HEK}{Human Embryonic Kidney}
\nomenclature[z-PIV3]{PIV3}{Parainfluenza Virus 3}
\nomenclature[z-PHV]{PHV}{Prospect Hill Virus}
\nomenclature[z-TULV]{TULV}{Tula Virus}
\nomenclature[z-HCV]{HCV}{Hepatitis C Virus}
\nomenclature[z-tRNA]{tRNA}{Transfer RNA}
\nomenclature[z-ORF]{ORF}{Open Reading Frame}
\nomenclature[z-ICAM-1]{ICAM-1}{Intercellular Adhesion Molecule-1}
\nomenclature[z-BAP31]{BAP31}{B-Cell Receptor-Associated Protein 31}
\nomenclature[z-TNF-\(\alpha\)]{TNF-\(\alpha\)}{Tumour Necrosis Factor Alpha}
\nomenclature[z-IFNAR1]{IFNAR1}{Interferon Alpha/Beta Receptor 1}
\nomenclature[z-PI3K]{PI3K}{Phosphoinositide 3-Kinase}
\nomenclature[z-IB]{IB}{Inclusion Body}
\nomenclature[z-LLPS]{LLPS}{Liquid-Liquid Phase Separation}
\nomenclature[z-pIB]{pIB}{Pseudo-Inclusion Body}
\nomenclature[z-IBAG]{IBAG}{IB-Associated Granule}
\nomenclature[z-MAPK]{MAPK}{p38 Mitogen-Activated Protein Kinase}
\nomenclature[z-OGT]{OGT}{O-Linked N-Acetylglucosamine Transferase}
\nomenclature[z-PABP]{PABP}{Poly(A)-Binding Protein}
\nomenclature[z-HPI]{HPI}{Hours Post-Infection}
\nomenclature[z-FISH]{FISH}{Fluorescence \textit{in situ} Hybridization}
\nomenclature[z-GBP5]{GBP5}{Guanylate Binding Protein 5}
\nomenclature[z-IFITM]{IFITM}{Interferon-Induced Transmembrane Proteins}
\nomenclature[z-IFI44]{IFI44}{Interferon-Induced Protein 44}
\nomenclature[z-IFI44L]{IFI44L}{Interferon-Induced Protein 44-Like Protein}
\nomenclature[z-APOBEC-3G]{APOBEC-3G}{Apolipoprotein B mRNA-Editing Enzyme-Catalytic Polypeptide 3G}

% chap 2
\nomenclature[z-DMEM]{DMEM}{Dulbecco's Modified Eagle's Medium} 
\nomenclature[z-FBS]{FBS}{Foetal Bovine Serum}    
\nomenclature[z-PBS]{PBS}{Phosphate Buffered Saline}    
\nomenclature[z-MOI]{MOI}{Multiplicity of Infection} 
\nomenclature[z-PEG]{PEG}{Polyethylene Glycol} 
\nomenclature[z-cDNA]{cDNA}{Complementary DNA}  
\nomenclature[z-MMLV RT]{MMLV RT}{Moloney Murine Leukaemia Virus Reverse Transcriptase}
\nomenclature[z-qPCR]{qPCR}{Quantitative Polymerase Chain Reaction}  
\nomenclature[z-Ct]{Ct}{Cycle Threshold}
\nomenclature[z-GFP]{GFP}{Green Fluorescent Protein}
\nomenclature[z-TBE]{TBE}{Tris-Borate EDTA} 
\nomenclature[z-BSA]{BSA}{Bovine Serum Albumin}
\nomenclature[z-DAPI]{DAPI}{4,6-diamidino-2-phenylindole} 
\nomenclature[z-ANOVA]{ANOVA}{Analysis of Variance}

% chap 3
\nomenclature[z-IRSE]{IRSE}{Interferon-Stimulated Response Elements}
\nomenclature[z-IRF]{IRF}{Interferon Regulatory Factor}
\nomenclature[z-hRSV]{hRSV}{Human Respiratory Syncytial Virus}
\nomenclature[z-RT-qPCR]{RT-qPCR}{Quantitative Real-Time Reverse Transcription Polymerase Chain Reaction}
\nomenclature[z-cDNA]{cDNA}{Complementary DNA}  
\nomenclature[z-GAPDH]{GAPDH}{Glyceraldehyde-3-Phosphate Dehydrogenase}
\nomenclature[z-IU]{IU}{International Units}
\nomenclature[z-IFN\(\gamma\)]{IFN\(\gamma\)}{Interferon Gamma}
\nomenclature[z-LPS]{LPS}{Lipopolysaccharide}
\nomenclature[z-IFN\(\alpha\)]{IFN\(\alpha\)}{Interferon Alpha}
\nomenclature[z-bRSV]{bRSV}{Bovine Respiratory Syncytial Virus}
\nomenclature[z-WT]{WT}{Wild-Type}
\nomenclature[z-SH]{SH}{Small-Hydrophobic}
\nomenclature[z-NS]{NS}{Non-Structural}

% chap 4

% chap 5

% chap 6

% chap 7