\chapter{Introducion} \label{ch:Introduction}
\section{Interferon-Induced Proteins with Tetratricopeptide Repeats} \label{sec:Interferon-Induced Proteins with Tetratricopeptide Repeats}
The interferon-induced proteins with tetratricopeptide repeats (\textit{IFIT}) gene family are interferon-stimulated genes (ISGs) that are activated early in antiviral response. In unstimulated cells, with the exception of some myeloid cell subsets, they are barely detectable; however, upon viral infection, their translated products become some of the most abundant proteins in the cell (\cite{Diamond2013TheProteins}). \textit{IFITs} are conserved in higher animals yet absent in the genomes of plants, insects, and yeast. 

With regards to their genomic composition, they are composed of two exons, the first encoding the ATG, two additional nucleotides and a 5' untranslated region (UTR), with the second exon containing the rest of the coding sequence and 3' UTR (\cite{deVeer1998IFI60/ISG60/IFIT4Genes}). Most of the IFIT family gene members contain multiple copies of specific sequences in their promoter regions called interferon-stimulated response elements (IRSE), which are targets for downstream effectors of the interferon signalling cascade, enabling the fast activation of IFIT gene transcription (\cite{Lou2009Ifr-9/stat2Stat1}).   

Most mammalian IFIT genes are categorised into 4 subgroups named IFIT1, IFIT2, IFIT3, and IFIT5 all with clear orthologous relationships (\cite{Sarkar2004NovelGenes}). Primates, along with some other mammalian species have a duplication of IFIT1 called IFIT1B, which lacks the ISRE in its gene promoter regions, and thus effectivelly only act as a pseudogene. Several rodents including mice and rats have lost the IFIT1 and IFIT5 genes and have duplications of IFIT1B and IFIT3 (\cite{Daugherty2016Evolution-guidedMammals.}) resulting in IFIT1B (typically referred to as IFIT1), IFIT1B-like gene 1, IFIT1B-like 2 gene, IFIT3 and IFIT3B genes. In contrast, avian species have lost most IFIT genes, with only one left resembling human IFIT5 (\cite{Liu2013Lineage-SpecificFamily}). These variations in IFIT gene numbers are most probably the result of differing evolutionary pressures posed by different viruses affecting their respective host species, although it is evident that maintaining IFIT genes in the genome must be beneficial. 

\subsection{Routes of \textit{IFIT} Expression Activation} \label{subsec:Routes of IFIT Expression Activation}
\subsubsection{Interferon Signalling} \label{Interferon Signalling}
\textit{IFIT} induction can be achieved by activating several arms of the innate immune system (Figure \ref{fig:Pathways Inducing ISG mRNA Production.}). The strongest inducers are type I interferons, such as Interferon alpha and beta (IFN\(\alpha\)/\(\beta\)).  Their signalling is mediated via activation of IFN\(\alpha\)/\(\beta\) receptor and the subsequent downstream Janus kinase (JAK), and Signal transducer and activator of transcription (STAT) signal transduction. As a result, interferon-stimulated gene factor 3 (ISGF3), consisting of phosphorylated STAT1 and STAT2 proteins bound to interferon regulatory factor (IRF) IRF9, translocates to the nucleus, binds to the ISRE in the IFIT promoters and induces their transcription (\cite{Der1998IdentificationArrays}; \cite{Mesev2019DecodingInfection}; \cite{Schoggins2011Interferon-stimulatedFunctions}). 

\subsubsection{Pattern Recognition Receptors} \label{Pattern Recognition Receptors}
Another arm of \textit{IFIT} activation is mediated through several pattern recognition receptors (PRRs), which can recognise various pathogen-associated molecular patterns (PAMPs). \textit{IFITs} have been reported to be induced by bacterial PAMPs such as lipopolysaccharide (LPS) from Neisseria meningitidis via activation of Toll-like receptor (TLR) 4 (\cite{Zhou2013InterferonDefense.}). Interestingly, TLR4 has also been observed to be activated by the glycoprotein of respiratory syncytial virus (RSV) (\cite{Funchal2015RespiratoryNeutrophils}). Other TLRs such as TLR3, TLR7, and TLR9 are capable of sensing PAMPs in the form of foreign nucleic acids in the endosomes. TLRs then interact with their adaptor proteins to in turn activate IRF3, IRF7, and nuclear factor kappa B (NF\(\kappa\)B), all of which have the capability to induce \textit{IFIT} family genes (\cite{Diamond2013TheProteins}). These pathways are often prevalent in lymphocytes, monocytes, and mast cells; however, cytosolic nucleic acid sensors are functional in a broader subset of cells (\cite{Ablasser2011WhereFit}).

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/01. IFIT transcription activation figure.png}
    \caption[Pathways Inducing ISG mRNA Production.]{\textbf{Pathways Inducing ISG mRNA Production.} Three routes of ISGs transcription induction are depicted. Pathway A shows virus releasing its genome upon its entry and subsequent foreign nucleic acid detection by cytosolic sensors RIG-I and MDA5. These detect single-stranded and double-stranded RNA respectively. Both activate MAVS, which in turn activates TBK1. STING protein facilitates this activation by enhancing MAVS and TBK1 interaction. Pathway B shows PAMP detection by TLR receptors. TLR4 recognises LPS in the extracellular space while TLR3, TLR7, and TLR9 detect foreign nucleic acids in endosomes. TBK1, as well as TLR activation, leads to the translocation of activated IRF3, IRF7, and NF\(\kappa\)B into the nucleus where they promote ISGs transcription activation. Pathway C depicts type 1 interferon signalling pathway. IFN\(\alpha\)/\(\beta\) receptor activation leads to STAT1 and STAT2 activation. Further addition of IRF9 forms ISGF3 complex, which translocates into the nucleus onto ISRE and promotes ISGs transcription activation. ISG, interferon-stimulated genes; RIG-I, retinoic acid-inducible gene-I; MDA5, melanoma differentiation-associated gene 5; MAVS, mitochondrial antiviral signalling protein; TBK1, TANK-binding kinase 1; STING, stimulator of interferon genes; PAMP, pathogen-associated molecular pattern; TLR, toll-like receptor; IRF, interferon regulatory factor; NF\(\kappa\)B, nuclear factor kappa B; IFN, interferon; STAT, signal transducer and activator of transcription; ssRNA, single-stranded RNA; dsRNA, double-stranded RNA; ISGF, interferon-stimulated gene factor; ISRE, interferon-stimulated response elements. The figure was adapted from Diamond and Farzan, (2013) and Natalya Odoardi's BioRender template. Created with BioRender.com.}
    \label{fig:Pathways Inducing ISG mRNA Production.}
\end{figure}

\subsubsection{Cytosolic Nuclec Acid Sensors} \label{Cytosolic Nuclec Acid Sensors}
Cytosolic RNA sensors include melanoma differentiation-associated gene 5 (MDA5) and retinoic acid-inducible gene I (RIG-I) (\cite{Vladimer2014IFITs:Proteins}). Both signal through mitochondrial antiviral signalling protein (MAVS), which in turn activates IRF3, IRF7, and NF\(\kappa\)B as their downstream effectors (\cite{Ashley2019Interferon-IndependentCytomegalovirus}). In order to prevent activation by cellular RNA molecules, a precise RNA-recognition mechanism has to be conducted by the sensors. While MDA5 senses long double-stranded RNA (dsRNA) (\cite{Brisse2019ComparativeMDA5.}), RIG-I is able to sense differences in the 5’ RNA modifications (\cite{Schlee2016DiscriminatingSensing}). During mRNA maturation in higher eukaryotes, 7-methyl guanosine (m7G) is connected by a 5'- 5' triphosphate bridge, which is referred to as capping (\cite{Devarkar2016StructuralRIG-I}; \cite{Ramanathan2016MRNAApplications}). Several viruses carry uncapped, 5'-triphosphorylated (5'-PPP) or incompletely capped RNA (cap 0), whereas host animals possess cap-1 and cap-2 mRNA moieties (\cite{Choi2018ACaps}), which are all depicted in Figure \ref{fig:Overview of 5'RNA Modifications.}. \textit{IFITs} can be activated by several signal transduction pathways, each with its own inducers and kinetics. A cross-play of these pathways is what in turn orchestrates IFIT response during viral infection.

\begin{figure}
    \centering
    \includegraphics[width=0.75\linewidth]{04. Introduction//Figs/02. 5-RNA Modifications.png}
    \caption[Overview of 5'RNA Modifications.]{\textbf{Overview of 5'RNA Modifications.} Mature mRNA is displayed. Higher eukaryotes modify their mRNA by the initial addition of guanosine (blue) via 5' - 5' triphosphate bridge to the 5' end. Subsequently, the guanosine is methylated into 7-methylguanosine (green) and this modification is referred to as Cap0 structure. Furthermore, the first 2 bases of RNA can be methylated (red) and make either Cap1 or Cap2 structural modifications (yellow arrows). The figure was adapted from \cite{Picard-Jean2013RNAGenomes}.}
    \label{fig:Overview of 5'RNA Modifications.}
\end{figure}

\subsection{Structural Features of IFIT Proteins} \label{subsec:Structural Features of IFIT Proteins}
IFITs are composed of multiple copies of tetratricopeptide repeats (TPR). These are motifs comprised of 3-16 tandem repeats of 34 amino acids, which adopt helix-turn-helix confirmations (\cite{DAndrea2003TPRHelix}). TPRs are conserved in all species from bacteria through plants to higher animals and are commonly found in scaffolding proteins (\cite{Vladimer2014IFITs:Proteins}). IFIT TPRs are comprised of degenerate sequences meaning the conservation of the motif is limited. This allows different IFIT proteins to have a broader profile of protein interactors while still maintaining the overall conformation (\cite{Fensterl2015Interferon-InducedPathogenesis}). IFIT proteins are often subjected to post-translational modifications such as ubiquitination or ISGylation (addition of small IFN-induced ubiquitin-like proteins), which may alter IFIT stability and function.

\subsubsection{IFITs and Their Interaction with RNA} \label{IFITs and Their Interaction with RNA}
IFIT1 and IFIT5 both form a positively charged channel with an affinity for 5' ends of single-stranded RNA (ssRNA) molecules in a sequence non-specific manner. IFIT5 can accommodate 5'PPP RNA and effectively acts as a sensor for these molecules (\cite{Abbas2013StructuralProteins}; \cite{Pichlmair2011IFIT1RNA}). On the other hand, IFIT1 can accommodate m7G, but certain residues inside its channel prevent efficient binding of cap1 and cap2 moieties (\cite{Diamond2014IFIT1:Translation}; \cite{Mears2018BetterResponse}). Through these interactions they can effectively discriminate between self and non-self RNA moieties. IFIT2 is also capable of RNA binding, albeit independent on the 5'-capping state. The C terminals of IFIT2 homo-dimer create a super-helical structure with a positively-charged nucleotide-binding channel on its inner surface, which was observed to bind to AU-rich RNA molecules (\cite{Yang2012CrystalMechanisms}). Mechanistic studies identified IFIT2 binding to mRNAs as important to prevent ribosomal pausing and thus increasing the transnational efficiency of IFIT2-bound mRNAs, a process which has been observed to be hijacked by influenza A virus (\cite{Tran2020InfluenzaMRNAs}).

\subsubsection{Formation of IFIT Protein Complexes} \label{Formation of IFIT Protein Complexes}
As shown in Figure \ref{fig:IFIT Structures and Multimer Formation.}, all IFIT proteins, apart from IFIT5, can form homo- and hetero- dimeric and trimeric complexes. IFIT1 homodimerizes via its C-terminal domain (\cite{Abbas2013StructuralProteins}). The same domain has been shown to interact with the C termini of both IFIT2 and IFIT3, although the IFIT1:IFIT3 complex is more thermodynamically stable (\cite{Fleith2018IFIT3RNA}). Compared to IFIT1, IFIT5 has its dimerization motif shielded by its C terminal TPR and thus stays monomeric in solution (\cite{Kumar2014InhibitionMRNAs}). In contrast, IFIT2 and IFIT3 are rarely seen as monomeric in solution and rather stay as their respective homodimers or IFIT2:3 heterodimers. This is predicted to be done by swapping the third TRP domain in their N-terminal domains (Figure \ref{fig:IFIT Structures and Multimer Formation.}), which keeps them in a more thermodynamically stable configuration (\cite{Yang2012CrystalMechanisms}). IFIT2 homodimer forms a large positively charged cavity which has a propensity to bind dsRNA and AU-rich RNA molecules (\cite{Vladimer2014IFITs:Proteins}; \cite{Yang2012CrystalMechanisms}). IFIT3 interacts with IFIT1 via its C-terminal domain and this interaction increases the half-life of IFIT1 and its specificity for cap0 RNA. Thus, IFIT3 acts as an enhancer of IFIT1 action (\cite{Fleith2018IFIT3RNA}; \cite{Johnson2018HumanStability}). Recently, it has been shown that IFIT1, IFIT2, and IFIT3 form a heterotrimer, although the precise function of this complex has yet to be elucidated (\cite{Fleith2018IFIT3RNA}). In summary, TPR motifs allow IFITs to have a multitude of possible interaction partners, including themselves. Formation of IFIT homo- and hetero-oligomers influences their function and half-life, allowing for variable possible outcomes to occur following IFIT protein production based on the level of each of the proteins. 

\begin{figure}
    \centering
    \includegraphics[width=0.75\linewidth]{04. Introduction//Figs/05. IFIT-complexes.png}
    \caption[IFIT Structures and Multimer Formation.]{\textbf{IFIT Structures and Multimer Formation.} IFIT tertiary and quaternary protein conformations are illustrated. IFIT1 is shown in yellow, IFIT2 in green, IFIT3 in blue, and IFIT5 is highlighted in purple. IFIT1 is able to homodimerise or heterooligomerise with IFIT2 and IFIT3 via its C terminal domain. IFIT5 has the corresponding interaction domain occluded by a helix and thus does not interact with the rest of IFIT proteins. IFIT2 and IFIT3 are capable of forming homodimers or heterodimers by ‘swapping’ TPRs in their N terminal regions. By combining the interactions IFIT1:2:3 heterotrimer is formed. The figure was adapted from \cite{Mears2018BetterResponse}.}
    \label{fig:IFIT Structures and Multimer Formation.}
\end{figure}

\subsection{Functions of IFIT Proteins} \label{subsec:Functions of IFIT Proteins}
\subsubsection{Inhibition of Translation and Viral Replication} \label{Inhibition of Translation and Viral Replication}
IFITs can restrict viral replication by several mechanisms. IFIT1 and IFIT5 can physically prevent non-self RNA from interacting with eukaryotic initiation factor (eIF) 4F (\cite{Kumar2014InhibitionMRNAs}). IFIT1 and 2 block the binding of eIF3 to the eIF2–GTP–Met-tRNA ternary complex by interacting with the eIF3E subunit, whereas human IFIT2, and mouse IFIT1 and IFIT2, can block the formation of the 43S–mRNA complex by binding to the eIF3C subunit (\cite{Diamond2014IFIT1:Translation}; \cite{Guo2000CharacterizationVirus}). This is a cap-independent mechanism for viral translation inhibition, however, extensive IFIT expression can negatively influence the whole cellular translation processes via this mechanism and can hinder normal inflammatory responses. Overexpression of IFIT1 in human embryonic kidney (HEK) 293T cells inhibited the activation of IRF3 and NF\(\kappa\)B and the transcription of IFN\(\beta\) in response to polyinosinic–polycytidylic acid (poly I:C) (\cite{Li2009ISG56Response}). The previously mentioned affinity of IFIT2 for AU-rich RNA can also regulate cellular translation as a lot of transcripts for cytokines or apoptotic factors are rich in adenine and uracil (\cite{Palanisamy2012ControlMicroRNAs}). 

\subsubsection{Modulation of Innate Immune Response Signalling} \label{Modulation of Innate Immune Response Signalling}
IFIT proteins also have the potential to influence innate immune responses by direct interaction with signalling cascade proteins. IFIT3 can potentiate RIG-I signalling by forming a scaffold between MAVS and TANK-binding kinase 1 (TBK1), (\cite{Liu2011IFN-InducedTBK1}), whereas IFIT5 enhances this pathway upstream by recruiting RIG-I to MAVS (\cite{Zhang2013IFIT5Pathways}). IFIT5 has also been reported to help facilitate NF\(\kappa\)B activation via scaffolding its activation kinases (\cite{Zhang2013IFIT5Pathways}). On the other hand, studies show that IFIT1 can both negatively and positively influence RIG-I signalling, upstream of MAVS. IFIT1 interacts with the stimulator of interferon genes (STING), an enhancer of MAVS and TBK1 interaction and it was proposed that these conflicting actions are caused by its multiple binding sites on STING (\cite{Li2009ISG56Response}; \cite{Reynaud2015IFIT1Interferon}). 

\subsubsection{Effects on Cell Cycle Progression and Apoptosis} \label{Effects on Cell Cycle Progression and Apoptosis}
IFIT2 and IFIT3 also have roles in cellular homeostasis. IFIT2 overexpression induces the mitochondrial dependant apoptotic pathway. IFIT2 has been shown to localise on mitochondrial membranes and to directly interact with pro-apoptotic factors which lead to apoptosome formation and subsequent caspase-9 activation (\cite{Chen2017InhibitionApoptosis}; \cite{Diamond2013TheProteins}). Expression of IFIT3 has been observed to alleviate these effects, most probably by the formation of IFIT2:3 dimers (\cite{Mears2018BetterResponse}; \cite{Stawowczyk2011TheApoptosis}). IFIT3 has also been observed to have negative effects on proliferation via indirect degradation of cyclin-dependant kinase inhibitor p27. As a result, cell cycle progression halts at the G1/S transition checkpoint (\cite{Xiao2006RIG-GProteins}). An overview of the IFIT mechanisms of action is depicted in Figure \ref{fig:Overview of IFIT Mechanisms of Action}.

Therefore, IFITs not only act as complementary RNA sensors to RIG-I but are also important as mediators of a plethora of other processes. These range from regulation of transcription, activation and inhibition of innate immune signalling, and regulation of cell life cycle. It is again possible that preference for these actions is influenced by the levels of particular IFIT proteins and their interplay. Taking into consideration the above-described modes of IFIT expression activation, we can speculate that different stimuli such as viruses or signalling molecules at different concentrations will result in quite diverging actions of IFIT proteins on the phenotype of the cell.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/06. IFIT Mechanism of Action.png}
    \caption[Overview of IFIT Mechanisms of Action.]{\textbf{Overview of IFIT Mechanisms of Action.} IFITs have been described to have multiple actions in cells. Pathway A shows their involvement in innate immune signalling modulation. IFIT5 potentiate RIG-I activation of MAVS by scaffolding the two proteins together. IFIT3:3 further scaffolds MAVS to its effector, TBK1, which induces IRF3, IRF7 and NF\(\kappa\)B nuclear translocation when activated. STING protein also potentiates MAVS and TBK1 interaction. IFIT1 has been observed to inhibit STING interaction with MAVS while potentiating its interaction with TBK1. IFIT5 can also indirectly activate NF\(\kappa\)B. Pathway B shows IFIT2:2 involvement with apoptosome formation. IFIT2:3 complex reduces the pro-apoptotic activity of IFIT2:2. IFIT3:3 is involved in cell cycle arrest, as depicted in pathway C. Pathway D shows IFIT inhibition of viral replication and cellular translation. IFIT1 and IFIT5 can directly bind to cap0 and 5’PPP mRNA respectively and prevent its translation. IFIT1 and IFIT2 can both prevent 43S complex formation, while IFIT2 can also inhibit 48S complex formation. IFIT, Interferon-Induced Protein with Tetratricopeptide Repeats; RIG-I, retinoic acid-inducible gene I; MAVS, mitochondrial antiviral signalling protein; TBK1, TANK-binding kinase 1; STING, stimulator of interferon genes; IRF, interferon regulatory factor; NF\(\kappa\)B, nuclear factor kappa B; eIF, eukaryotic elongation factor; ssRNA, single-stranded RNA; tRNA, transfer RNA. The figure was adapted from \cite{Mears2018BetterResponse} and \cite{Diamond2013TheProteins}. Created using BioRender.com.}
    \label{fig:Overview of IFIT Mechanisms of Action}
\end{figure}

\subsection{IFIT Responses to Viral Infections} \label{subsec:IFIT Responses to Viral Infections}
Human IFIT effects on single-stranded RNA viruses have been studied extensively within the past decade. \cite{Rabbani2016Identification3} reported that IFIT1 and IFIT3 restrict human parainfluenza virus 3 (PIV3, negative-sense ssRNA virus), while ectopic expression of IFIT2 and IFIT5 did not affect the viral fitness. \textit{In vitro} IFIT1 knock-down experiments using short hairpin RNA showed marked enhancement of protein expression of human PIV2, PIV5, and mumps virus, all negative-sense ssRNA viruses (\cite{Andrejeva2013ISG56/IFIT1Synthesis}; \cite{Young2016HumanFamily}). Hantaviruses, a family of negative-sense ssRNA viruses such as Prospect Hill virus (PHV), and Tula virus (TULV), strongly induce IFIT3 expression during the course of their infection (\cite{Matthys2011TheInduction}). Most importantly within the non-segmented negative-sense ssRNA viruses, IFIT1, IFIT2, and IFIT3 have been reported to be globally antiviral against respiratory syncytial virus (RSV). Dori and colleagues either overexpressed or silenced these 3 IFITs and assessed RSV fitness by quantifying the viral mRNA produced. They reported decreased viral fitness following overexpression, while silencing of IFITs increased the viral mRNA production (\cite{Drori2020InfluenzaProteins}). Another negative-sense ssRNA virus, which bears a segmented genome, influenza A virus, has been shown to upregulate IFIT1, IFIT2, and IFIT3 in primary macrophages during infection (\cite{Lietzen2011QuantitativeMacrophages}). Recent studies also show the ability of IFIT1 and IFIT2 to negatively influence the viral fitness of influenza, as the knock-down of these IFITs resulted in increased viral RNA and protein production, while the opposite was true if IFIT1 and IFIT2 were overexpressed (\cite{Zhu2023TheSynthesis}). Human IFIT1 has been observed to be differentially expressed during hepatitis C virus (HCV) infection. HCV was described to suppress IFIT1 upregulation and subsequent experiments showed an inverse relationship between artificial levels of IFIT1 and HCV ability to infect host cells (\cite{Raychoudhuri2011ISG56Replication}, \cite{Ishida2019HepaticInfection}). Currently, there is very limited information about the effect of bovine IFIT expression on the viral fitness of bovine viruses.




\section{Respiratory Syncytial Virus} \label{sec:Respiratory Syncytial Virus}
Human and bovine Respiratory Syncytial Virus (hRSV and bRSV) are enveloped, negative-sense, single-stranded RNA viruses belonging to the Orthopneumovirus genus (\cite{Afonso2016Taxonomy2016.}). These viruses are recognized for inducing acute respiratory disease, which is confined to their respective hosts. Despite this host specificity, shared genetic and antigenic characteristics are observed among these closely related viruses (\cite{Buchholz2000ChimericVaccine}). hRSV primarily infects humans, while bRSV infects cattle. They are known to be the leading causes of lower respiratory tract illness in their respective hosts (\cite{Nair2013GlobalAnalysis.}; \cite{Sacco2014RespiratoryCattle}). Although these viruses can infect individuals of all ages, severe respiratory illness, including bronchiolitis and pneumonia, is more common in infants, the elderly, and immunocompromised individuals (\cite{AR2005RespiratoryAdults}; \cite{Coultas2019RespiratoryAge}). On a global scale, hRSV affects nearly all children by the age of five, resulting in approximately 33 million cases and 3.2 million hospital admissions annually (\cite{Shi2017GlobalStudy}). Tragically, around 60,000 hospitalized children under the age of five succumb to the infection, with a notably higher mortality rate among infants under six months, especially those at high risk due to factors such as premature birth and pre-existing chronic lung and heart conditions (\cite{Shi2017GlobalStudy}; \cite{Jha2016RespiratoryVirus}; \cite{Coultas2019RespiratoryAge}). Bovine RSV infection also poses a significant threat to the global cattle farming industry, resulting in substantial economic losses (\cite{Brodersen2010BovineVirus}; \cite{Valarcher2007BovineInfection}).

Despite extensive research and development efforts spanning several decades, the availability of approved vaccines against hRSV remains limited. Historical challenges include the failure of the formalin-inactivated RSV vaccine, which led to enhanced natural infection in some vaccinated children (\cite{Fulginiti1969RespiratoryVaccine}; \cite{Kim1969RespiratoryVaccine}). Additional hurdles involve the heterogeneity of RSV subtypes and the incomplete immune responses generated against the virus. However, recent advances in the field have introduced a range of vaccine strategies, including live-attenuated wild-type virus, vector-based, viral protein sub-unit, mRNA-based, and DNA-based candidates. Presently, 24 vaccines are in various stages of clinical development, including two licensed vaccines: Arexvy (GSK) and Abrysvo (Pfizer). These vaccines are used as immunizing agents to prevent lower respiratory tract disease in older adults, with Abrysvo additionally indicated for passive immunization of infants through maternal administration during pregnancy (\cite{Topalidou2023RespiratoryVaccines}). In contrast, there are several vaccines available for bRSV, with the first one dating back to the 1970s. However, these vaccines offer only mild protection. Consequently, there is a pressing need to develop more efficacious vaccines for both hRSV and bRSV (\cite{Ellis2017HowCattle}).

% from FJ
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Genomic and Virion Composition} \label{subsec:Genomic and Virion Composition}
Orthopneumoviruses have a linear negative-sense RNA genome, (15,223 nucleotides [nt] in length for hRSV A2 [GenBank accession no. KT992094] and 15,140 nt for bRSV A51908 [GenBank accession no. NC038272] which encodes genes for 11 proteins from 10 mRNAs. These are, in order from the 3' end: NS1, NS2, N, P, M, SH, G, F, M2, and L (Fig 1.1); and are flanked by 3' leader and 5' trailer sequences. This follows the typical genome organisation of mononegaviruses, encoding genes for a nucleoprotein (N), polymerase cofactor (P), matrix protein (M), surface glycoprotein (G) and polymerase (L) (Pfaller et al., 2015). M21 is The only exception is the M2 gene which has two slightly overlapping open reading frames (ORFs). 

Orthopneumoviruses may be distinguished from related viruses by the presence of four extra genes: NS1, NS2, SH and M2. Three of the encoded proteins (non-structural proteins; NS1 and NS2 and small hydrophobic; SH protein) are non-essential for virus replication but function in inhibiting innate immune signalling (Schlender et al., 2000, Spann et al., 2004, Pollock et al., 2017, Taylor et al., 2014, Bitko et al., 2007, Atreya and Kulkarni, 1999). NS1 and NS2 are not found in any other virus in the Mononegavirales order and have been shown to antagonise apoptosis (Bitko et al., 2007) and interferon (IFN)-mediated host responses by targeting both type I and II IFN induction (Schlender et al., 2000, Spann et al., 2004, Spann et al., 2005) and signalling (Lo et al., 2005). The two proteins localise in the cytoplasm, can form homotetramers, and work synergistically or individually (Schlender et al., 2000, Swedan et al., 2009). Specifically, NS2 interacts with a cytoplasmic nucleic acid receptor (RIG-I) inhibiting its interaction with the mitochondrial antiviral-signalling protein (MAVS) (Ling et al., 2009). Similarly, NS1 can inhibit phosphorylation of interferon regulatory factor 3 (IRF-3) by interacting with MAVS (Boyapalle et al., 2012) or by decreasing the levels of IKKε, a key protein kinase that specifically phosphorylates IRF3 (Swedan et al., 2009). The proteins can also decrease levels of other components of the interferon signalling pathway. NS1, and to a lesser extent NS2, mediate a decrease in the levels of TRAF3 in a non-proteasomal mechanism (Swedan et al., 2009). Recently, the NS proteins have also been shown to be involved in formation of an “NS -degradasome” that promotes the degradation of components of IFN induction or signalling, such as, RIG-I and IRF-3 and -7, TBK1 and STAT2 (Goswami et al., 2013). Consequently, activation of the cytotoxic T lymphocyte component of the adaptive immune response is also suppressed (Kotelkin et al., 2006).

The M2 gene which encodes the M2-1 and M2-2 proteins with roles in the regulation of viral RNA synthesis, is unique to members of the Pneumoviridae family and the basis for their recent separation from paramyxoviruses (Rima et al., 2017). These are expressed from overlapping ORFs, with expression of the second ORF using a unique mechanism of translation of the mRNA (Ahmadian et al., 2000, Gould and Easton, 2007, Powell, 2010). M2-1 is essential for virus replication and mainly localises in inclusion bodies (IBs) formed in the cytoplasm of infected cells (Rincheval et al., 2017). The details of viral transcription are discussed later, but briefly, once initiated at the 3' extragenic leader sequence, viral mRNA transcripts are produced sequentially from the 3' end of the genome in a stop-restart mechanism directed by gs and ge sequences that flank each gene (Kuo et al., 1996b). M2-1 acts as a transcription factor that prevents the early termination of the transcription machinery at ge signals, thus enhancing the transcription of genes towards the 5' end of the genome (Collins et al., 1995, Collins et al., 1996, Fearns and Collins, 1999b). M2-1 forms tetramers that can exist in phosphorylated and non-phosphorylated forms (Lambert et al., 1988, Tran et al., 2009). Phosphorylation was found to be important for hMPV replication and pathogenesis (Cai et al., 2016). The phosphorylation status of RSV M2-1 has been suggested to determine its interacting partners, localisation (cytoplasmic or intra-IB) and subsequent function (Richard et al., 2018). Both forms can interact with the P protein. However, interaction of phosphorylated M2-1 in the cytoplasm with P bound to a phosphatase, PP1, facilitates its dephosphorylation and recruitment into IBs, where they regulate viral mRNA transcription (Richard et al., 2018). M2-1 then binds and concentrates with the newly synthesised mRNA, an interaction which displaces P (Blondot et al., 2012, Tran et al., 2009, Cuesta et al., 2000). Phosphorylation then releases M2-1 from the mRNA destined for translation and free M2-1 may then restart the cycle by interacting with P (Richard et al., 2018). Whether M2-1 plays a role in translation remains unclear but an interactomics study has identified that M2-1 interacts with several cellular proteins involved in mRNA metabolism and translation (Bouillier et al., 2019). Separately, a CCCH zinc-binding motif, that usually plays a role in RNA binding, is found in the conserved functionally essential N-terminus of both RSV (Hardy and Wertz, 2000, Tang et al., 2001, Zhou et al., 2003) and MPV M2-1 (Cai et al., 2015). The exact role of this motif, other than stabilisation of the tetrameric complex (Esperante et al., 2013), is presently unclear. M2-1 was also shown to play a role in M protein IB localisation and its interaction with the ribonucleoprotein complex (Li et al., 2008, Kiss et al., 2014). M2-2 is encoded by the second ORF of the M2 gene and unlike M2-1 is an accessory protein, not essential for virus growth. However, deletion of this ORF from hRSV A2 resulted in host range-specific partial attenuation of the virus (Jin et al., 2000a). Its absence also increased viral mRNA transcription and expression of the antigenic proteins (F and G) whilst genome and antigenome replication were reduced in comparison (Bermingham and Collins, 1999, Jin et al., 2000a). This suggests that accumulation of the M2-2 protein, perhaps in later stages of infection when significant viral proteins are expressed, directs RNA synthesis in favour of genome and antigenome replication by downregulating transcription (Collins et al., 2013). Since both processes use the same machinery and RNA template, a balance is important for efficient virus replication. In effect, M2-2 acts as an RNA synthesis regulatory factor, although the exact mechanism of this switch is not clearly understood. Viruses lacking the expression of this protein are currently being investigated as live-attenuated vaccine candidates with promising results (McFarland et al., 2020, McFarland et al., 2018). Recently, hMPV M2-2 protein was found to inhibit innate immune signalling by targeting MAVS (Chen et al., 2016, Kitagawa et al., 2017, Ren et al., 2014, Ren et al., 2012). A similar role has not been reported for orthopneumovirus M2-2 proteins and there is no significant sequence identity between the pneumovirus M2-2 proteins (Collins et al., 2013). The final extra orthopneumovirus gene, SH, encodes one of three surface glycoproteins found in the virus lipid envelope (Fig 1.2). Unlike the unique NS and M2 proteins, SH orthologs are found in some members of the paramyxoviridae family and their role will be discussed in section 1.3. The other encoded proteins, nonglycosylated structural protein M, glycoproteins G and F involved in attachment and entry into host cells, and components of the RNA-dependent RNA polymerase complex (RdRp), N, P and L, will be discussed in the next two sections.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/07. RSV-composition.png}
    \caption[RSV-composition]{\textbf{RSV-composition} }
    \label{fig:RSV-composition}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/08. N-structure.jpeg}
    \caption[N-structure]{\textbf{N-structure} }
    \label{fig:N-structure}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction//Figs/09. N_p_l_m21-interaction-overview.png}  
    \caption[Nplm21-interaction-overview]{\textbf{Nplm21-interaction-overview} }
    \label{fig:Nplm21-interaction-overview}
\end{figure}

Bovine and human RSV are enveloped viruses possessing negative-sense ssRNA genome approximately 15 kilobase-pairs in size. Upon viral attachment and entry, the genome is released into the cytoplasm in the form of a ribonucleoprotein i.e. a complex of RSV genome with its nucleoprotein (N protein) (\cite{Noton2015InitiationReplication}). Viral replication also requires the RNA-dependent RNA polymerase (L protein) and the phosphoprotein (P protein), whereas RSV transcription also requires M2-1 protein, which is hypothesised to prevent premature termination of this process (\cite{Tanner2014CrystalPhosphorylation}). RSV transcripts are modified with 5’ cap structures prior to their translation. Both viral replication and transcription were previously believed to happen diffusely throughout the cytoplasm; however, recent studies suggest viral inclusion bodies (IBs) are the main site for RSV replication and transcription (\cite{Rincheval2017FunctionalVirus}). 

\subsection{Viral Life Cycle} \label{subsec:Viral Life Cycle}
put this quite simplified

\subsection{RSV Inclusion Bodies} \label{subsec:RSV Inclusion Bodies}
Human RSV infection is characterised by the formation of intracytoplasmic granules known as inclusion bodies (IBs; Fig 1.8) (Fricke et al., 2013, Lifland et al., 2012, Rincheval et al., 2017). These have been consistently observed both in vitro and in vivo (Norrby et al., 1970, Neilson and Yunis, 1990, Garcia et al., 1993, Lifland et al., 2012, Rincheval et al., 2017, Fricke et al., 2013), and are structurally and functionally similar to inclusions formed by several other viruses classified within the Mononegavirales order. This includes IBs induced by rabies virus infection termed Negri bodies (Lahaye et al., 2009, Nikolic et al., 2016, Nikolic et al., 2017), hMPV (Cifuentes-Munoz et al., 2017), MeV (Zhou et al., 2019), NiV (Ringel et al., 2019), and Ebola virus (Hoenen et al., 2012), and likely represent an essential component of the cellular cycle of many negative-sense RNA viruses. Recently, IBs were also observed in IAV infected cells, negative strand RNA viruses with a segmented genome (Alenquer et al., 2019). Interestingly, viral IBs also share many characteristics with biomolecular condensates called liquid organelles (Nevers et al., 2020). These form by liquid-liquid phase separation, LLPS, a process which favours macromolecular-macromolecular over macromolecular-water interactions (Banani et al., 2017, Uversky, 2017, Nevers et al., 2020, Alberti et al., 2019). The resulting structures have high molecular density, are composed of RNA and proteins (mostly RNA-binding proteins, RBPs), and also lack surrounding membranes, unlike classical organelles. These organelles are not exclusive to pathogens; cellular examples of biomolecular condensates include stress granules and P-bodies that form in the cytoplasm, and nucleoli and Cajal bodies found in the nucleus (Gomes and Shorter, 2019, Banani et al., 2017). They can be identified by their liquid-like properties based on the following criteria: spherical shape, ability to rapidly undergo fusion and fission, and their high fluidity/dynamic state when assessed by fluorescence recovery after photobleaching (FRAP) (Gomes and Shorter, 2019, McSwiggen et al., 2019, Alberti et al., 2019).



\subsubsection{Assembly of RSV IBs} \label{Assembly of RSV IBs}
With regards to viral IBs – biomolecular condensates formed in the context of virus infection – several have been shown to have liquid-like properties based on the criteria currently used for defining liquid organelles. These include cytoplasmic inclusions induced in RSV (Rincheval et al., 2017), MeV (Zhou et al., 2019), VSV (Heinrich et al., 2018), rabies (Nikolic et al., 2017) and ebola (Hoenen et al., 2012) virus infected cells, formed through the process of LLPS (Alberti et al., 2019, Nevers et al., 2020). However, the exact mechanism of their formation is less well characterised. RSV IBs appear in the cytoplasm at around 6h post infection, as viral proteins are expressed and grow in size as infection proceeds (Fricke et al., 2013, Lifland et al., 2012, Rincheval et al., 2017). Although they colocalise all proteins of the polymerase complex (Fig 1.8), for RSV, rabies and measles viruses, the N and P proteins are the minimum components essential for their formation in the absence of infection (Lifland et al., 2012, Nikolic et al., 2017, Rincheval et al., 2017, Zhou et al., 2019). Co-expression of the N and P proteins alone results in the formation of IB-like structures (pseudo-IBs) with similar properties to the viral inclusions. For VSV, the minimal system requires the presence of L (Heinrich et al., 2018). Viral IBs are essentially condensates of RNA and protein as proven by the large amounts of nucleocapsids and viral mRNA contained in these structures (Fig 1.9). The minimal system however does not rule out the involvement of RNA interactions in formation of the structures as cellular RNA may be used in the absence of virus infection. More recently, the N and P proteins of RSV (Galloux et al., 2020b) and MeV (Guseva et al., 2020) were shown to induce liquid droplets outside of the cellular environment, thus showing they are the main drivers of viral phase separation. However, there was a discrepancy between the two viruses in cells as the formation of pseudo-IBs by RSV N and P required N:RNA interaction (Galloux et al., 2020b); however, the MeV proteins did not (Guseva et al., 2020, Zhou et al., 2019). Guseva et al. further showed that although RNA is dispensable for MeV N:P phase separation, it preferentially colocalises to and triggers assembly of nucleocapsid-like particles within the structures formed.

Similar to cellular proteins involved in inducing LLPS, assembly of ectopically expressed N and P proteins into inclusions has been shown to involve oligomerisation and intrinsically disordered domains within these viral proteins (Zhou et al., 2019, Nikolic et al., 2017, Galloux et al., 2020a). The higher degree of intrinsic disorder within P proteins of RSV (Gilman et al., 2019, Pereira et al., 2017), MeV and rabies virus (Nevers et al., 2020, Longhi et al., 2017) suggests it has a greater propensity to phase separate but the involvement of N shows there are crucial mechanisms provided by this protein. Mechanistically, for RSV, the central oligomerisation domain and Cterminal domain (an intrinsically disordered region) of P, plus the ability of N to interact with both P and RNA, were shown to be required for pseudo-IB formation (Galloux et al., 2020b). Despite sequence variability, similar intrinsically disordered and oligomerisation domains are present in rabies and MeV P proteins (Nevers et al., 2020), and were also shown to be required for pseudo-IB formation (Nikolic et al., 2017, Zhou et al., 2019). Therefore, it appears that the N and P proteins and RNA (depending on the virus) are the “scaffolds” required for IB formation. Thus, as the concentrations of the viral proteins increase during infection, oligomerisation of P, multiple interactions facilitated by the intrinsically disordered regions of N and P (Pereira et al., 2017) and interactions with viral RNA (Galloux et al., 2015) could act in synergy to initiate phase separate and nucleation of the IB. These may then mature and grow in size as they further colocalise newly synthesised viral RNA and protein. Of note, several other viral protein-protein and protein-RNA interactions which could contribute towards maturation exist in IBs. For instance, the viral nucleocapsid which concentrates in IBs (Fig 1.8 and Fig 1.9), is composed of genomic RNA, N, L and P proteins (Fig 1.2) (Easton et al., 2004). M2-1 also can form multiple interactions - with P (Mason et al., 2003, Richard et al., 2018), M (Li et al., 2008) and viral mRNA (Rincheval et al., 2017). However, the process of maturation appears to be a complex and regulated process as indicated by the organisation of the condensates into multi-phasic structures. Rincheval et al. recently demonstrated the presence of sub-compartments within RSV IBs with distinct protein and RNA composition, biophysical properties and functions termed inclusion body associated granules; IBAGs (Rincheval et al., 2017). M2-1 and newly synthesised viral mRNA, presumably interacting, mainly localised to these IBAGs (Rincheval et al., 2017). This is similar to the distinct, coexisting liquid phases that make up the nucleolus (Feric et al., 2016). The IBAGs are dynamic structures, and their contents are subsequently released to the cytoplasm. However, further investigation is required to understand the mechanism underlying the evolution of biomolecular condensates from a single phase to sub-compartments carrying out distinct functions.

\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{04. Introduction/Figs/11. RSV IBs.png}  
    \caption[RSV IBs]{\textbf{RSV IBs} }
    \label{fig:RSV IBs}
\end{figure}

\subsubsection{Function of RSV IBs} \label{Function of RSV IBs}
The formation of membrane-less liquid organelles allows cells to spatially compartmentalise specific cellular processes whilst allowing biomolecule exchange with the cyto-/nucleoplasm (Banani et al., 2017, Gomes and Shorter, 2019). Viruses also appear to use biomolecular condensation for a multitude of virus-virus/virus-host interactions with the advantage of increasing the efficiency of replication. Prominent examples include IBs which are essentially replication complexes and sites of nucleocapsid assembly (Nevers et al., 2020), and also lipid rafts where particle assembly takes place (McDonald et al., 2004). The functional compartmentalisation of viral proteins and replication intermediates into inclusion bodies also minimises their interaction with factors of the host immune response. The exact role of IBs induced during infection was previously unclear and initially thought of as “aggregates of viral nucleocapsids” (Garcia et al., 1993). Recent evidence shows it is a common mechanism used by several mononegaviruses and further demonstrates their involvement in processes such as viral RNA replication and host immune modulation (Nevers et al., 2020).

Analysis of hRSV infected cells by immunofluorescence (IF) microscopy shows concentration of several viral proteins within IBs including N, P, L, M2-1, M and NS2 (Carromeu et al., 2007, Garcia et al., 1993, Lindquist et al., 2010, Brown et al., 2005, McDonald et al., 2004, Ghildyal et al., 2002). Most of these viral proteins are directly/indirectly associated with the nucleocapsid, explaining their colocalization within the structure. Interestingly, all of the proteins involved in transcription and replication (N, P, L and M2-1) co-localise in the IBs, whereas the surface glycoproteins are excluded. Recent identification of viral genomic RNA and mRNA within the IB (Fig 1.9) strongly suggests that these organelles are functional and are the primary site for viral RNA replication and transcription within the infected cell (Rincheval et al., 2017). Interestingly, distinct spatial organisations were observed between viral genomic and mRNA. As already noted above, mRNA concentrated in IBAGs whereas genomic RNA mostly localised towards the boundary of the IB (Fig 1.9). Further study is required to understand the significance of this organisation. Similarly, IBs produced in rabies (Lahaye et al., 2009) and ebola (Hoenen et al., 2012) virus infected cells were also shown as the sites of RNA replication and transcription. However, this does not appear to be universal a trend since viral RNA replication of Nipah virus (NiV) was recently shown to occur outside both its structurally distinct IB populations (Ringel et al., 2019).

As discussed in sub-section 1.2.3, a number of host cell proteins have been shown to interact with or co-localise within RSV induced IBs. These include antiviral proteins MAVS and MDA5 (Lifland et al., 2012), and signalling proteins involved in the cellular response to viral infection – p38 mitogen-activated protein kinase (MAPK) and O-linked N-acetylglucosamine transferase (OGT) (Fricke et al., 2013). In addition, the chaperone protein, HSP70 that protects cells from stress, has also been observed within RSV IBs (Brown et al., 2005). MDA5 and MAVS sequestration into IBs were shown to result in a reduction of IFNβ mRNA induction (Lifland et al., 2012). Similarly, sequestration of p38 MAPK and OGT suppressed MK2 activity and stress granule assembly. Recruitment of these key signalling proteins into IBs following RSV infection appears to be an additional mechanism of immune antagonism whereby their roles are inhibited through sequestration. However, whether these integrate with the scaffolds during the process of IB formation or associate with the granules as they mature, remains to be elucidated. Of note, viral IBs are induced in the early stages of infection, at a time when activation of innate immune signalling pathways is crucial. Thus, the use of IBs for innate immune antagonism is an important phenomenon for enhancing virus fitness during infection. Currently, it is unclear how broad this mechanism for regulating host antiviral responses is for viruses that induce cytoplasmic IBs. Much of what is currently known is on RSV IBs (Lifland et al., 2012, Fricke et al., 2013, Brown et al., 2005), although cellular WD repeat-containing protein 5 (Zhou et al., 2019, Ma et al., 2018), heat shock protein 72 (Zhang et al., 2005b) and actin-modulating protein cofilin (Koga et al., 2015) were also observed in measles virus IBs. Localisation of WD repeat-containing protein 5 in IBs was shown to enhance virus infection (Ma et al., 2018). It is therefore essential that further research is conducted to determine the scope of this strategy of immune modulation and to examine the implications for host responses to infection.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

IBs are amorphous membranelles aggregates, which have been observed to form as early as 6 hours post-infection and often localise in close proximity to the nucleus (\cite{Bachi1973MorphogenesisVirus}; \cite{Jobe2020RespiratorySignaling}). They have characteristics of biomolecular condensates (with a similar electron density under electron microscopy to the nucleoli) and resemble cytoplasmic inclusions associated with other viral infections e.g. Negri bodies forming during rabies virus infection (\cite{Nikolic2017NegriOrganelles}). RSV N and P proteins are found on the edges of IBs, while L and M2-1 proteins are located inside the structure. (\cite{Rincheval2017FunctionalVirus}), using super-resolution imaging, discovered internal IB compartments called IB-associated granules (IBAGs). These not only contained different viral proteins compared to the rest of the IB, but nascent RNA labelling also suggested IBAGs to be the sites where newly synthesized mRNA concentrates (\cite{Jobe2020RespiratorySignaling}; \cite{Richard2018RSVTranscription}). Taken together, data suggests that IBs are specialised sites which enable viral replication and transcription and that these processes may be compartmentalised.

It is currently unknown if IFIT proteins interact with these structures. Given the well-described roles of IFITs in sensing viral RNA, it is possible that different IFITs may localise into IBs and IBAG, potentially interacting with viral RNA before the capping process takes place or directly interfere with the translational machinery. It is also possible that IFITs are restricted from accessing IBs by an unknown mechanism. Understanding these processes may reveal novel routes of increasing viral sensitivity to the innate immune response.



\section{Aims of the Study} \label{sec:Aims}
Here have a page of the aims.

A detailed comparative study of human and bovine IFITs and their interaction with RSV. Looking at their induction, expression, and changes in localisation. Looking at interaction with RSV IBs. Assessing the involvement of IFIT2 in the RSV life cycle.

For your Aims make sure these are very detailed, so 2 pages roughly. Not 3-4 bullet points of 2 lines each, which I have seen!

clearly human ifits are global restictors of hrsv. but is it relevant, do they get induced? can we isolate what is inducing them if so? 

are human ifits induced by brsv and thus have a potential to crossprotect from it?

does this reproduce in bifit with brsv and crossprotection from hrsv?

through what mechanism are ifits inhibitory? where are they located during infection?

do they interact with parts of rsv stuff?

is there a difference between ifits in interaction with htese structures?

%acronyms
\nomenclature[z-DEM]{DEM}{Untranslated}

\nomenclature[z-IBAG]{IBAG}{IB-Associated Granule}
\nomenclature[z-IB]{IB}{Inclusion Body}
\nomenclature[z-BVDV]{BVDV}{Bovine Viral Diarrhoea Virus}
\nomenclature[z-HCV]{HCV}{Hepatitis C Virus}
\nomenclature[z-TULV]{TULV}{Tula Virus}
\nomenclature[z-PHV]{PHV}{Prospect Hill Virus}
\nomenclature[z-PIV3]{PIV3}{Parainfluenza Virus 3}
\nomenclature[z-STING]{STING}{Stimulator of Interferon Genes}
\nomenclature[z-TBK1]{TBK1}{TANK-Binding Kinase 1}
\nomenclature[z-HEK]{HEK}{Human Embryonic Kidney}
\nomenclature[z-eIF]{eIF}{Eukaryotic Initiation Factor}
\nomenclature[z-ssRNA]{ssRNA}{Single-Stranded RNA}
\nomenclature[z-TPR]{TPR}{Tetratricopeptide Repeat}
\nomenclature[z-IFIT]{IFIT}{Interferon-Induced Protein with Tetratricopeptide Repeats}
\nomenclature[z-ISG]{ISG}{interferon-Stimulated Gene}
\nomenclature[z-UTR]{UTR}{Untranslated Region}
\nomenclature[z-IRSE]{IRSE}{Interferon-Stimulated Response Elements}
\nomenclature[z-STAT]{STAT}{Signal Transducer and Activator of Transcription}
\nomenclature[z-JAK]{JAK}{Janus Kinase}
\nomenclature[z-IRF]{IRF}{Interferon Regulatory Factor}
\nomenclature[z-ISGF3]{ISGF3}{Interferon-Stimulated Gene Factor 3}
\nomenclature[z-5'-PPP]{5'-PPP}{5'-triphosphoate}
\nomenclature[z-m7G]{m7G}{7-methyl guanosine}
\nomenclature[z-dsRNA]{dsRNA}{Double-Stranded RNA}
\nomenclature[z-MAVS]{MAVS}{Mitochondrial Antiviral Signalling Protein}
\nomenclature[z-RIG-I]{RIG-I}{Retinoic Acid-Inducible Gene I}
\nomenclature[z-MDA5]{MDA5}{Melanoma Differentiation-Associated Gene 5}
\nomenclature[z-NF\(\kappa\)B]{NF\(\kappa\)B}{Nuclear Factor kappa B}
\nomenclature[z-RSV]{RSV}{Respiratory Syncytial Virus}
\nomenclature[z-PRR]{PRR}{Pattern Recognition Receptor}
\nomenclature[z-PAMP]{PAMP}{Pathogen-Associated Molecular Pattern}
\nomenclature[z-LPS]{LPS}{Lipopolysaccharide}
\nomenclature[z-TLR]{TLR}{Toll-Like Receptor}


%Words in text: 2625
%Words in headers: 112