\chapter{Assessment of Transcriptional Induction of Human IFITs in the Context of RSV} \label{ch:Assessment of Transcriptional Induction of Human IFITs in the Context of RSV}
\section{Introduction and Aims} \label{sec:Introduction and Aims-Chapter 1}
% expand a bit both
As discussed in Section \ref{sec:Interferon-Induced Proteins with Tetratricopeptide Repeats}, \textit{IFIT} gene family are interferon-stimulated genes (ISGs) that are generally activated early in antiviral response. Their transcription is mainly regulated via interferon-stimulated response elements (IRSE), activated by ISGF3 and interferon regulatory factors (IRF) 3 and 7. In short, these are activated downstream of interferon receptor signalling cascades, and via intracellular and extracellular recognition of PAMPs. As mentioned in Section \ref{subsec:IFIT Responses to Viral Infections}, human IFITs are also known to restrict a plethora of viruses. The examples include IFIT1 and IFIT3 caused restriction of PIV3 (\cite{Rabbani2016Identification3}), and more significantly IFIT1, IFIT2 and IFIT3 caused restiction of human RSV (\cite{Drori2020InfluenzaProteins}). In the latter it was shown that overexpression of these IFITs negativelly influences human RSV replication kinetics, while the silencing of \textit{hIFIT1}, \textit{hIFIT2}, and \textit{hIFIT3} caused the opposite effect. Clearly, IFITs seem to have a important role in RSV life cycle which should be investigated further. It is not know, however, if \textit{hIFITs} are indeed induced following RSV infection and thus able to negatively influence RSV replication in homeostatic pathophysiological conditions.

We hypothesised human \textit{IFITs} to be induced by human RSV infection, which would allow them to enact their antiviral roles as were shown before in the literature. We aimed to systematically test this by initially confirming that our model cell lines are capable of IFIT induction following the treatment of known innate immune system activators such as interferons, LPS (extracellular PAMP), and poly I:C (intracellular PAMP). These assays also allowed us to decipher which innate immune response pathways are responsible for \textit{IFIT} induction. Further, we assessed the \textit{IFITs} induction by hRSV as a function of concentration and time of infection. Lastly we investigated the potential of species cross-protection protection by assessing \textit{hIFIT} induction by bovine RSV infection. All data was also validated in more physiologically relevant cell line. 

\section{Results} \label{sec:Results-Chapter 1}
\input{06. Chapter 1/Subchapters/01. human induction}

\subsection{Validation Proteomics Data and Metaanalysis} \label{subsec:Validation Proteomics Data and Metaanalysis}
\begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{08. Chapter 3/Figs/01. Human/01. merged_proteomics.pdf}
    \caption[Human IFIT proteins detected per fraction.]{\textbf{Human IFIT proteins detected per fraction.} Analysis of summed peptide intensities of (a) hIFIT1, (b) hIFIT2, and (c) hIFIT3 detected per either cytosolic or membrane fractions of A549 cells which were either mock-infected or infected with hRSV MOI 1 for either 24 or 48 hours. There were no hits for IFIT5. This data is from a proteomic study, published in \cite{Jobe2023ViralCondensates}. Samples are composed of biological quintuplicates. Numeric values signify the p-values compared to the respective mock, i.e. top numbers for cytosolic fraction, bottom for membrane fraction.}
    \label{fig:Human IFIT proteomics.}
\end{figure}

To conclude the assessment of human IFIT responses to human RSV, I was kindly provided with a quantitative mass spectrometry dataset of human IFITs detected in either cytosolic or membrane fraction of either mock-infected or cells infected with human RSV at MOI 1, processed 24 or 48 HPI. This dataset was provided by Dr Kelly and Dr Jobe from the Pirbright Institute and its findings have now been published (\cite{Jobe2023ViralCondensates}). Quintuplicates of cytosolic and membrane fractions were isolated by in-gel digestion and analysed by label-free quantitative mass spectrometry. Normalised and log-transformed ion abundance data for hIFIT1, hIFIT2, and hIFIT3 can be seen in Figure \ref{fig:Human IFIT proteomics.}. hIFIT1 cytoplasmic and hIFIT2 membrane datasets had normal distributions and equal variances, while all the others displayed normal distributions with non-equal variances. Human IFIT5 did not yield any hits and thus was excluded from the analysis and we can conclude that its proteome is not drastically changed by hRSV infection, which is in line with our qPCR data. hIFIT1 is the most basally abundant, followed by hIFIT2 and hIFIT3. The basal abundance of hIFITs is around equal between the fractions. The infection at 24 HPI causes increased abundance of all hIFITs in all fractions, more specifically hIFIT1 increased 5x in the cytosolic fraction and 1.3x in membrane fraction; hIFIT2 increased 4x in cytosolic and 2x in membrane fractions; and hIFIT3 increased relatively the most by 13.5x in the cytosolic fraction and 3x in the membrane fraction. The highest total abundance at 24 HPI was still hIFIT1. At 48 HPI we can observe a further increase in abundance for all other than hIFIT1 in membrane fraction which decreased to mock levels. In more detail, hIFIT1 in cytosolic fraction increased further 2x; hIFIT2 increased further 5x in cytosolic and 4x in membrane fraction; and hIFIT3 further increased 3x in cytosolic fraction and 2.5x in membrane fraction. These data together validate our qPCR results seen in Figure \ref{fig:A549 response to hRSV timepoints} and Figure \ref{fig:The effect of ultra-purification, UV-inactivation and INFR inhibition on hIFIT induction following hRSV infection in A549} and highlight the differential spatio-temporal expression dynamics of human IFITs.

normal unequal - 2c 3c 3m

normal euqal - 1c 1m 2m 

\subsection{Bioinformatic Analysis of Human \textit{IFIT} Promoters and Enhancers} \label{subsec:Bioinformatic Analysis of Human \textit{IFIT} Promoters and Enhancers}
write here about what genomic pathways could be responsible for what we see from qpcr


\section{Discussion} \label{sec:Discussion Chapter 1}


cells capable of induction but beas2b less. maybe thats what we should trust (and confirm by bioinformatics)

both hrsv and brsv induce ifits. in human we know its dependant on viral replication and subsequent ifn signalling. we can assume that with brsv it works the same.

although pIC induces loads, replicating rsv withour IFN receptor doesnt, suggesting that even with poly IC what we see is IFN mediated

Interestingly, the maximal induction of \textit{hIFIT5} observed in this study seems to be 10-16 fold, an order of magnitude lower than what is observed for the other \textit{hIFIT}.


maybe make a diagram of the induction (for inducers, for human, for bovine) -> like bar of proportions of inducer invelvement by ifits

its weird that different ifit have different regulation, also between cell lines. i1 is different, i2 and i3 similar and i5 super different. could be different genomic regulation, to be answered by bioinformatics

clearly rsv can induce human ifits and thus they can negatively influence it. the question is, through what mechanism? 



\nomenclature[z-hRSV]{hRSV}{Human Respiratory Syncytial Virus}
\nomenclature[z-IFIT]{IFIT}{Interferon-Induced Protein with Tetratricopeptide Repeats}
\nomenclature[z-RT-qPCR]{RT-qPCR}{Quantitative Real-Time Reverse Transcription Polymerase Chain Reaction}
\nomenclature[z-cDNA]{cDNA}{Complementary DNA}
\nomenclature[z-GAPDH]{GAPDH}{Glyceraldehyde-3-Phosphate Dehydrogenase}
\nomenclature[z-ISGs]{ISGs}{interferon Stimulated Genes}
\nomenclature[z-IFN\(\gamma\)]{IFN\(\gamma\)}{Interferon Gamma}
\nomenclature[z-TLR4]{TLR4}{Toll-like Receptor 4}
\nomenclature[z-LPS]{LPS}{Lipopolysaccharide}
\nomenclature[z-IFN\(\alpha\)]{IFN\(\alpha\)}{Human Interferon Alpha}
\nomenclature[z-MOI]{MOI}{Multiplicities of Infection}
\nomenclature[z-HPI]{HPI}{Hours Post-infection}
\nomenclature[z-bRSV]{bRSV}{Bovine Respiratory Syncytial Virus}
\nomenclature[z-WT]{WT}{Wild-Type}
\nomenclature[z-SH]{SH}{Small-Hydrophobic}
\nomenclature[z-NS]{NS}{Non-Structural}
\nomenclature[z-qPCR]{qPCR}{Quantitative Real-Time PCR}
\nomenclature[z-IRSE]{IRSE}{Interferon-Stimulated Response Elements}
\nomenclature[z-IRF]{IRF}{Interferon Regulatory Factor}
\nomenclature[z-IU]{IU}{International}
\nomenclature[z-IU]{IU}{International}
\nomenclature[z-IU]{IU}{International}
\nomenclature[z-IU]{IU}{International}
\nomenclature[z-IU]{IU}{International}
\nomenclature[z-IU]{IU}{International}
\nomenclature[z-IU]{IU}{International}
\nomenclature[z-IU]{IU}{International}